// steam-session/dist/WebClient.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const debug_1 = __importDefault(require("debug"));
const https_1 = __importDefault(require("https"));
const querystring_1 = __importDefault(require("querystring"));
const stream_1 = require("stream");
const crypto_1 = require("crypto");
const debug = (0, debug_1.default)('steam-session:WebClient');
class WebClient {
    constructor(options) {
        this._agent = options.agent || new https_1.default.Agent({ keepAlive: true });
    }
    async get(url, options) {
        return await this._req(buildUrl(url, options), null, buildRequestOptions('GET', options));
    }
    async post(url, body, options) {
        return await this._req(buildUrl(url, options), body, buildRequestOptions('POST', options));
    }
    async postEncoded(url, body, encoding = 'json', options) {
        let encodedBody, contentType;
        switch (encoding) {
            case 'json':
                contentType = 'application/json';
                encodedBody = JSON.stringify(body);
                break;
            case 'url':
                contentType = 'application/x-www-form-urlencoded';
                encodedBody = querystring_1.default.stringify(body);
                break;
            case 'multipart':
                let boundary = '-----------------------------' + (0, crypto_1.randomBytes)(20).toString('hex');
                contentType = `multipart/form-data; boundary=${boundary}`;
                let encodedBodyParts = [];
                for (let i in body) {
                    let head = `--${boundary}\r\nContent-Disposition: form-data; name="${i}"\r\n\r\n`;
                    let tail = '\r\n';
                    encodedBodyParts = encodedBodyParts.concat([
                        Buffer.from(head, 'utf8'),
                        Buffer.isBuffer(body[i]) ? body[i] : Buffer.from(body[i].toString(), 'utf8'),
                        Buffer.from(tail, 'utf8')
                    ]);
                }
                encodedBodyParts.push(Buffer.from(`--${boundary}--\r\n`, 'utf8'));
                encodedBody = Buffer.concat(encodedBodyParts);
                break;
            default:
                throw new Error(`Unsupported encoding "${encoding}"`);
        }
        return await this._req(buildUrl(url, options), encodedBody, buildRequestOptions('POST', options, { 'content-type': contentType }));
    }
    _req(url, body, options) {
        return new Promise((resolve, reject) => {
            let parsedUrl = urlToHttpOptions(new URL(url));
            if (Buffer.isBuffer(body) || typeof body == 'string') {
                options.headers = options.headers || {};
                options.headers['content-length'] = Buffer.byteLength(body);
            }
            else if (!body) {
                options.headers = options.headers || {};
                options.headers['content-length'] = 0;
            }
            let req = https_1.default.request({ ...parsedUrl, ...options, agent: this._agent }, (res) => {
                let chunks = [];
                res.on('data', chunk => chunks.push(chunk));
                res.on('end', () => {
                    let body = Buffer.concat(chunks);
                    let contentType = (res.headers['content-type'] || '').split(';')[0].toLowerCase().trim();
                    if (contentType.startsWith('text/') || contentType == 'application/json') {
                        body = body.toString('utf8');
                    }
                    if (contentType == 'application/json') {
                        try {
                            body = JSON.parse(body);
                        }
                        catch (ex) {
                            debug('error parsing json response %o', ex);
                        }
                    }
                    resolve({ res, body });
                });
            });
            req.on('error', reject);
            if (body instanceof stream_1.Readable) {
                body.pipe(req);
            }
            else if (Buffer.isBuffer(body) || typeof body == 'string') {
                req.end(body);
            }
            else {
                req.end();
            }
        });
    }
}
exports.default = WebClient;
function buildUrl(url, options) {
    options = options || {};
    if (options.queryString && Object.keys(options.queryString).length > 0) {
        url += url.includes('?') ? '&' : '?';
        url += querystring_1.default.stringify(options.queryString);
    }
    return url;
}
function buildRequestOptions(method, options, extraHeaders) {
    options = options || {};
    let output = { method };
    if (options.headers) {
        output.headers = options.headers;
    }
    if (extraHeaders) {
        output.headers = output.headers || {};
        output.headers = { ...output.headers, ...extraHeaders };
    }
    output.rejectUnauthorized = false;
    return output;
}
// Polyfill for node versions prior to 14.18.0
function urlToHttpOptions(url) {
    let options = {
        protocol: url.protocol,
        hostname: typeof url.hostname == 'string' &&
            url.hostname.startsWith('[')
            ? url.hostname.slice(1, -1)
            : url.hostname,
        hash: url.hash,
        search: url.search,
        pathname: url.pathname,
        path: `${url.pathname || ''}${url.search || ''}`,
        href: url.href,
    };
    if (url.port !== '') {
        options.port = Number(url.port);
    }
    if (url.username || url.password) {
        options.auth = [url.username, url.password].map(v => decodeURIComponent(v || '')).join(':');
    }
    return options;
}
