// steam-session/dist/enums-steam/EBanContentCheckResult.js
"use strict";
/* eslint-disable */
// Auto-generated by generate-enums script on Thu Mar 02 2023 22:18:52 GMT-0500 (Eastern Standard Time)
Object.defineProperty(exports, "__esModule", { value: true });
var EBanContentCheckResult;
(function (EBanContentCheckResult) {
    EBanContentCheckResult[EBanContentCheckResult["NotScanned"] = 0] = "NotScanned";
    EBanContentCheckResult[EBanContentCheckResult["Reset"] = 1] = "Reset";
    EBanContentCheckResult[EBanContentCheckResult["NeedsChecking"] = 2] = "NeedsChecking";
    EBanContentCheckResult[EBanContentCheckResult["VeryUnlikely"] = 5] = "VeryUnlikely";
    EBanContentCheckResult[EBanContentCheckResult["Unlikely"] = 30] = "Unlikely";
    EBanContentCheckResult[EBanContentCheckResult["Possible"] = 50] = "Possible";
    EBanContentCheckResult[EBanContentCheckResult["Likely"] = 75] = "Likely";
    EBanContentCheckResult[EBanContentCheckResult["VeryLikely"] = 100] = "VeryLikely";
})(EBanContentCheckResult || (EBanContentCheckResult = {}));
exports.default = EBanContentCheckResult;
