// steam-session/dist/enums-steam/EMsg.js
"use strict";
/* eslint-disable */
// Auto-generated by generate-enums script on Thu Mar 02 2023 22:18:52 GMT-0500 (Eastern Standard Time)
Object.defineProperty(exports, "__esModule", { value: true });
var EMsg;
(function (EMsg) {
    EMsg[EMsg["Invalid"] = 0] = "Invalid";
    EMsg[EMsg["Multi"] = 1] = "Multi";
    EMsg[EMsg["ProtobufWrapped"] = 2] = "ProtobufWrapped";
    EMsg[EMsg["BaseGeneral"] = 100] = "BaseGeneral";
    EMsg[EMsg["GenericReply"] = 100] = "GenericReply";
    EMsg[EMsg["DestJobFailed"] = 113] = "DestJobFailed";
    EMsg[EMsg["Alert"] = 115] = "Alert";
    EMsg[EMsg["SCIDRequest"] = 120] = "SCIDRequest";
    EMsg[EMsg["SCIDResponse"] = 121] = "SCIDResponse";
    EMsg[EMsg["JobHeartbeat"] = 123] = "JobHeartbeat";
    EMsg[EMsg["HubConnect"] = 124] = "HubConnect";
    EMsg[EMsg["Subscribe"] = 126] = "Subscribe";
    EMsg[EMsg["RouteMessage"] = 127] = "RouteMessage";
    EMsg[EMsg["RemoteSysID"] = 128] = "RemoteSysID";
    EMsg[EMsg["AMCreateAccountResponse"] = 129] = "AMCreateAccountResponse";
    EMsg[EMsg["WGRequest"] = 130] = "WGRequest";
    EMsg[EMsg["WGResponse"] = 131] = "WGResponse";
    EMsg[EMsg["KeepAlive"] = 132] = "KeepAlive";
    EMsg[EMsg["WebAPIJobRequest"] = 133] = "WebAPIJobRequest";
    EMsg[EMsg["WebAPIJobResponse"] = 134] = "WebAPIJobResponse";
    EMsg[EMsg["ClientSessionStart"] = 135] = "ClientSessionStart";
    EMsg[EMsg["ClientSessionEnd"] = 136] = "ClientSessionEnd";
    EMsg[EMsg["ClientSessionUpdateAuthTicket"] = 137] = "ClientSessionUpdateAuthTicket";
    EMsg[EMsg["ClientSessionUpdate"] = 137] = "ClientSessionUpdate";
    EMsg[EMsg["StatsDeprecated"] = 138] = "StatsDeprecated";
    EMsg[EMsg["Ping"] = 139] = "Ping";
    EMsg[EMsg["PingResponse"] = 140] = "PingResponse";
    EMsg[EMsg["Stats"] = 141] = "Stats";
    EMsg[EMsg["RequestFullStatsBlock"] = 142] = "RequestFullStatsBlock";
    EMsg[EMsg["LoadDBOCacheItem"] = 143] = "LoadDBOCacheItem";
    EMsg[EMsg["LoadDBOCacheItemResponse"] = 144] = "LoadDBOCacheItemResponse";
    EMsg[EMsg["InvalidateDBOCacheItems"] = 145] = "InvalidateDBOCacheItems";
    EMsg[EMsg["ServiceMethod"] = 146] = "ServiceMethod";
    EMsg[EMsg["ServiceMethodResponse"] = 147] = "ServiceMethodResponse";
    EMsg[EMsg["ClientPackageVersions"] = 148] = "ClientPackageVersions";
    EMsg[EMsg["TimestampRequest"] = 149] = "TimestampRequest";
    EMsg[EMsg["TimestampResponse"] = 150] = "TimestampResponse";
    EMsg[EMsg["ServiceMethodCallFromClient"] = 151] = "ServiceMethodCallFromClient";
    EMsg[EMsg["ServiceMethodSendToClient"] = 152] = "ServiceMethodSendToClient";
    EMsg[EMsg["BaseShell"] = 200] = "BaseShell";
    EMsg[EMsg["AssignSysID"] = 200] = "AssignSysID";
    EMsg[EMsg["Exit"] = 201] = "Exit";
    EMsg[EMsg["DirRequest"] = 202] = "DirRequest";
    EMsg[EMsg["DirResponse"] = 203] = "DirResponse";
    EMsg[EMsg["ZipRequest"] = 204] = "ZipRequest";
    EMsg[EMsg["ZipResponse"] = 205] = "ZipResponse";
    EMsg[EMsg["UpdateRecordResponse"] = 215] = "UpdateRecordResponse";
    EMsg[EMsg["UpdateCreditCardRequest"] = 221] = "UpdateCreditCardRequest";
    EMsg[EMsg["UpdateUserBanResponse"] = 225] = "UpdateUserBanResponse";
    EMsg[EMsg["PrepareToExit"] = 226] = "PrepareToExit";
    EMsg[EMsg["ContentDescriptionUpdate"] = 227] = "ContentDescriptionUpdate";
    EMsg[EMsg["TestResetServer"] = 228] = "TestResetServer";
    EMsg[EMsg["UniverseChanged"] = 229] = "UniverseChanged";
    EMsg[EMsg["ShellConfigInfoUpdate"] = 230] = "ShellConfigInfoUpdate";
    EMsg[EMsg["RequestWindowsEventLogEntries"] = 233] = "RequestWindowsEventLogEntries";
    EMsg[EMsg["ProvideWindowsEventLogEntries"] = 234] = "ProvideWindowsEventLogEntries";
    EMsg[EMsg["ShellSearchLogs"] = 235] = "ShellSearchLogs";
    EMsg[EMsg["ShellSearchLogsResponse"] = 236] = "ShellSearchLogsResponse";
    EMsg[EMsg["ShellCheckWindowsUpdates"] = 237] = "ShellCheckWindowsUpdates";
    EMsg[EMsg["ShellCheckWindowsUpdatesResponse"] = 238] = "ShellCheckWindowsUpdatesResponse";
    EMsg[EMsg["ShellFlushUserLicenseCache"] = 239] = "ShellFlushUserLicenseCache";
    EMsg[EMsg["TestFlushDelayedSQL"] = 240] = "TestFlushDelayedSQL";
    EMsg[EMsg["TestFlushDelayedSQLResponse"] = 241] = "TestFlushDelayedSQLResponse";
    EMsg[EMsg["EnsureExecuteScheduledTask_TEST"] = 242] = "EnsureExecuteScheduledTask_TEST";
    EMsg[EMsg["EnsureExecuteScheduledTaskResponse_TEST"] = 243] = "EnsureExecuteScheduledTaskResponse_TEST";
    EMsg[EMsg["UpdateScheduledTaskEnableState_TEST"] = 244] = "UpdateScheduledTaskEnableState_TEST";
    EMsg[EMsg["UpdateScheduledTaskEnableStateResponse_TEST"] = 245] = "UpdateScheduledTaskEnableStateResponse_TEST";
    EMsg[EMsg["ContentDescriptionDeltaUpdate"] = 246] = "ContentDescriptionDeltaUpdate";
    EMsg[EMsg["GMShellAndServerAddressUpdates"] = 247] = "GMShellAndServerAddressUpdates";
    EMsg[EMsg["BaseGM"] = 300] = "BaseGM";
    EMsg[EMsg["Heartbeat"] = 300] = "Heartbeat";
    EMsg[EMsg["ShellFailed"] = 301] = "ShellFailed";
    EMsg[EMsg["ExitShells"] = 307] = "ExitShells";
    EMsg[EMsg["ExitShell"] = 308] = "ExitShell";
    EMsg[EMsg["GracefulExitShell"] = 309] = "GracefulExitShell";
    EMsg[EMsg["LicenseProcessingComplete"] = 316] = "LicenseProcessingComplete";
    EMsg[EMsg["SetTestFlag"] = 317] = "SetTestFlag";
    EMsg[EMsg["QueuedEmailsComplete"] = 318] = "QueuedEmailsComplete";
    EMsg[EMsg["GMReportPHPError"] = 319] = "GMReportPHPError";
    EMsg[EMsg["GMDRMSync"] = 320] = "GMDRMSync";
    EMsg[EMsg["PhysicalBoxInventory"] = 321] = "PhysicalBoxInventory";
    EMsg[EMsg["UpdateConfigFile"] = 322] = "UpdateConfigFile";
    EMsg[EMsg["TestInitDB"] = 323] = "TestInitDB";
    EMsg[EMsg["GMWriteConfigToSQL"] = 324] = "GMWriteConfigToSQL";
    EMsg[EMsg["GMLoadActivationCodes"] = 325] = "GMLoadActivationCodes";
    EMsg[EMsg["GMQueueForFBS"] = 326] = "GMQueueForFBS";
    EMsg[EMsg["GMSchemaConversionResults"] = 327] = "GMSchemaConversionResults";
    EMsg[EMsg["GMSchemaConversionResultsResponse"] = 328] = "GMSchemaConversionResultsResponse";
    EMsg[EMsg["GMWriteShellFailureToSQL"] = 329] = "GMWriteShellFailureToSQL";
    EMsg[EMsg["GMWriteStatsToSOS"] = 330] = "GMWriteStatsToSOS";
    EMsg[EMsg["GMGetServiceMethodRouting"] = 331] = "GMGetServiceMethodRouting";
    EMsg[EMsg["GMGetServiceMethodRoutingResponse"] = 332] = "GMGetServiceMethodRoutingResponse";
    EMsg[EMsg["GMConvertUserWallets"] = 333] = "GMConvertUserWallets";
    EMsg[EMsg["GMTestNextBuildSchemaConversion"] = 334] = "GMTestNextBuildSchemaConversion";
    EMsg[EMsg["GMTestNextBuildSchemaConversionResponse"] = 335] = "GMTestNextBuildSchemaConversionResponse";
    EMsg[EMsg["ExpectShellRestart"] = 336] = "ExpectShellRestart";
    EMsg[EMsg["HotFixProgress"] = 337] = "HotFixProgress";
    EMsg[EMsg["GMStatsForwardToAdminConnections"] = 338] = "GMStatsForwardToAdminConnections";
    EMsg[EMsg["GMGetModifiedConVars"] = 339] = "GMGetModifiedConVars";
    EMsg[EMsg["GMGetModifiedConVarsResponse"] = 340] = "GMGetModifiedConVarsResponse";
    EMsg[EMsg["BaseAIS"] = 400] = "BaseAIS";
    EMsg[EMsg["AISRefreshContentDescription"] = 401] = "AISRefreshContentDescription";
    EMsg[EMsg["AISRequestContentDescription"] = 402] = "AISRequestContentDescription";
    EMsg[EMsg["AISUpdateAppInfo"] = 403] = "AISUpdateAppInfo";
    EMsg[EMsg["AISUpdatePackageCosts"] = 404] = "AISUpdatePackageCosts";
    EMsg[EMsg["AISUpdatePackageInfo"] = 404] = "AISUpdatePackageInfo";
    EMsg[EMsg["AISGetPackageChangeNumber"] = 405] = "AISGetPackageChangeNumber";
    EMsg[EMsg["AISGetPackageChangeNumberResponse"] = 406] = "AISGetPackageChangeNumberResponse";
    EMsg[EMsg["AISAppInfoTableChanged"] = 407] = "AISAppInfoTableChanged";
    EMsg[EMsg["AISUpdatePackageCostsResponse"] = 408] = "AISUpdatePackageCostsResponse";
    EMsg[EMsg["AISCreateMarketingMessage"] = 409] = "AISCreateMarketingMessage";
    EMsg[EMsg["AISCreateMarketingMessageResponse"] = 410] = "AISCreateMarketingMessageResponse";
    EMsg[EMsg["AISGetMarketingMessage"] = 411] = "AISGetMarketingMessage";
    EMsg[EMsg["AISGetMarketingMessageResponse"] = 412] = "AISGetMarketingMessageResponse";
    EMsg[EMsg["AISUpdateMarketingMessage"] = 413] = "AISUpdateMarketingMessage";
    EMsg[EMsg["AISUpdateMarketingMessageResponse"] = 414] = "AISUpdateMarketingMessageResponse";
    EMsg[EMsg["AISRequestMarketingMessageUpdate"] = 415] = "AISRequestMarketingMessageUpdate";
    EMsg[EMsg["AISDeleteMarketingMessage"] = 416] = "AISDeleteMarketingMessage";
    EMsg[EMsg["AISGetMarketingTreatments"] = 419] = "AISGetMarketingTreatments";
    EMsg[EMsg["AISGetMarketingTreatmentsResponse"] = 420] = "AISGetMarketingTreatmentsResponse";
    EMsg[EMsg["AISRequestMarketingTreatmentUpdate"] = 421] = "AISRequestMarketingTreatmentUpdate";
    EMsg[EMsg["AISTestAddPackage"] = 422] = "AISTestAddPackage";
    EMsg[EMsg["AIGetAppGCFlags"] = 423] = "AIGetAppGCFlags";
    EMsg[EMsg["AIGetAppGCFlagsResponse"] = 424] = "AIGetAppGCFlagsResponse";
    EMsg[EMsg["AIGetAppList"] = 425] = "AIGetAppList";
    EMsg[EMsg["AIGetAppListResponse"] = 426] = "AIGetAppListResponse";
    EMsg[EMsg["AIGetAppInfo"] = 427] = "AIGetAppInfo";
    EMsg[EMsg["AIGetAppInfoResponse"] = 428] = "AIGetAppInfoResponse";
    EMsg[EMsg["AISGetCouponDefinition"] = 429] = "AISGetCouponDefinition";
    EMsg[EMsg["AISGetCouponDefinitionResponse"] = 430] = "AISGetCouponDefinitionResponse";
    EMsg[EMsg["AISUpdateSlaveContentDescription"] = 431] = "AISUpdateSlaveContentDescription";
    EMsg[EMsg["AISUpdateSlaveContentDescriptionResponse"] = 432] = "AISUpdateSlaveContentDescriptionResponse";
    EMsg[EMsg["AISTestEnableGC"] = 433] = "AISTestEnableGC";
    EMsg[EMsg["BaseAM"] = 500] = "BaseAM";
    EMsg[EMsg["AMUpdateUserBanRequest"] = 504] = "AMUpdateUserBanRequest";
    EMsg[EMsg["AMAddLicense"] = 505] = "AMAddLicense";
    EMsg[EMsg["AMBeginProcessingLicenses"] = 507] = "AMBeginProcessingLicenses";
    EMsg[EMsg["AMSendSystemIMToUser"] = 508] = "AMSendSystemIMToUser";
    EMsg[EMsg["AMExtendLicense"] = 509] = "AMExtendLicense";
    EMsg[EMsg["AMAddMinutesToLicense"] = 510] = "AMAddMinutesToLicense";
    EMsg[EMsg["AMCancelLicense"] = 511] = "AMCancelLicense";
    EMsg[EMsg["AMInitPurchase"] = 512] = "AMInitPurchase";
    EMsg[EMsg["AMPurchaseResponse"] = 513] = "AMPurchaseResponse";
    EMsg[EMsg["AMGetFinalPrice"] = 514] = "AMGetFinalPrice";
    EMsg[EMsg["AMGetFinalPriceResponse"] = 515] = "AMGetFinalPriceResponse";
    EMsg[EMsg["AMGetLegacyGameKey"] = 516] = "AMGetLegacyGameKey";
    EMsg[EMsg["AMGetLegacyGameKeyResponse"] = 517] = "AMGetLegacyGameKeyResponse";
    EMsg[EMsg["AMFindHungTransactions"] = 518] = "AMFindHungTransactions";
    EMsg[EMsg["AMSetAccountTrustedRequest"] = 519] = "AMSetAccountTrustedRequest";
    EMsg[EMsg["AMCompletePurchase"] = 521] = "AMCompletePurchase";
    EMsg[EMsg["AMCancelPurchase"] = 522] = "AMCancelPurchase";
    EMsg[EMsg["AMNewChallenge"] = 523] = "AMNewChallenge";
    EMsg[EMsg["AMLoadOEMTickets"] = 524] = "AMLoadOEMTickets";
    EMsg[EMsg["AMFixPendingPurchase"] = 525] = "AMFixPendingPurchase";
    EMsg[EMsg["AMFixPendingPurchaseResponse"] = 526] = "AMFixPendingPurchaseResponse";
    EMsg[EMsg["AMIsUserBanned"] = 527] = "AMIsUserBanned";
    EMsg[EMsg["AMRegisterKey"] = 528] = "AMRegisterKey";
    EMsg[EMsg["AMLoadActivationCodes"] = 529] = "AMLoadActivationCodes";
    EMsg[EMsg["AMLoadActivationCodesResponse"] = 530] = "AMLoadActivationCodesResponse";
    EMsg[EMsg["AMLookupKeyResponse"] = 531] = "AMLookupKeyResponse";
    EMsg[EMsg["AMLookupKey"] = 532] = "AMLookupKey";
    EMsg[EMsg["AMChatCleanup"] = 533] = "AMChatCleanup";
    EMsg[EMsg["AMClanCleanup"] = 534] = "AMClanCleanup";
    EMsg[EMsg["AMFixPendingRefund"] = 535] = "AMFixPendingRefund";
    EMsg[EMsg["AMReverseChargeback"] = 536] = "AMReverseChargeback";
    EMsg[EMsg["AMReverseChargebackResponse"] = 537] = "AMReverseChargebackResponse";
    EMsg[EMsg["AMClanCleanupList"] = 538] = "AMClanCleanupList";
    EMsg[EMsg["AMGetLicenses"] = 539] = "AMGetLicenses";
    EMsg[EMsg["AMGetLicensesResponse"] = 540] = "AMGetLicensesResponse";
    EMsg[EMsg["AMSendCartRepurchase"] = 541] = "AMSendCartRepurchase";
    EMsg[EMsg["AMSendCartRepurchaseResponse"] = 542] = "AMSendCartRepurchaseResponse";
    EMsg[EMsg["AllowUserToPlayQuery"] = 550] = "AllowUserToPlayQuery";
    EMsg[EMsg["AllowUserToPlayResponse"] = 551] = "AllowUserToPlayResponse";
    EMsg[EMsg["AMVerfiyUser"] = 552] = "AMVerfiyUser";
    EMsg[EMsg["AMClientNotPlaying"] = 553] = "AMClientNotPlaying";
    EMsg[EMsg["ClientRequestFriendship"] = 554] = "ClientRequestFriendship";
    EMsg[EMsg["AMClientRequestFriendship"] = 554] = "AMClientRequestFriendship";
    EMsg[EMsg["AMRelayPublishStatus"] = 555] = "AMRelayPublishStatus";
    EMsg[EMsg["AMResetCommunityContent"] = 556] = "AMResetCommunityContent";
    EMsg[EMsg["AMPrimePersonaStateCache"] = 557] = "AMPrimePersonaStateCache";
    EMsg[EMsg["AMAllowUserContentQuery"] = 558] = "AMAllowUserContentQuery";
    EMsg[EMsg["AMAllowUserContentResponse"] = 559] = "AMAllowUserContentResponse";
    EMsg[EMsg["AMInitPurchaseResponse"] = 560] = "AMInitPurchaseResponse";
    EMsg[EMsg["AMRevokePurchaseResponse"] = 561] = "AMRevokePurchaseResponse";
    EMsg[EMsg["AMLockProfile"] = 562] = "AMLockProfile";
    EMsg[EMsg["AMRefreshGuestPasses"] = 563] = "AMRefreshGuestPasses";
    EMsg[EMsg["AMInviteUserToClan"] = 564] = "AMInviteUserToClan";
    EMsg[EMsg["AMAcknowledgeClanInvite"] = 565] = "AMAcknowledgeClanInvite";
    EMsg[EMsg["AMGrantGuestPasses"] = 566] = "AMGrantGuestPasses";
    EMsg[EMsg["AMClanDataUpdated"] = 567] = "AMClanDataUpdated";
    EMsg[EMsg["AMReloadAccount"] = 568] = "AMReloadAccount";
    EMsg[EMsg["AMClientChatMsgRelay"] = 569] = "AMClientChatMsgRelay";
    EMsg[EMsg["AMChatMulti"] = 570] = "AMChatMulti";
    EMsg[EMsg["AMClientChatInviteRelay"] = 571] = "AMClientChatInviteRelay";
    EMsg[EMsg["AMChatInvite"] = 572] = "AMChatInvite";
    EMsg[EMsg["AMClientJoinChatRelay"] = 573] = "AMClientJoinChatRelay";
    EMsg[EMsg["AMClientChatMemberInfoRelay"] = 574] = "AMClientChatMemberInfoRelay";
    EMsg[EMsg["AMPublishChatMemberInfo"] = 575] = "AMPublishChatMemberInfo";
    EMsg[EMsg["AMClientAcceptFriendInvite"] = 576] = "AMClientAcceptFriendInvite";
    EMsg[EMsg["AMChatEnter"] = 577] = "AMChatEnter";
    EMsg[EMsg["AMClientPublishRemovalFromSource"] = 578] = "AMClientPublishRemovalFromSource";
    EMsg[EMsg["AMChatActionResult"] = 579] = "AMChatActionResult";
    EMsg[EMsg["AMFindAccounts"] = 580] = "AMFindAccounts";
    EMsg[EMsg["AMFindAccountsResponse"] = 581] = "AMFindAccountsResponse";
    EMsg[EMsg["AMRequestAccountData"] = 582] = "AMRequestAccountData";
    EMsg[EMsg["AMRequestAccountDataResponse"] = 583] = "AMRequestAccountDataResponse";
    EMsg[EMsg["AMSetAccountFlags"] = 584] = "AMSetAccountFlags";
    EMsg[EMsg["AMCreateClan"] = 586] = "AMCreateClan";
    EMsg[EMsg["AMCreateClanResponse"] = 587] = "AMCreateClanResponse";
    EMsg[EMsg["AMGetClanDetails"] = 588] = "AMGetClanDetails";
    EMsg[EMsg["AMGetClanDetailsResponse"] = 589] = "AMGetClanDetailsResponse";
    EMsg[EMsg["AMSetPersonaName"] = 590] = "AMSetPersonaName";
    EMsg[EMsg["AMSetAvatar"] = 591] = "AMSetAvatar";
    EMsg[EMsg["AMAuthenticateUser"] = 592] = "AMAuthenticateUser";
    EMsg[EMsg["AMAuthenticateUserResponse"] = 593] = "AMAuthenticateUserResponse";
    EMsg[EMsg["AMGetAccountFriendsCount"] = 594] = "AMGetAccountFriendsCount";
    EMsg[EMsg["AMGetAccountFriendsCountResponse"] = 595] = "AMGetAccountFriendsCountResponse";
    EMsg[EMsg["AMP2PIntroducerMessage"] = 596] = "AMP2PIntroducerMessage";
    EMsg[EMsg["ClientChatAction"] = 597] = "ClientChatAction";
    EMsg[EMsg["AMClientChatActionRelay"] = 598] = "AMClientChatActionRelay";
    EMsg[EMsg["BaseVS"] = 600] = "BaseVS";
    EMsg[EMsg["ReqChallenge"] = 600] = "ReqChallenge";
    EMsg[EMsg["VACResponse"] = 601] = "VACResponse";
    EMsg[EMsg["ReqChallengeTest"] = 602] = "ReqChallengeTest";
    EMsg[EMsg["VSMarkCheat"] = 604] = "VSMarkCheat";
    EMsg[EMsg["VSAddCheat"] = 605] = "VSAddCheat";
    EMsg[EMsg["VSPurgeCodeModDB"] = 606] = "VSPurgeCodeModDB";
    EMsg[EMsg["VSGetChallengeResults"] = 607] = "VSGetChallengeResults";
    EMsg[EMsg["VSChallengeResultText"] = 608] = "VSChallengeResultText";
    EMsg[EMsg["VSReportLingerer"] = 609] = "VSReportLingerer";
    EMsg[EMsg["VSRequestManagedChallenge"] = 610] = "VSRequestManagedChallenge";
    EMsg[EMsg["VSLoadDBFinished"] = 611] = "VSLoadDBFinished";
    EMsg[EMsg["BaseDRMS"] = 625] = "BaseDRMS";
    EMsg[EMsg["DRMBuildBlobRequest"] = 628] = "DRMBuildBlobRequest";
    EMsg[EMsg["DRMBuildBlobResponse"] = 629] = "DRMBuildBlobResponse";
    EMsg[EMsg["DRMResolveGuidRequest"] = 630] = "DRMResolveGuidRequest";
    EMsg[EMsg["DRMResolveGuidResponse"] = 631] = "DRMResolveGuidResponse";
    EMsg[EMsg["DRMVariabilityReport"] = 633] = "DRMVariabilityReport";
    EMsg[EMsg["DRMVariabilityReportResponse"] = 634] = "DRMVariabilityReportResponse";
    EMsg[EMsg["DRMStabilityReport"] = 635] = "DRMStabilityReport";
    EMsg[EMsg["DRMStabilityReportResponse"] = 636] = "DRMStabilityReportResponse";
    EMsg[EMsg["DRMDetailsReportRequest"] = 637] = "DRMDetailsReportRequest";
    EMsg[EMsg["DRMDetailsReportResponse"] = 638] = "DRMDetailsReportResponse";
    EMsg[EMsg["DRMProcessFile"] = 639] = "DRMProcessFile";
    EMsg[EMsg["DRMAdminUpdate"] = 640] = "DRMAdminUpdate";
    EMsg[EMsg["DRMAdminUpdateResponse"] = 641] = "DRMAdminUpdateResponse";
    EMsg[EMsg["DRMSync"] = 642] = "DRMSync";
    EMsg[EMsg["DRMSyncResponse"] = 643] = "DRMSyncResponse";
    EMsg[EMsg["DRMProcessFileResponse"] = 644] = "DRMProcessFileResponse";
    EMsg[EMsg["DRMEmptyGuidCache"] = 645] = "DRMEmptyGuidCache";
    EMsg[EMsg["DRMEmptyGuidCacheResponse"] = 646] = "DRMEmptyGuidCacheResponse";
    EMsg[EMsg["BaseCS"] = 650] = "BaseCS";
    EMsg[EMsg["CSUserContentRequest"] = 652] = "CSUserContentRequest";
    EMsg[EMsg["BaseClient"] = 700] = "BaseClient";
    EMsg[EMsg["ClientLogOn_Deprecated"] = 701] = "ClientLogOn_Deprecated";
    EMsg[EMsg["ClientAnonLogOn_Deprecated"] = 702] = "ClientAnonLogOn_Deprecated";
    EMsg[EMsg["ClientHeartBeat"] = 703] = "ClientHeartBeat";
    EMsg[EMsg["ClientVACResponse"] = 704] = "ClientVACResponse";
    EMsg[EMsg["ClientGamesPlayed_obsolete"] = 705] = "ClientGamesPlayed_obsolete";
    EMsg[EMsg["ClientLogOff"] = 706] = "ClientLogOff";
    EMsg[EMsg["ClientNoUDPConnectivity"] = 707] = "ClientNoUDPConnectivity";
    EMsg[EMsg["ClientInformOfCreateAccount"] = 708] = "ClientInformOfCreateAccount";
    EMsg[EMsg["ClientAckVACBan"] = 709] = "ClientAckVACBan";
    EMsg[EMsg["ClientConnectionStats"] = 710] = "ClientConnectionStats";
    EMsg[EMsg["ClientInitPurchase"] = 711] = "ClientInitPurchase";
    EMsg[EMsg["ClientPingResponse"] = 712] = "ClientPingResponse";
    EMsg[EMsg["ClientRemoveFriend"] = 714] = "ClientRemoveFriend";
    EMsg[EMsg["ClientGamesPlayedNoDataBlob"] = 715] = "ClientGamesPlayedNoDataBlob";
    EMsg[EMsg["ClientChangeStatus"] = 716] = "ClientChangeStatus";
    EMsg[EMsg["ClientVacStatusResponse"] = 717] = "ClientVacStatusResponse";
    EMsg[EMsg["ClientFriendMsg"] = 718] = "ClientFriendMsg";
    EMsg[EMsg["ClientGameConnect_obsolete"] = 719] = "ClientGameConnect_obsolete";
    EMsg[EMsg["ClientGamesPlayed2_obsolete"] = 720] = "ClientGamesPlayed2_obsolete";
    EMsg[EMsg["ClientGameEnded_obsolete"] = 721] = "ClientGameEnded_obsolete";
    EMsg[EMsg["ClientGetFinalPrice"] = 722] = "ClientGetFinalPrice";
    EMsg[EMsg["ClientSystemIM"] = 726] = "ClientSystemIM";
    EMsg[EMsg["ClientSystemIMAck"] = 727] = "ClientSystemIMAck";
    EMsg[EMsg["ClientGetLicenses"] = 728] = "ClientGetLicenses";
    EMsg[EMsg["ClientCancelLicense"] = 729] = "ClientCancelLicense";
    EMsg[EMsg["ClientGetLegacyGameKey"] = 730] = "ClientGetLegacyGameKey";
    EMsg[EMsg["ClientContentServerLogOn_Deprecated"] = 731] = "ClientContentServerLogOn_Deprecated";
    EMsg[EMsg["ClientAckVACBan2"] = 732] = "ClientAckVACBan2";
    EMsg[EMsg["ClientAckMessageByGID"] = 735] = "ClientAckMessageByGID";
    EMsg[EMsg["ClientGetPurchaseReceipts"] = 736] = "ClientGetPurchaseReceipts";
    EMsg[EMsg["ClientAckPurchaseReceipt"] = 737] = "ClientAckPurchaseReceipt";
    EMsg[EMsg["ClientGamesPlayed3_obsolete"] = 738] = "ClientGamesPlayed3_obsolete";
    EMsg[EMsg["ClientSendGuestPass"] = 739] = "ClientSendGuestPass";
    EMsg[EMsg["ClientAckGuestPass"] = 740] = "ClientAckGuestPass";
    EMsg[EMsg["ClientRedeemGuestPass"] = 741] = "ClientRedeemGuestPass";
    EMsg[EMsg["ClientGamesPlayed"] = 742] = "ClientGamesPlayed";
    EMsg[EMsg["ClientRegisterKey"] = 743] = "ClientRegisterKey";
    EMsg[EMsg["ClientInviteUserToClan"] = 744] = "ClientInviteUserToClan";
    EMsg[EMsg["ClientAcknowledgeClanInvite"] = 745] = "ClientAcknowledgeClanInvite";
    EMsg[EMsg["ClientPurchaseWithMachineID"] = 746] = "ClientPurchaseWithMachineID";
    EMsg[EMsg["ClientAppUsageEvent"] = 747] = "ClientAppUsageEvent";
    EMsg[EMsg["ClientGetGiftTargetList"] = 748] = "ClientGetGiftTargetList";
    EMsg[EMsg["ClientGetGiftTargetListResponse"] = 749] = "ClientGetGiftTargetListResponse";
    EMsg[EMsg["ClientLogOnResponse"] = 751] = "ClientLogOnResponse";
    EMsg[EMsg["ClientVACChallenge"] = 753] = "ClientVACChallenge";
    EMsg[EMsg["ClientSetHeartbeatRate"] = 755] = "ClientSetHeartbeatRate";
    EMsg[EMsg["ClientNotLoggedOnDeprecated"] = 756] = "ClientNotLoggedOnDeprecated";
    EMsg[EMsg["ClientLoggedOff"] = 757] = "ClientLoggedOff";
    EMsg[EMsg["GSApprove"] = 758] = "GSApprove";
    EMsg[EMsg["GSDeny"] = 759] = "GSDeny";
    EMsg[EMsg["GSKick"] = 760] = "GSKick";
    EMsg[EMsg["ClientCreateAcctResponse"] = 761] = "ClientCreateAcctResponse";
    EMsg[EMsg["ClientPurchaseResponse"] = 763] = "ClientPurchaseResponse";
    EMsg[EMsg["ClientPing"] = 764] = "ClientPing";
    EMsg[EMsg["ClientNOP"] = 765] = "ClientNOP";
    EMsg[EMsg["ClientPersonaState"] = 766] = "ClientPersonaState";
    EMsg[EMsg["ClientFriendsList"] = 767] = "ClientFriendsList";
    EMsg[EMsg["ClientAccountInfo"] = 768] = "ClientAccountInfo";
    EMsg[EMsg["ClientVacStatusQuery"] = 770] = "ClientVacStatusQuery";
    EMsg[EMsg["ClientNewsUpdate"] = 771] = "ClientNewsUpdate";
    EMsg[EMsg["ClientGameConnectDeny"] = 773] = "ClientGameConnectDeny";
    EMsg[EMsg["GSStatusReply"] = 774] = "GSStatusReply";
    EMsg[EMsg["ClientGetFinalPriceResponse"] = 775] = "ClientGetFinalPriceResponse";
    EMsg[EMsg["ClientGameConnectTokens"] = 779] = "ClientGameConnectTokens";
    EMsg[EMsg["ClientLicenseList"] = 780] = "ClientLicenseList";
    EMsg[EMsg["ClientCancelLicenseResponse"] = 781] = "ClientCancelLicenseResponse";
    EMsg[EMsg["ClientVACBanStatus"] = 782] = "ClientVACBanStatus";
    EMsg[EMsg["ClientCMList"] = 783] = "ClientCMList";
    EMsg[EMsg["ClientEncryptPct"] = 784] = "ClientEncryptPct";
    EMsg[EMsg["ClientGetLegacyGameKeyResponse"] = 785] = "ClientGetLegacyGameKeyResponse";
    EMsg[EMsg["ClientFavoritesList"] = 786] = "ClientFavoritesList";
    EMsg[EMsg["CSUserContentApprove"] = 787] = "CSUserContentApprove";
    EMsg[EMsg["CSUserContentDeny"] = 788] = "CSUserContentDeny";
    EMsg[EMsg["ClientInitPurchaseResponse"] = 789] = "ClientInitPurchaseResponse";
    EMsg[EMsg["ClientAddFriend"] = 791] = "ClientAddFriend";
    EMsg[EMsg["ClientAddFriendResponse"] = 792] = "ClientAddFriendResponse";
    EMsg[EMsg["ClientInviteFriend"] = 793] = "ClientInviteFriend";
    EMsg[EMsg["ClientInviteFriendResponse"] = 794] = "ClientInviteFriendResponse";
    EMsg[EMsg["ClientSendGuestPassResponse"] = 795] = "ClientSendGuestPassResponse";
    EMsg[EMsg["ClientAckGuestPassResponse"] = 796] = "ClientAckGuestPassResponse";
    EMsg[EMsg["ClientRedeemGuestPassResponse"] = 797] = "ClientRedeemGuestPassResponse";
    EMsg[EMsg["ClientUpdateGuestPassesList"] = 798] = "ClientUpdateGuestPassesList";
    EMsg[EMsg["ClientChatMsg"] = 799] = "ClientChatMsg";
    EMsg[EMsg["ClientChatInvite"] = 800] = "ClientChatInvite";
    EMsg[EMsg["ClientJoinChat"] = 801] = "ClientJoinChat";
    EMsg[EMsg["ClientChatMemberInfo"] = 802] = "ClientChatMemberInfo";
    EMsg[EMsg["ClientLogOnWithCredentials_Deprecated"] = 803] = "ClientLogOnWithCredentials_Deprecated";
    EMsg[EMsg["ClientPasswordChangeResponse"] = 805] = "ClientPasswordChangeResponse";
    EMsg[EMsg["ClientChatEnter"] = 807] = "ClientChatEnter";
    EMsg[EMsg["ClientFriendRemovedFromSource"] = 808] = "ClientFriendRemovedFromSource";
    EMsg[EMsg["ClientCreateChat"] = 809] = "ClientCreateChat";
    EMsg[EMsg["ClientCreateChatResponse"] = 810] = "ClientCreateChatResponse";
    EMsg[EMsg["ClientUpdateChatMetadata"] = 811] = "ClientUpdateChatMetadata";
    EMsg[EMsg["ClientP2PIntroducerMessage"] = 813] = "ClientP2PIntroducerMessage";
    EMsg[EMsg["ClientChatActionResult"] = 814] = "ClientChatActionResult";
    EMsg[EMsg["ClientRequestFriendData"] = 815] = "ClientRequestFriendData";
    EMsg[EMsg["ClientGetUserStats"] = 818] = "ClientGetUserStats";
    EMsg[EMsg["ClientGetUserStatsResponse"] = 819] = "ClientGetUserStatsResponse";
    EMsg[EMsg["ClientStoreUserStats"] = 820] = "ClientStoreUserStats";
    EMsg[EMsg["ClientStoreUserStatsResponse"] = 821] = "ClientStoreUserStatsResponse";
    EMsg[EMsg["ClientClanState"] = 822] = "ClientClanState";
    EMsg[EMsg["ClientServiceModule"] = 830] = "ClientServiceModule";
    EMsg[EMsg["ClientServiceCall"] = 831] = "ClientServiceCall";
    EMsg[EMsg["ClientServiceCallResponse"] = 832] = "ClientServiceCallResponse";
    EMsg[EMsg["ClientPackageInfoRequest"] = 833] = "ClientPackageInfoRequest";
    EMsg[EMsg["ClientPackageInfoResponse"] = 834] = "ClientPackageInfoResponse";
    EMsg[EMsg["ClientNatTraversalStatEvent"] = 839] = "ClientNatTraversalStatEvent";
    EMsg[EMsg["ClientAppInfoRequest"] = 840] = "ClientAppInfoRequest";
    EMsg[EMsg["ClientAppInfoResponse"] = 841] = "ClientAppInfoResponse";
    EMsg[EMsg["ClientSteamUsageEvent"] = 842] = "ClientSteamUsageEvent";
    EMsg[EMsg["ClientCheckPassword"] = 845] = "ClientCheckPassword";
    EMsg[EMsg["ClientResetPassword"] = 846] = "ClientResetPassword";
    EMsg[EMsg["ClientCheckPasswordResponse"] = 848] = "ClientCheckPasswordResponse";
    EMsg[EMsg["ClientResetPasswordResponse"] = 849] = "ClientResetPasswordResponse";
    EMsg[EMsg["ClientSessionToken"] = 850] = "ClientSessionToken";
    EMsg[EMsg["ClientDRMProblemReport"] = 851] = "ClientDRMProblemReport";
    EMsg[EMsg["ClientSetIgnoreFriend"] = 855] = "ClientSetIgnoreFriend";
    EMsg[EMsg["ClientSetIgnoreFriendResponse"] = 856] = "ClientSetIgnoreFriendResponse";
    EMsg[EMsg["ClientGetAppOwnershipTicket"] = 857] = "ClientGetAppOwnershipTicket";
    EMsg[EMsg["ClientGetAppOwnershipTicketResponse"] = 858] = "ClientGetAppOwnershipTicketResponse";
    EMsg[EMsg["ClientGetLobbyListResponse"] = 860] = "ClientGetLobbyListResponse";
    EMsg[EMsg["ClientGetLobbyMetadata"] = 861] = "ClientGetLobbyMetadata";
    EMsg[EMsg["ClientGetLobbyMetadataResponse"] = 862] = "ClientGetLobbyMetadataResponse";
    EMsg[EMsg["ClientVTTCert"] = 863] = "ClientVTTCert";
    EMsg[EMsg["ClientAppInfoUpdate"] = 866] = "ClientAppInfoUpdate";
    EMsg[EMsg["ClientAppInfoChanges"] = 867] = "ClientAppInfoChanges";
    EMsg[EMsg["ClientServerList"] = 880] = "ClientServerList";
    EMsg[EMsg["ClientEmailChangeResponse"] = 891] = "ClientEmailChangeResponse";
    EMsg[EMsg["ClientSecretQAChangeResponse"] = 892] = "ClientSecretQAChangeResponse";
    EMsg[EMsg["ClientDRMBlobRequest"] = 896] = "ClientDRMBlobRequest";
    EMsg[EMsg["ClientDRMBlobResponse"] = 897] = "ClientDRMBlobResponse";
    EMsg[EMsg["ClientLookupKey"] = 898] = "ClientLookupKey";
    EMsg[EMsg["ClientLookupKeyResponse"] = 899] = "ClientLookupKeyResponse";
    EMsg[EMsg["BaseGameServer"] = 900] = "BaseGameServer";
    EMsg[EMsg["GSDisconnectNotice"] = 901] = "GSDisconnectNotice";
    EMsg[EMsg["GSStatus"] = 903] = "GSStatus";
    EMsg[EMsg["GSUserPlaying"] = 905] = "GSUserPlaying";
    EMsg[EMsg["GSStatus2"] = 906] = "GSStatus2";
    EMsg[EMsg["GSStatusUpdate_Unused"] = 907] = "GSStatusUpdate_Unused";
    EMsg[EMsg["GSServerType"] = 908] = "GSServerType";
    EMsg[EMsg["GSPlayerList"] = 909] = "GSPlayerList";
    EMsg[EMsg["GSGetUserAchievementStatus"] = 910] = "GSGetUserAchievementStatus";
    EMsg[EMsg["GSGetUserAchievementStatusResponse"] = 911] = "GSGetUserAchievementStatusResponse";
    EMsg[EMsg["GSGetPlayStats"] = 918] = "GSGetPlayStats";
    EMsg[EMsg["GSGetPlayStatsResponse"] = 919] = "GSGetPlayStatsResponse";
    EMsg[EMsg["GSGetUserGroupStatus"] = 920] = "GSGetUserGroupStatus";
    EMsg[EMsg["AMGetUserGroupStatus"] = 921] = "AMGetUserGroupStatus";
    EMsg[EMsg["AMGetUserGroupStatusResponse"] = 922] = "AMGetUserGroupStatusResponse";
    EMsg[EMsg["GSGetUserGroupStatusResponse"] = 923] = "GSGetUserGroupStatusResponse";
    EMsg[EMsg["GSGetReputation"] = 936] = "GSGetReputation";
    EMsg[EMsg["GSGetReputationResponse"] = 937] = "GSGetReputationResponse";
    EMsg[EMsg["GSAssociateWithClan"] = 938] = "GSAssociateWithClan";
    EMsg[EMsg["GSAssociateWithClanResponse"] = 939] = "GSAssociateWithClanResponse";
    EMsg[EMsg["GSComputeNewPlayerCompatibility"] = 940] = "GSComputeNewPlayerCompatibility";
    EMsg[EMsg["GSComputeNewPlayerCompatibilityResponse"] = 941] = "GSComputeNewPlayerCompatibilityResponse";
    EMsg[EMsg["BaseAdmin"] = 1000] = "BaseAdmin";
    EMsg[EMsg["AdminCmd"] = 1000] = "AdminCmd";
    EMsg[EMsg["AdminCmdResponse"] = 1004] = "AdminCmdResponse";
    EMsg[EMsg["AdminLogListenRequest"] = 1005] = "AdminLogListenRequest";
    EMsg[EMsg["AdminLogEvent"] = 1006] = "AdminLogEvent";
    EMsg[EMsg["LogSearchRequest"] = 1007] = "LogSearchRequest";
    EMsg[EMsg["LogSearchResponse"] = 1008] = "LogSearchResponse";
    EMsg[EMsg["LogSearchCancel"] = 1009] = "LogSearchCancel";
    EMsg[EMsg["UniverseData"] = 1010] = "UniverseData";
    EMsg[EMsg["RequestStatHistory"] = 1014] = "RequestStatHistory";
    EMsg[EMsg["StatHistory"] = 1015] = "StatHistory";
    EMsg[EMsg["AdminPwLogon"] = 1017] = "AdminPwLogon";
    EMsg[EMsg["AdminPwLogonResponse"] = 1018] = "AdminPwLogonResponse";
    EMsg[EMsg["AdminSpew"] = 1019] = "AdminSpew";
    EMsg[EMsg["AdminConsoleTitle"] = 1020] = "AdminConsoleTitle";
    EMsg[EMsg["AdminGCSpew"] = 1023] = "AdminGCSpew";
    EMsg[EMsg["AdminGCCommand"] = 1024] = "AdminGCCommand";
    EMsg[EMsg["AdminGCGetCommandList"] = 1025] = "AdminGCGetCommandList";
    EMsg[EMsg["AdminGCGetCommandListResponse"] = 1026] = "AdminGCGetCommandListResponse";
    EMsg[EMsg["FBSConnectionData"] = 1027] = "FBSConnectionData";
    EMsg[EMsg["AdminMsgSpew"] = 1028] = "AdminMsgSpew";
    EMsg[EMsg["BaseFBS"] = 1100] = "BaseFBS";
    EMsg[EMsg["FBSReqVersion"] = 1100] = "FBSReqVersion";
    EMsg[EMsg["FBSVersionInfo"] = 1101] = "FBSVersionInfo";
    EMsg[EMsg["FBSForceRefresh"] = 1102] = "FBSForceRefresh";
    EMsg[EMsg["FBSForceBounce"] = 1103] = "FBSForceBounce";
    EMsg[EMsg["FBSDeployPackage"] = 1104] = "FBSDeployPackage";
    EMsg[EMsg["FBSDeployResponse"] = 1105] = "FBSDeployResponse";
    EMsg[EMsg["FBSUpdateBootstrapper"] = 1106] = "FBSUpdateBootstrapper";
    EMsg[EMsg["FBSSetState"] = 1107] = "FBSSetState";
    EMsg[EMsg["FBSApplyOSUpdates"] = 1108] = "FBSApplyOSUpdates";
    EMsg[EMsg["FBSRunCMDScript"] = 1109] = "FBSRunCMDScript";
    EMsg[EMsg["FBSRebootBox"] = 1110] = "FBSRebootBox";
    EMsg[EMsg["FBSSetBigBrotherMode"] = 1111] = "FBSSetBigBrotherMode";
    EMsg[EMsg["FBSMinidumpServer"] = 1112] = "FBSMinidumpServer";
    EMsg[EMsg["FBSSetShellCount_obsolete"] = 1113] = "FBSSetShellCount_obsolete";
    EMsg[EMsg["FBSDeployHotFixPackage"] = 1114] = "FBSDeployHotFixPackage";
    EMsg[EMsg["FBSDeployHotFixResponse"] = 1115] = "FBSDeployHotFixResponse";
    EMsg[EMsg["FBSDownloadHotFix"] = 1116] = "FBSDownloadHotFix";
    EMsg[EMsg["FBSDownloadHotFixResponse"] = 1117] = "FBSDownloadHotFixResponse";
    EMsg[EMsg["FBSUpdateTargetConfigFile"] = 1118] = "FBSUpdateTargetConfigFile";
    EMsg[EMsg["FBSApplyAccountCred"] = 1119] = "FBSApplyAccountCred";
    EMsg[EMsg["FBSApplyAccountCredResponse"] = 1120] = "FBSApplyAccountCredResponse";
    EMsg[EMsg["FBSSetShellCount"] = 1121] = "FBSSetShellCount";
    EMsg[EMsg["FBSTerminateShell"] = 1122] = "FBSTerminateShell";
    EMsg[EMsg["FBSQueryGMForRequest"] = 1123] = "FBSQueryGMForRequest";
    EMsg[EMsg["FBSQueryGMResponse"] = 1124] = "FBSQueryGMResponse";
    EMsg[EMsg["FBSTerminateZombies"] = 1125] = "FBSTerminateZombies";
    EMsg[EMsg["FBSInfoFromBootstrapper"] = 1126] = "FBSInfoFromBootstrapper";
    EMsg[EMsg["FBSRebootBoxResponse"] = 1127] = "FBSRebootBoxResponse";
    EMsg[EMsg["FBSBootstrapperPackageRequest"] = 1128] = "FBSBootstrapperPackageRequest";
    EMsg[EMsg["FBSBootstrapperPackageResponse"] = 1129] = "FBSBootstrapperPackageResponse";
    EMsg[EMsg["FBSBootstrapperGetPackageChunk"] = 1130] = "FBSBootstrapperGetPackageChunk";
    EMsg[EMsg["FBSBootstrapperGetPackageChunkResponse"] = 1131] = "FBSBootstrapperGetPackageChunkResponse";
    EMsg[EMsg["FBSBootstrapperPackageTransferProgress"] = 1132] = "FBSBootstrapperPackageTransferProgress";
    EMsg[EMsg["FBSRestartBootstrapper"] = 1133] = "FBSRestartBootstrapper";
    EMsg[EMsg["FBSPauseFrozenDumps"] = 1134] = "FBSPauseFrozenDumps";
    EMsg[EMsg["BaseFileXfer"] = 1200] = "BaseFileXfer";
    EMsg[EMsg["FileXferRequest"] = 1200] = "FileXferRequest";
    EMsg[EMsg["FileXferResponse"] = 1201] = "FileXferResponse";
    EMsg[EMsg["FileXferData"] = 1202] = "FileXferData";
    EMsg[EMsg["FileXferEnd"] = 1203] = "FileXferEnd";
    EMsg[EMsg["FileXferDataAck"] = 1204] = "FileXferDataAck";
    EMsg[EMsg["BaseChannelAuth"] = 1300] = "BaseChannelAuth";
    EMsg[EMsg["ChannelAuthChallenge"] = 1300] = "ChannelAuthChallenge";
    EMsg[EMsg["ChannelAuthResponse"] = 1301] = "ChannelAuthResponse";
    EMsg[EMsg["ChannelAuthResult"] = 1302] = "ChannelAuthResult";
    EMsg[EMsg["ChannelEncryptRequest"] = 1303] = "ChannelEncryptRequest";
    EMsg[EMsg["ChannelEncryptResponse"] = 1304] = "ChannelEncryptResponse";
    EMsg[EMsg["ChannelEncryptResult"] = 1305] = "ChannelEncryptResult";
    EMsg[EMsg["BaseBS"] = 1400] = "BaseBS";
    EMsg[EMsg["BSPurchaseStart"] = 1401] = "BSPurchaseStart";
    EMsg[EMsg["BSPurchaseResponse"] = 1402] = "BSPurchaseResponse";
    EMsg[EMsg["BSAuthenticateCCTrans"] = 1403] = "BSAuthenticateCCTrans";
    EMsg[EMsg["BSSettleNOVA"] = 1404] = "BSSettleNOVA";
    EMsg[EMsg["BSAuthenticateCCTransResponse"] = 1404] = "BSAuthenticateCCTransResponse";
    EMsg[EMsg["BSSettleComplete"] = 1406] = "BSSettleComplete";
    EMsg[EMsg["BSBannedRequest"] = 1407] = "BSBannedRequest";
    EMsg[EMsg["BSInitPayPalTxn"] = 1408] = "BSInitPayPalTxn";
    EMsg[EMsg["BSInitPayPalTxnResponse"] = 1409] = "BSInitPayPalTxnResponse";
    EMsg[EMsg["BSGetPayPalUserInfo"] = 1410] = "BSGetPayPalUserInfo";
    EMsg[EMsg["BSGetPayPalUserInfoResponse"] = 1411] = "BSGetPayPalUserInfoResponse";
    EMsg[EMsg["BSRefundTxn"] = 1413] = "BSRefundTxn";
    EMsg[EMsg["BSRefundTxnResponse"] = 1414] = "BSRefundTxnResponse";
    EMsg[EMsg["BSGetEvents"] = 1415] = "BSGetEvents";
    EMsg[EMsg["BSChaseRFRRequest"] = 1416] = "BSChaseRFRRequest";
    EMsg[EMsg["BSPaymentInstrBan"] = 1417] = "BSPaymentInstrBan";
    EMsg[EMsg["BSPaymentInstrBanResponse"] = 1418] = "BSPaymentInstrBanResponse";
    EMsg[EMsg["BSProcessGCReports"] = 1419] = "BSProcessGCReports";
    EMsg[EMsg["BSProcessPPReports"] = 1420] = "BSProcessPPReports";
    EMsg[EMsg["BSInitGCBankXferTxn"] = 1421] = "BSInitGCBankXferTxn";
    EMsg[EMsg["BSInitGCBankXferTxnResponse"] = 1422] = "BSInitGCBankXferTxnResponse";
    EMsg[EMsg["BSQueryGCBankXferTxn"] = 1423] = "BSQueryGCBankXferTxn";
    EMsg[EMsg["BSQueryGCBankXferTxnResponse"] = 1424] = "BSQueryGCBankXferTxnResponse";
    EMsg[EMsg["BSCommitGCTxn"] = 1425] = "BSCommitGCTxn";
    EMsg[EMsg["BSQueryTransactionStatus"] = 1426] = "BSQueryTransactionStatus";
    EMsg[EMsg["BSQueryTransactionStatusResponse"] = 1427] = "BSQueryTransactionStatusResponse";
    EMsg[EMsg["BSQueryCBOrderStatus"] = 1428] = "BSQueryCBOrderStatus";
    EMsg[EMsg["BSQueryCBOrderStatusResponse"] = 1429] = "BSQueryCBOrderStatusResponse";
    EMsg[EMsg["BSRunRedFlagReport"] = 1430] = "BSRunRedFlagReport";
    EMsg[EMsg["BSQueryPaymentInstUsage"] = 1431] = "BSQueryPaymentInstUsage";
    EMsg[EMsg["BSQueryPaymentInstResponse"] = 1432] = "BSQueryPaymentInstResponse";
    EMsg[EMsg["BSQueryTxnExtendedInfo"] = 1433] = "BSQueryTxnExtendedInfo";
    EMsg[EMsg["BSQueryTxnExtendedInfoResponse"] = 1434] = "BSQueryTxnExtendedInfoResponse";
    EMsg[EMsg["BSUpdateConversionRates"] = 1435] = "BSUpdateConversionRates";
    EMsg[EMsg["BSProcessUSBankReports"] = 1436] = "BSProcessUSBankReports";
    EMsg[EMsg["BSPurchaseRunFraudChecks"] = 1437] = "BSPurchaseRunFraudChecks";
    EMsg[EMsg["BSPurchaseRunFraudChecksResponse"] = 1438] = "BSPurchaseRunFraudChecksResponse";
    EMsg[EMsg["BSStartShippingJobs"] = 1439] = "BSStartShippingJobs";
    EMsg[EMsg["BSQueryBankInformation"] = 1440] = "BSQueryBankInformation";
    EMsg[EMsg["BSQueryBankInformationResponse"] = 1441] = "BSQueryBankInformationResponse";
    EMsg[EMsg["BSValidateXsollaSignature"] = 1445] = "BSValidateXsollaSignature";
    EMsg[EMsg["BSValidateXsollaSignatureResponse"] = 1446] = "BSValidateXsollaSignatureResponse";
    EMsg[EMsg["BSQiwiWalletInvoice"] = 1448] = "BSQiwiWalletInvoice";
    EMsg[EMsg["BSQiwiWalletInvoiceResponse"] = 1449] = "BSQiwiWalletInvoiceResponse";
    EMsg[EMsg["BSUpdateInventoryFromProPack"] = 1450] = "BSUpdateInventoryFromProPack";
    EMsg[EMsg["BSUpdateInventoryFromProPackResponse"] = 1451] = "BSUpdateInventoryFromProPackResponse";
    EMsg[EMsg["BSSendShippingRequest"] = 1452] = "BSSendShippingRequest";
    EMsg[EMsg["BSSendShippingRequestResponse"] = 1453] = "BSSendShippingRequestResponse";
    EMsg[EMsg["BSGetProPackOrderStatus"] = 1454] = "BSGetProPackOrderStatus";
    EMsg[EMsg["BSGetProPackOrderStatusResponse"] = 1455] = "BSGetProPackOrderStatusResponse";
    EMsg[EMsg["BSCheckJobRunning"] = 1456] = "BSCheckJobRunning";
    EMsg[EMsg["BSCheckJobRunningResponse"] = 1457] = "BSCheckJobRunningResponse";
    EMsg[EMsg["BSResetPackagePurchaseRateLimit"] = 1458] = "BSResetPackagePurchaseRateLimit";
    EMsg[EMsg["BSResetPackagePurchaseRateLimitResponse"] = 1459] = "BSResetPackagePurchaseRateLimitResponse";
    EMsg[EMsg["BSUpdatePaymentData"] = 1460] = "BSUpdatePaymentData";
    EMsg[EMsg["BSUpdatePaymentDataResponse"] = 1461] = "BSUpdatePaymentDataResponse";
    EMsg[EMsg["BSGetBillingAddress"] = 1462] = "BSGetBillingAddress";
    EMsg[EMsg["BSGetBillingAddressResponse"] = 1463] = "BSGetBillingAddressResponse";
    EMsg[EMsg["BSGetCreditCardInfo"] = 1464] = "BSGetCreditCardInfo";
    EMsg[EMsg["BSGetCreditCardInfoResponse"] = 1465] = "BSGetCreditCardInfoResponse";
    EMsg[EMsg["BSRemoveExpiredPaymentData"] = 1468] = "BSRemoveExpiredPaymentData";
    EMsg[EMsg["BSRemoveExpiredPaymentDataResponse"] = 1469] = "BSRemoveExpiredPaymentDataResponse";
    EMsg[EMsg["BSConvertToCurrentKeys"] = 1470] = "BSConvertToCurrentKeys";
    EMsg[EMsg["BSConvertToCurrentKeysResponse"] = 1471] = "BSConvertToCurrentKeysResponse";
    EMsg[EMsg["BSInitPurchase"] = 1472] = "BSInitPurchase";
    EMsg[EMsg["BSInitPurchaseResponse"] = 1473] = "BSInitPurchaseResponse";
    EMsg[EMsg["BSCompletePurchase"] = 1474] = "BSCompletePurchase";
    EMsg[EMsg["BSCompletePurchaseResponse"] = 1475] = "BSCompletePurchaseResponse";
    EMsg[EMsg["BSPruneCardUsageStats"] = 1476] = "BSPruneCardUsageStats";
    EMsg[EMsg["BSPruneCardUsageStatsResponse"] = 1477] = "BSPruneCardUsageStatsResponse";
    EMsg[EMsg["BSStoreBankInformation"] = 1478] = "BSStoreBankInformation";
    EMsg[EMsg["BSStoreBankInformationResponse"] = 1479] = "BSStoreBankInformationResponse";
    EMsg[EMsg["BSVerifyPOSAKey"] = 1480] = "BSVerifyPOSAKey";
    EMsg[EMsg["BSVerifyPOSAKeyResponse"] = 1481] = "BSVerifyPOSAKeyResponse";
    EMsg[EMsg["BSReverseRedeemPOSAKey"] = 1482] = "BSReverseRedeemPOSAKey";
    EMsg[EMsg["BSReverseRedeemPOSAKeyResponse"] = 1483] = "BSReverseRedeemPOSAKeyResponse";
    EMsg[EMsg["BSQueryFindCreditCard"] = 1484] = "BSQueryFindCreditCard";
    EMsg[EMsg["BSQueryFindCreditCardResponse"] = 1485] = "BSQueryFindCreditCardResponse";
    EMsg[EMsg["BSStatusInquiryPOSAKey"] = 1486] = "BSStatusInquiryPOSAKey";
    EMsg[EMsg["BSStatusInquiryPOSAKeyResponse"] = 1487] = "BSStatusInquiryPOSAKeyResponse";
    EMsg[EMsg["BSValidateMoPaySignature"] = 1488] = "BSValidateMoPaySignature";
    EMsg[EMsg["BSValidateMoPaySignatureResponse"] = 1489] = "BSValidateMoPaySignatureResponse";
    EMsg[EMsg["BSMoPayConfirmProductDelivery"] = 1490] = "BSMoPayConfirmProductDelivery";
    EMsg[EMsg["BSMoPayConfirmProductDeliveryResponse"] = 1491] = "BSMoPayConfirmProductDeliveryResponse";
    EMsg[EMsg["BSGenerateMoPayMD5"] = 1492] = "BSGenerateMoPayMD5";
    EMsg[EMsg["BSGenerateMoPayMD5Response"] = 1493] = "BSGenerateMoPayMD5Response";
    EMsg[EMsg["BSBoaCompraConfirmProductDelivery"] = 1494] = "BSBoaCompraConfirmProductDelivery";
    EMsg[EMsg["BSBoaCompraConfirmProductDeliveryResponse"] = 1495] = "BSBoaCompraConfirmProductDeliveryResponse";
    EMsg[EMsg["BSGenerateBoaCompraMD5"] = 1496] = "BSGenerateBoaCompraMD5";
    EMsg[EMsg["BSGenerateBoaCompraMD5Response"] = 1497] = "BSGenerateBoaCompraMD5Response";
    EMsg[EMsg["BSCommitWPTxn"] = 1498] = "BSCommitWPTxn";
    EMsg[EMsg["BSCommitAdyenTxn"] = 1499] = "BSCommitAdyenTxn";
    EMsg[EMsg["BaseATS"] = 1500] = "BaseATS";
    EMsg[EMsg["ATSStartStressTest"] = 1501] = "ATSStartStressTest";
    EMsg[EMsg["ATSStopStressTest"] = 1502] = "ATSStopStressTest";
    EMsg[EMsg["ATSRunFailServerTest"] = 1503] = "ATSRunFailServerTest";
    EMsg[EMsg["ATSUFSPerfTestTask"] = 1504] = "ATSUFSPerfTestTask";
    EMsg[EMsg["ATSUFSPerfTestResponse"] = 1505] = "ATSUFSPerfTestResponse";
    EMsg[EMsg["ATSCycleTCM"] = 1506] = "ATSCycleTCM";
    EMsg[EMsg["ATSInitDRMSStressTest"] = 1507] = "ATSInitDRMSStressTest";
    EMsg[EMsg["ATSCallTest"] = 1508] = "ATSCallTest";
    EMsg[EMsg["ATSCallTestReply"] = 1509] = "ATSCallTestReply";
    EMsg[EMsg["ATSStartExternalStress"] = 1510] = "ATSStartExternalStress";
    EMsg[EMsg["ATSExternalStressJobStart"] = 1511] = "ATSExternalStressJobStart";
    EMsg[EMsg["ATSExternalStressJobQueued"] = 1512] = "ATSExternalStressJobQueued";
    EMsg[EMsg["ATSExternalStressJobRunning"] = 1513] = "ATSExternalStressJobRunning";
    EMsg[EMsg["ATSExternalStressJobStopped"] = 1514] = "ATSExternalStressJobStopped";
    EMsg[EMsg["ATSExternalStressJobStopAll"] = 1515] = "ATSExternalStressJobStopAll";
    EMsg[EMsg["ATSExternalStressActionResult"] = 1516] = "ATSExternalStressActionResult";
    EMsg[EMsg["ATSStarted"] = 1517] = "ATSStarted";
    EMsg[EMsg["ATSCSPerfTestTask"] = 1518] = "ATSCSPerfTestTask";
    EMsg[EMsg["ATSCSPerfTestResponse"] = 1519] = "ATSCSPerfTestResponse";
    EMsg[EMsg["BaseDP"] = 1600] = "BaseDP";
    EMsg[EMsg["DPSetPublishingState"] = 1601] = "DPSetPublishingState";
    EMsg[EMsg["DPGamePlayedStats"] = 1602] = "DPGamePlayedStats";
    EMsg[EMsg["DPUniquePlayersStat"] = 1603] = "DPUniquePlayersStat";
    EMsg[EMsg["DPStreamingUniquePlayersStat"] = 1604] = "DPStreamingUniquePlayersStat";
    EMsg[EMsg["DPVacInfractionStats"] = 1605] = "DPVacInfractionStats";
    EMsg[EMsg["DPVacBanStats"] = 1606] = "DPVacBanStats";
    EMsg[EMsg["DPBlockingStats"] = 1607] = "DPBlockingStats";
    EMsg[EMsg["DPNatTraversalStats"] = 1608] = "DPNatTraversalStats";
    EMsg[EMsg["DPSteamUsageEvent"] = 1609] = "DPSteamUsageEvent";
    EMsg[EMsg["DPVacCertBanStats"] = 1610] = "DPVacCertBanStats";
    EMsg[EMsg["DPVacCafeBanStats"] = 1611] = "DPVacCafeBanStats";
    EMsg[EMsg["DPCloudStats"] = 1612] = "DPCloudStats";
    EMsg[EMsg["DPAchievementStats"] = 1613] = "DPAchievementStats";
    EMsg[EMsg["DPAccountCreationStats"] = 1614] = "DPAccountCreationStats";
    EMsg[EMsg["DPGetPlayerCount"] = 1615] = "DPGetPlayerCount";
    EMsg[EMsg["DPGetPlayerCountResponse"] = 1616] = "DPGetPlayerCountResponse";
    EMsg[EMsg["DPGameServersPlayersStats"] = 1617] = "DPGameServersPlayersStats";
    EMsg[EMsg["DPDownloadRateStatistics"] = 1618] = "DPDownloadRateStatistics";
    EMsg[EMsg["DPFacebookStatistics"] = 1619] = "DPFacebookStatistics";
    EMsg[EMsg["ClientDPCheckSpecialSurvey"] = 1620] = "ClientDPCheckSpecialSurvey";
    EMsg[EMsg["ClientDPCheckSpecialSurveyResponse"] = 1621] = "ClientDPCheckSpecialSurveyResponse";
    EMsg[EMsg["ClientDPSendSpecialSurveyResponse"] = 1622] = "ClientDPSendSpecialSurveyResponse";
    EMsg[EMsg["ClientDPSendSpecialSurveyResponseReply"] = 1623] = "ClientDPSendSpecialSurveyResponseReply";
    EMsg[EMsg["DPStoreSaleStatistics"] = 1624] = "DPStoreSaleStatistics";
    EMsg[EMsg["ClientDPUpdateAppJobReport"] = 1625] = "ClientDPUpdateAppJobReport";
    EMsg[EMsg["DPUpdateContentEvent"] = 1626] = "DPUpdateContentEvent";
    EMsg[EMsg["ClientDPSteam2AppStarted"] = 1627] = "ClientDPSteam2AppStarted";
    EMsg[EMsg["ClientDPUnsignedInstallScript"] = 1627] = "ClientDPUnsignedInstallScript";
    EMsg[EMsg["DPPartnerMicroTxns"] = 1628] = "DPPartnerMicroTxns";
    EMsg[EMsg["DPPartnerMicroTxnsResponse"] = 1629] = "DPPartnerMicroTxnsResponse";
    EMsg[EMsg["ClientDPContentStatsReport"] = 1630] = "ClientDPContentStatsReport";
    EMsg[EMsg["DPVRUniquePlayersStat"] = 1631] = "DPVRUniquePlayersStat";
    EMsg[EMsg["BaseCM"] = 1700] = "BaseCM";
    EMsg[EMsg["CMSetAllowState"] = 1701] = "CMSetAllowState";
    EMsg[EMsg["CMSpewAllowState"] = 1702] = "CMSpewAllowState";
    EMsg[EMsg["CMAppInfoResponseDeprecated"] = 1703] = "CMAppInfoResponseDeprecated";
    EMsg[EMsg["CMSessionRejected"] = 1703] = "CMSessionRejected";
    EMsg[EMsg["CMSetSecrets"] = 1704] = "CMSetSecrets";
    EMsg[EMsg["CMGetSecrets"] = 1705] = "CMGetSecrets";
    EMsg[EMsg["BaseDSS"] = 1800] = "BaseDSS";
    EMsg[EMsg["DSSNewFile"] = 1801] = "DSSNewFile";
    EMsg[EMsg["DSSCurrentFileList"] = 1802] = "DSSCurrentFileList";
    EMsg[EMsg["DSSSynchList"] = 1803] = "DSSSynchList";
    EMsg[EMsg["DSSSynchListResponse"] = 1804] = "DSSSynchListResponse";
    EMsg[EMsg["DSSSynchSubscribe"] = 1805] = "DSSSynchSubscribe";
    EMsg[EMsg["DSSSynchUnsubscribe"] = 1806] = "DSSSynchUnsubscribe";
    EMsg[EMsg["BaseEPM"] = 1900] = "BaseEPM";
    EMsg[EMsg["EPMStartProcess"] = 1901] = "EPMStartProcess";
    EMsg[EMsg["EPMStopProcess"] = 1902] = "EPMStopProcess";
    EMsg[EMsg["EPMRestartProcess"] = 1903] = "EPMRestartProcess";
    EMsg[EMsg["GCSendClient"] = 2200] = "GCSendClient";
    EMsg[EMsg["BaseGC"] = 2200] = "BaseGC";
    EMsg[EMsg["AMRelayToGC"] = 2201] = "AMRelayToGC";
    EMsg[EMsg["GCUpdatePlayedState"] = 2202] = "GCUpdatePlayedState";
    EMsg[EMsg["GCCmdRevive"] = 2203] = "GCCmdRevive";
    EMsg[EMsg["GCCmdBounce"] = 2204] = "GCCmdBounce";
    EMsg[EMsg["GCCmdForceBounce"] = 2205] = "GCCmdForceBounce";
    EMsg[EMsg["GCCmdDown"] = 2206] = "GCCmdDown";
    EMsg[EMsg["GCCmdDeploy"] = 2207] = "GCCmdDeploy";
    EMsg[EMsg["GCCmdDeployResponse"] = 2208] = "GCCmdDeployResponse";
    EMsg[EMsg["GCCmdSwitch"] = 2209] = "GCCmdSwitch";
    EMsg[EMsg["AMRefreshSessions"] = 2210] = "AMRefreshSessions";
    EMsg[EMsg["GCUpdateGSState"] = 2211] = "GCUpdateGSState";
    EMsg[EMsg["GCAchievementAwarded"] = 2212] = "GCAchievementAwarded";
    EMsg[EMsg["GCSystemMessage"] = 2213] = "GCSystemMessage";
    EMsg[EMsg["GCValidateSession"] = 2214] = "GCValidateSession";
    EMsg[EMsg["GCValidateSessionResponse"] = 2215] = "GCValidateSessionResponse";
    EMsg[EMsg["GCCmdStatus"] = 2216] = "GCCmdStatus";
    EMsg[EMsg["GCRegisterWebInterfaces"] = 2217] = "GCRegisterWebInterfaces";
    EMsg[EMsg["GCRegisterWebInterfaces_Deprecated"] = 2217] = "GCRegisterWebInterfaces_Deprecated";
    EMsg[EMsg["GCGetAccountDetails"] = 2218] = "GCGetAccountDetails";
    EMsg[EMsg["GCGetAccountDetails_DEPRECATED"] = 2218] = "GCGetAccountDetails_DEPRECATED";
    EMsg[EMsg["GCInterAppMessage"] = 2219] = "GCInterAppMessage";
    EMsg[EMsg["GCGetEmailTemplate"] = 2220] = "GCGetEmailTemplate";
    EMsg[EMsg["GCGetEmailTemplateResponse"] = 2221] = "GCGetEmailTemplateResponse";
    EMsg[EMsg["ISRelayToGCH"] = 2222] = "ISRelayToGCH";
    EMsg[EMsg["GCHRelay"] = 2222] = "GCHRelay";
    EMsg[EMsg["GCHRelayClientToIS"] = 2223] = "GCHRelayClientToIS";
    EMsg[EMsg["GCHRelayToClient"] = 2223] = "GCHRelayToClient";
    EMsg[EMsg["GCHUpdateSession"] = 2224] = "GCHUpdateSession";
    EMsg[EMsg["GCHRequestUpdateSession"] = 2225] = "GCHRequestUpdateSession";
    EMsg[EMsg["GCHRequestStatus"] = 2226] = "GCHRequestStatus";
    EMsg[EMsg["GCHRequestStatusResponse"] = 2227] = "GCHRequestStatusResponse";
    EMsg[EMsg["GCHAccountVacStatusChange"] = 2228] = "GCHAccountVacStatusChange";
    EMsg[EMsg["GCHSpawnGC"] = 2229] = "GCHSpawnGC";
    EMsg[EMsg["GCHSpawnGCResponse"] = 2230] = "GCHSpawnGCResponse";
    EMsg[EMsg["GCHKillGC"] = 2231] = "GCHKillGC";
    EMsg[EMsg["GCHKillGCResponse"] = 2232] = "GCHKillGCResponse";
    EMsg[EMsg["GCHAccountTradeBanStatusChange"] = 2233] = "GCHAccountTradeBanStatusChange";
    EMsg[EMsg["GCHAccountLockStatusChange"] = 2234] = "GCHAccountLockStatusChange";
    EMsg[EMsg["GCHVacVerificationChange"] = 2235] = "GCHVacVerificationChange";
    EMsg[EMsg["GCHAccountPhoneNumberChange"] = 2236] = "GCHAccountPhoneNumberChange";
    EMsg[EMsg["GCHAccountTwoFactorChange"] = 2237] = "GCHAccountTwoFactorChange";
    EMsg[EMsg["GCHInviteUserToLobby"] = 2238] = "GCHInviteUserToLobby";
    EMsg[EMsg["GCHUpdateMultipleSessions"] = 2239] = "GCHUpdateMultipleSessions";
    EMsg[EMsg["GCHMarkAppSessionsAuthoritative"] = 2240] = "GCHMarkAppSessionsAuthoritative";
    EMsg[EMsg["GCHRecurringSubscriptionStatusChange"] = 2241] = "GCHRecurringSubscriptionStatusChange";
    EMsg[EMsg["GCHAppCheersReceived"] = 2242] = "GCHAppCheersReceived";
    EMsg[EMsg["GCHAppCheersGetAllowedTypes"] = 2243] = "GCHAppCheersGetAllowedTypes";
    EMsg[EMsg["GCHAppCheersGetAllowedTypesResponse"] = 2244] = "GCHAppCheersGetAllowedTypesResponse";
    EMsg[EMsg["GCHRoutingRulesFromGCHtoGM"] = 2245] = "GCHRoutingRulesFromGCHtoGM";
    EMsg[EMsg["GCHRoutingRulesToGCHfromGM"] = 2246] = "GCHRoutingRulesToGCHfromGM";
    EMsg[EMsg["UpdateCMMessageRateRules"] = 2247] = "UpdateCMMessageRateRules";
    EMsg[EMsg["BaseP2P"] = 2500] = "BaseP2P";
    EMsg[EMsg["P2PIntroducerMessage"] = 2502] = "P2PIntroducerMessage";
    EMsg[EMsg["BaseSM"] = 2900] = "BaseSM";
    EMsg[EMsg["SMExpensiveReport"] = 2902] = "SMExpensiveReport";
    EMsg[EMsg["SMHourlyReport"] = 2903] = "SMHourlyReport";
    EMsg[EMsg["SMFishingReport"] = 2904] = "SMFishingReport";
    EMsg[EMsg["SMPartitionRenames"] = 2905] = "SMPartitionRenames";
    EMsg[EMsg["SMMonitorSpace"] = 2906] = "SMMonitorSpace";
    EMsg[EMsg["SMGetSchemaConversionResults"] = 2907] = "SMGetSchemaConversionResults";
    EMsg[EMsg["SMTestNextBuildSchemaConversion"] = 2907] = "SMTestNextBuildSchemaConversion";
    EMsg[EMsg["SMGetSchemaConversionResultsResponse"] = 2908] = "SMGetSchemaConversionResultsResponse";
    EMsg[EMsg["SMTestNextBuildSchemaConversionResponse"] = 2908] = "SMTestNextBuildSchemaConversionResponse";
    EMsg[EMsg["BaseTest"] = 3000] = "BaseTest";
    EMsg[EMsg["FailServer"] = 3000] = "FailServer";
    EMsg[EMsg["JobHeartbeatTest"] = 3001] = "JobHeartbeatTest";
    EMsg[EMsg["JobHeartbeatTestResponse"] = 3002] = "JobHeartbeatTestResponse";
    EMsg[EMsg["BaseFTSRange"] = 3100] = "BaseFTSRange";
    EMsg[EMsg["FTSGetBrowseCounts"] = 3101] = "FTSGetBrowseCounts";
    EMsg[EMsg["FTSGetBrowseCountsResponse"] = 3102] = "FTSGetBrowseCountsResponse";
    EMsg[EMsg["FTSBrowseClans"] = 3103] = "FTSBrowseClans";
    EMsg[EMsg["FTSBrowseClansResponse"] = 3104] = "FTSBrowseClansResponse";
    EMsg[EMsg["FTSSearchClansByLocation"] = 3105] = "FTSSearchClansByLocation";
    EMsg[EMsg["FTSSearchClansByLocationResponse"] = 3106] = "FTSSearchClansByLocationResponse";
    EMsg[EMsg["FTSSearchPlayersByLocation"] = 3107] = "FTSSearchPlayersByLocation";
    EMsg[EMsg["FTSSearchPlayersByLocationResponse"] = 3108] = "FTSSearchPlayersByLocationResponse";
    EMsg[EMsg["FTSClanDeleted"] = 3109] = "FTSClanDeleted";
    EMsg[EMsg["FTSSearch"] = 3110] = "FTSSearch";
    EMsg[EMsg["FTSSearchResponse"] = 3111] = "FTSSearchResponse";
    EMsg[EMsg["FTSSearchStatus"] = 3112] = "FTSSearchStatus";
    EMsg[EMsg["FTSSearchStatusResponse"] = 3113] = "FTSSearchStatusResponse";
    EMsg[EMsg["FTSGetGSPlayStats"] = 3114] = "FTSGetGSPlayStats";
    EMsg[EMsg["FTSGetGSPlayStatsResponse"] = 3115] = "FTSGetGSPlayStatsResponse";
    EMsg[EMsg["FTSGetGSPlayStatsForServer"] = 3116] = "FTSGetGSPlayStatsForServer";
    EMsg[EMsg["FTSGetGSPlayStatsForServerResponse"] = 3117] = "FTSGetGSPlayStatsForServerResponse";
    EMsg[EMsg["FTSReportIPUpdates"] = 3118] = "FTSReportIPUpdates";
    EMsg[EMsg["BaseCCSRange"] = 3150] = "BaseCCSRange";
    EMsg[EMsg["CCSGetComments"] = 3151] = "CCSGetComments";
    EMsg[EMsg["CCSGetCommentsResponse"] = 3152] = "CCSGetCommentsResponse";
    EMsg[EMsg["CCSAddComment"] = 3153] = "CCSAddComment";
    EMsg[EMsg["CCSAddCommentResponse"] = 3154] = "CCSAddCommentResponse";
    EMsg[EMsg["CCSDeleteComment"] = 3155] = "CCSDeleteComment";
    EMsg[EMsg["CCSDeleteCommentResponse"] = 3156] = "CCSDeleteCommentResponse";
    EMsg[EMsg["CCSPreloadComments"] = 3157] = "CCSPreloadComments";
    EMsg[EMsg["CCSNotifyCommentCount"] = 3158] = "CCSNotifyCommentCount";
    EMsg[EMsg["CCSGetCommentsForNews"] = 3159] = "CCSGetCommentsForNews";
    EMsg[EMsg["CCSGetCommentsForNewsResponse"] = 3160] = "CCSGetCommentsForNewsResponse";
    EMsg[EMsg["CCSDeleteAllCommentsByAuthor"] = 3161] = "CCSDeleteAllCommentsByAuthor";
    EMsg[EMsg["CCSDeleteAllCommentsByAuthorResponse"] = 3162] = "CCSDeleteAllCommentsByAuthorResponse";
    EMsg[EMsg["BaseLBSRange"] = 3200] = "BaseLBSRange";
    EMsg[EMsg["LBSSetScore"] = 3201] = "LBSSetScore";
    EMsg[EMsg["LBSSetScoreResponse"] = 3202] = "LBSSetScoreResponse";
    EMsg[EMsg["LBSFindOrCreateLB"] = 3203] = "LBSFindOrCreateLB";
    EMsg[EMsg["LBSFindOrCreateLBResponse"] = 3204] = "LBSFindOrCreateLBResponse";
    EMsg[EMsg["LBSGetLBEntries"] = 3205] = "LBSGetLBEntries";
    EMsg[EMsg["LBSGetLBEntriesResponse"] = 3206] = "LBSGetLBEntriesResponse";
    EMsg[EMsg["LBSGetLBList"] = 3207] = "LBSGetLBList";
    EMsg[EMsg["LBSGetLBListResponse"] = 3208] = "LBSGetLBListResponse";
    EMsg[EMsg["LBSSetLBDetails"] = 3209] = "LBSSetLBDetails";
    EMsg[EMsg["LBSDeleteLB"] = 3210] = "LBSDeleteLB";
    EMsg[EMsg["LBSDeleteLBEntry"] = 3211] = "LBSDeleteLBEntry";
    EMsg[EMsg["LBSResetLB"] = 3212] = "LBSResetLB";
    EMsg[EMsg["LBSResetLBResponse"] = 3213] = "LBSResetLBResponse";
    EMsg[EMsg["LBSDeleteLBResponse"] = 3214] = "LBSDeleteLBResponse";
    EMsg[EMsg["BaseOGS"] = 3400] = "BaseOGS";
    EMsg[EMsg["OGSBeginSession"] = 3401] = "OGSBeginSession";
    EMsg[EMsg["OGSBeginSessionResponse"] = 3402] = "OGSBeginSessionResponse";
    EMsg[EMsg["OGSEndSession"] = 3403] = "OGSEndSession";
    EMsg[EMsg["OGSEndSessionResponse"] = 3404] = "OGSEndSessionResponse";
    EMsg[EMsg["OGSWriteAppSessionRow"] = 3406] = "OGSWriteAppSessionRow";
    EMsg[EMsg["BaseBRP"] = 3600] = "BaseBRP";
    EMsg[EMsg["BRPStartShippingJobs"] = 3601] = "BRPStartShippingJobs";
    EMsg[EMsg["BRPProcessUSBankReports"] = 3602] = "BRPProcessUSBankReports";
    EMsg[EMsg["BRPProcessGCReports"] = 3603] = "BRPProcessGCReports";
    EMsg[EMsg["BRPProcessPPReports"] = 3604] = "BRPProcessPPReports";
    EMsg[EMsg["BRPSettleNOVA"] = 3605] = "BRPSettleNOVA";
    EMsg[EMsg["BRPSettleCB"] = 3606] = "BRPSettleCB";
    EMsg[EMsg["BRPCommitGC"] = 3607] = "BRPCommitGC";
    EMsg[EMsg["BRPCommitGCResponse"] = 3608] = "BRPCommitGCResponse";
    EMsg[EMsg["BRPFindHungTransactions"] = 3609] = "BRPFindHungTransactions";
    EMsg[EMsg["BRPCheckFinanceCloseOutDate"] = 3610] = "BRPCheckFinanceCloseOutDate";
    EMsg[EMsg["BRPProcessLicenses"] = 3611] = "BRPProcessLicenses";
    EMsg[EMsg["BRPProcessLicensesResponse"] = 3612] = "BRPProcessLicensesResponse";
    EMsg[EMsg["BRPRemoveExpiredPaymentData"] = 3613] = "BRPRemoveExpiredPaymentData";
    EMsg[EMsg["BRPRemoveExpiredPaymentDataResponse"] = 3614] = "BRPRemoveExpiredPaymentDataResponse";
    EMsg[EMsg["BRPConvertToCurrentKeys"] = 3615] = "BRPConvertToCurrentKeys";
    EMsg[EMsg["BRPConvertToCurrentKeysResponse"] = 3616] = "BRPConvertToCurrentKeysResponse";
    EMsg[EMsg["BRPPruneCardUsageStats"] = 3617] = "BRPPruneCardUsageStats";
    EMsg[EMsg["BRPPruneCardUsageStatsResponse"] = 3618] = "BRPPruneCardUsageStatsResponse";
    EMsg[EMsg["BRPCheckActivationCodes"] = 3619] = "BRPCheckActivationCodes";
    EMsg[EMsg["BRPCheckActivationCodesResponse"] = 3620] = "BRPCheckActivationCodesResponse";
    EMsg[EMsg["BRPCommitWP"] = 3621] = "BRPCommitWP";
    EMsg[EMsg["BRPCommitWPResponse"] = 3622] = "BRPCommitWPResponse";
    EMsg[EMsg["BRPProcessWPReports"] = 3623] = "BRPProcessWPReports";
    EMsg[EMsg["BRPProcessPaymentRules"] = 3624] = "BRPProcessPaymentRules";
    EMsg[EMsg["BRPProcessPartnerPayments"] = 3625] = "BRPProcessPartnerPayments";
    EMsg[EMsg["BRPCheckSettlementReports"] = 3626] = "BRPCheckSettlementReports";
    EMsg[EMsg["BRPPostTaxToAvalara"] = 3628] = "BRPPostTaxToAvalara";
    EMsg[EMsg["BRPPostTransactionTax"] = 3629] = "BRPPostTransactionTax";
    EMsg[EMsg["BRPPostTransactionTaxResponse"] = 3630] = "BRPPostTransactionTaxResponse";
    EMsg[EMsg["BRPProcessIMReports"] = 3631] = "BRPProcessIMReports";
    EMsg[EMsg["BaseAMRange2"] = 4000] = "BaseAMRange2";
    EMsg[EMsg["AMCreateChat"] = 4001] = "AMCreateChat";
    EMsg[EMsg["AMCreateChatResponse"] = 4002] = "AMCreateChatResponse";
    EMsg[EMsg["AMUpdateChatMetadata"] = 4003] = "AMUpdateChatMetadata";
    EMsg[EMsg["AMPublishChatMetadata"] = 4004] = "AMPublishChatMetadata";
    EMsg[EMsg["AMSetProfileURL"] = 4005] = "AMSetProfileURL";
    EMsg[EMsg["AMGetAccountEmailAddress"] = 4006] = "AMGetAccountEmailAddress";
    EMsg[EMsg["AMGetAccountEmailAddressResponse"] = 4007] = "AMGetAccountEmailAddressResponse";
    EMsg[EMsg["AMRequestClanData"] = 4008] = "AMRequestClanData";
    EMsg[EMsg["AMRouteToClients"] = 4009] = "AMRouteToClients";
    EMsg[EMsg["AMLeaveClan"] = 4010] = "AMLeaveClan";
    EMsg[EMsg["AMClanPermissions"] = 4011] = "AMClanPermissions";
    EMsg[EMsg["AMClanPermissionsResponse"] = 4012] = "AMClanPermissionsResponse";
    EMsg[EMsg["AMCreateClanEvent"] = 4013] = "AMCreateClanEvent";
    EMsg[EMsg["AMCreateClanEventDummyForRateLimiting"] = 4013] = "AMCreateClanEventDummyForRateLimiting";
    EMsg[EMsg["AMCreateClanEventResponse"] = 4014] = "AMCreateClanEventResponse";
    EMsg[EMsg["AMUpdateClanEvent"] = 4015] = "AMUpdateClanEvent";
    EMsg[EMsg["AMUpdateClanEventDummyForRateLimiting"] = 4015] = "AMUpdateClanEventDummyForRateLimiting";
    EMsg[EMsg["AMUpdateClanEventResponse"] = 4016] = "AMUpdateClanEventResponse";
    EMsg[EMsg["AMGetClanEvents"] = 4017] = "AMGetClanEvents";
    EMsg[EMsg["AMGetClanEventsResponse"] = 4018] = "AMGetClanEventsResponse";
    EMsg[EMsg["AMDeleteClanEvent"] = 4019] = "AMDeleteClanEvent";
    EMsg[EMsg["AMDeleteClanEventResponse"] = 4020] = "AMDeleteClanEventResponse";
    EMsg[EMsg["AMSetClanPermissionSettings"] = 4021] = "AMSetClanPermissionSettings";
    EMsg[EMsg["AMSetClanPermissionSettingsResponse"] = 4022] = "AMSetClanPermissionSettingsResponse";
    EMsg[EMsg["AMGetClanPermissionSettings"] = 4023] = "AMGetClanPermissionSettings";
    EMsg[EMsg["AMGetClanPermissionSettingsResponse"] = 4024] = "AMGetClanPermissionSettingsResponse";
    EMsg[EMsg["AMPublishChatRoomInfo"] = 4025] = "AMPublishChatRoomInfo";
    EMsg[EMsg["ClientChatRoomInfo"] = 4026] = "ClientChatRoomInfo";
    EMsg[EMsg["AMCreateClanAnnouncement"] = 4027] = "AMCreateClanAnnouncement";
    EMsg[EMsg["AMCreateClanAnnouncementResponse"] = 4028] = "AMCreateClanAnnouncementResponse";
    EMsg[EMsg["AMUpdateClanAnnouncement"] = 4029] = "AMUpdateClanAnnouncement";
    EMsg[EMsg["AMUpdateClanAnnouncementResponse"] = 4030] = "AMUpdateClanAnnouncementResponse";
    EMsg[EMsg["AMGetClanAnnouncementsCount"] = 4031] = "AMGetClanAnnouncementsCount";
    EMsg[EMsg["AMGetClanAnnouncementsCountResponse"] = 4032] = "AMGetClanAnnouncementsCountResponse";
    EMsg[EMsg["AMGetClanAnnouncements"] = 4033] = "AMGetClanAnnouncements";
    EMsg[EMsg["AMGetClanAnnouncementsResponse"] = 4034] = "AMGetClanAnnouncementsResponse";
    EMsg[EMsg["AMDeleteClanAnnouncement"] = 4035] = "AMDeleteClanAnnouncement";
    EMsg[EMsg["AMDeleteClanAnnouncementResponse"] = 4036] = "AMDeleteClanAnnouncementResponse";
    EMsg[EMsg["AMGetSingleClanAnnouncement"] = 4037] = "AMGetSingleClanAnnouncement";
    EMsg[EMsg["AMGetSingleClanAnnouncementResponse"] = 4038] = "AMGetSingleClanAnnouncementResponse";
    EMsg[EMsg["AMGetClanHistory"] = 4039] = "AMGetClanHistory";
    EMsg[EMsg["AMGetClanHistoryResponse"] = 4040] = "AMGetClanHistoryResponse";
    EMsg[EMsg["AMGetClanPermissionBits"] = 4041] = "AMGetClanPermissionBits";
    EMsg[EMsg["AMGetClanPermissionBitsResponse"] = 4042] = "AMGetClanPermissionBitsResponse";
    EMsg[EMsg["AMSetClanPermissionBits"] = 4043] = "AMSetClanPermissionBits";
    EMsg[EMsg["AMSetClanPermissionBitsResponse"] = 4044] = "AMSetClanPermissionBitsResponse";
    EMsg[EMsg["AMSessionInfoRequest"] = 4045] = "AMSessionInfoRequest";
    EMsg[EMsg["AMSessionInfoResponse"] = 4046] = "AMSessionInfoResponse";
    EMsg[EMsg["AMValidateWGToken"] = 4047] = "AMValidateWGToken";
    EMsg[EMsg["AMGetSingleClanEvent"] = 4048] = "AMGetSingleClanEvent";
    EMsg[EMsg["AMGetSingleClanEventResponse"] = 4049] = "AMGetSingleClanEventResponse";
    EMsg[EMsg["AMGetClanRank"] = 4050] = "AMGetClanRank";
    EMsg[EMsg["AMGetClanRankResponse"] = 4051] = "AMGetClanRankResponse";
    EMsg[EMsg["AMSetClanRank"] = 4052] = "AMSetClanRank";
    EMsg[EMsg["AMSetClanRankResponse"] = 4053] = "AMSetClanRankResponse";
    EMsg[EMsg["AMGetClanPOTW"] = 4054] = "AMGetClanPOTW";
    EMsg[EMsg["AMGetClanPOTWResponse"] = 4055] = "AMGetClanPOTWResponse";
    EMsg[EMsg["AMSetClanPOTW"] = 4056] = "AMSetClanPOTW";
    EMsg[EMsg["AMSetClanPOTWResponse"] = 4057] = "AMSetClanPOTWResponse";
    EMsg[EMsg["AMRequestChatMetadata"] = 4058] = "AMRequestChatMetadata";
    EMsg[EMsg["AMDumpUser"] = 4059] = "AMDumpUser";
    EMsg[EMsg["AMKickUserFromClan"] = 4060] = "AMKickUserFromClan";
    EMsg[EMsg["AMAddFounderToClan"] = 4061] = "AMAddFounderToClan";
    EMsg[EMsg["AMValidateWGTokenResponse"] = 4062] = "AMValidateWGTokenResponse";
    EMsg[EMsg["AMSetCommunityState"] = 4063] = "AMSetCommunityState";
    EMsg[EMsg["AMSetAccountDetails"] = 4064] = "AMSetAccountDetails";
    EMsg[EMsg["AMGetChatBanList"] = 4065] = "AMGetChatBanList";
    EMsg[EMsg["AMGetChatBanListResponse"] = 4066] = "AMGetChatBanListResponse";
    EMsg[EMsg["AMUnBanFromChat"] = 4067] = "AMUnBanFromChat";
    EMsg[EMsg["AMSetClanDetails"] = 4068] = "AMSetClanDetails";
    EMsg[EMsg["AMGetAccountLinks"] = 4069] = "AMGetAccountLinks";
    EMsg[EMsg["AMGetAccountLinksResponse"] = 4070] = "AMGetAccountLinksResponse";
    EMsg[EMsg["AMSetAccountLinks"] = 4071] = "AMSetAccountLinks";
    EMsg[EMsg["AMSetAccountLinksResponse"] = 4072] = "AMSetAccountLinksResponse";
    EMsg[EMsg["AMGetUserGameStats"] = 4073] = "AMGetUserGameStats";
    EMsg[EMsg["UGSGetUserGameStats"] = 4073] = "UGSGetUserGameStats";
    EMsg[EMsg["AMGetUserGameStatsResponse"] = 4074] = "AMGetUserGameStatsResponse";
    EMsg[EMsg["UGSGetUserGameStatsResponse"] = 4074] = "UGSGetUserGameStatsResponse";
    EMsg[EMsg["AMCheckClanMembership"] = 4075] = "AMCheckClanMembership";
    EMsg[EMsg["AMGetClanMembers"] = 4076] = "AMGetClanMembers";
    EMsg[EMsg["AMGetClanMembersResponse"] = 4077] = "AMGetClanMembersResponse";
    EMsg[EMsg["AMJoinPublicClan"] = 4078] = "AMJoinPublicClan";
    EMsg[EMsg["AMNotifyChatOfClanChange"] = 4079] = "AMNotifyChatOfClanChange";
    EMsg[EMsg["AMResubmitPurchase"] = 4080] = "AMResubmitPurchase";
    EMsg[EMsg["AMAddFriend"] = 4081] = "AMAddFriend";
    EMsg[EMsg["AMAddFriendResponse"] = 4082] = "AMAddFriendResponse";
    EMsg[EMsg["AMRemoveFriend"] = 4083] = "AMRemoveFriend";
    EMsg[EMsg["AMDumpClan"] = 4084] = "AMDumpClan";
    EMsg[EMsg["AMChangeClanOwner"] = 4085] = "AMChangeClanOwner";
    EMsg[EMsg["AMCancelEasyCollect"] = 4086] = "AMCancelEasyCollect";
    EMsg[EMsg["AMCancelEasyCollectResponse"] = 4087] = "AMCancelEasyCollectResponse";
    EMsg[EMsg["AMGetClanMembershipList"] = 4088] = "AMGetClanMembershipList";
    EMsg[EMsg["AMGetClanMembershipListResponse"] = 4089] = "AMGetClanMembershipListResponse";
    EMsg[EMsg["AMClansInCommon"] = 4090] = "AMClansInCommon";
    EMsg[EMsg["AMClansInCommonResponse"] = 4091] = "AMClansInCommonResponse";
    EMsg[EMsg["AMIsValidAccountID"] = 4092] = "AMIsValidAccountID";
    EMsg[EMsg["AMConvertClan"] = 4093] = "AMConvertClan";
    EMsg[EMsg["AMGetGiftTargetListRelay"] = 4094] = "AMGetGiftTargetListRelay";
    EMsg[EMsg["AMWipeFriendsList"] = 4095] = "AMWipeFriendsList";
    EMsg[EMsg["AMSetIgnored"] = 4096] = "AMSetIgnored";
    EMsg[EMsg["AMClansInCommonCountResponse"] = 4097] = "AMClansInCommonCountResponse";
    EMsg[EMsg["AMFriendsList"] = 4098] = "AMFriendsList";
    EMsg[EMsg["AMFriendsListResponse"] = 4099] = "AMFriendsListResponse";
    EMsg[EMsg["AMFriendsInCommon"] = 4100] = "AMFriendsInCommon";
    EMsg[EMsg["AMFriendsInCommonResponse"] = 4101] = "AMFriendsInCommonResponse";
    EMsg[EMsg["AMFriendsInCommonCountResponse"] = 4102] = "AMFriendsInCommonCountResponse";
    EMsg[EMsg["AMClansInCommonCount"] = 4103] = "AMClansInCommonCount";
    EMsg[EMsg["AMChallengeVerdict"] = 4104] = "AMChallengeVerdict";
    EMsg[EMsg["AMChallengeNotification"] = 4105] = "AMChallengeNotification";
    EMsg[EMsg["AMFindGSByIP"] = 4106] = "AMFindGSByIP";
    EMsg[EMsg["AMFoundGSByIP"] = 4107] = "AMFoundGSByIP";
    EMsg[EMsg["AMGiftRevoked"] = 4108] = "AMGiftRevoked";
    EMsg[EMsg["AMCreateAccountRecord"] = 4109] = "AMCreateAccountRecord";
    EMsg[EMsg["AMUserClanList"] = 4110] = "AMUserClanList";
    EMsg[EMsg["AMUserClanListResponse"] = 4111] = "AMUserClanListResponse";
    EMsg[EMsg["AMGetAccountDetails2"] = 4112] = "AMGetAccountDetails2";
    EMsg[EMsg["AMGetAccountDetailsResponse2"] = 4113] = "AMGetAccountDetailsResponse2";
    EMsg[EMsg["AMSetCommunityProfileSettings"] = 4114] = "AMSetCommunityProfileSettings";
    EMsg[EMsg["AMSetCommunityProfileSettingsResponse"] = 4115] = "AMSetCommunityProfileSettingsResponse";
    EMsg[EMsg["AMGetCommunityPrivacyState"] = 4116] = "AMGetCommunityPrivacyState";
    EMsg[EMsg["AMGetCommunityPrivacyStateResponse"] = 4117] = "AMGetCommunityPrivacyStateResponse";
    EMsg[EMsg["AMCheckClanInviteRateLimiting"] = 4118] = "AMCheckClanInviteRateLimiting";
    EMsg[EMsg["AMGetUserAchievementStatus"] = 4119] = "AMGetUserAchievementStatus";
    EMsg[EMsg["UGSGetUserAchievementStatus"] = 4119] = "UGSGetUserAchievementStatus";
    EMsg[EMsg["AMGetIgnored"] = 4120] = "AMGetIgnored";
    EMsg[EMsg["AMGetIgnoredResponse"] = 4121] = "AMGetIgnoredResponse";
    EMsg[EMsg["AMSetIgnoredResponse"] = 4122] = "AMSetIgnoredResponse";
    EMsg[EMsg["AMSetFriendRelationshipNone"] = 4123] = "AMSetFriendRelationshipNone";
    EMsg[EMsg["AMGetFriendRelationship"] = 4124] = "AMGetFriendRelationship";
    EMsg[EMsg["AMGetFriendRelationshipResponse"] = 4125] = "AMGetFriendRelationshipResponse";
    EMsg[EMsg["AMServiceModulesCache"] = 4126] = "AMServiceModulesCache";
    EMsg[EMsg["AMServiceModulesCall"] = 4127] = "AMServiceModulesCall";
    EMsg[EMsg["AMServiceModulesCallResponse"] = 4128] = "AMServiceModulesCallResponse";
    EMsg[EMsg["AMGetCaptchaDataForIP"] = 4129] = "AMGetCaptchaDataForIP";
    EMsg[EMsg["AMGetCaptchaDataForIPResponse"] = 4130] = "AMGetCaptchaDataForIPResponse";
    EMsg[EMsg["AMValidateCaptchaDataForIP"] = 4131] = "AMValidateCaptchaDataForIP";
    EMsg[EMsg["AMValidateCaptchaDataForIPResponse"] = 4132] = "AMValidateCaptchaDataForIPResponse";
    EMsg[EMsg["AMTrackFailedAuthByIP"] = 4133] = "AMTrackFailedAuthByIP";
    EMsg[EMsg["AMGetCaptchaDataByGID"] = 4134] = "AMGetCaptchaDataByGID";
    EMsg[EMsg["AMGetCaptchaDataByGIDResponse"] = 4135] = "AMGetCaptchaDataByGIDResponse";
    EMsg[EMsg["AMGetLobbyList"] = 4136] = "AMGetLobbyList";
    EMsg[EMsg["AMGetLobbyListResponse"] = 4137] = "AMGetLobbyListResponse";
    EMsg[EMsg["AMGetLobbyMetadata"] = 4138] = "AMGetLobbyMetadata";
    EMsg[EMsg["AMGetLobbyMetadataResponse"] = 4139] = "AMGetLobbyMetadataResponse";
    EMsg[EMsg["CommunityAddFriendNews"] = 4140] = "CommunityAddFriendNews";
    EMsg[EMsg["AMAddClanNews"] = 4141] = "AMAddClanNews";
    EMsg[EMsg["AMWriteNews"] = 4142] = "AMWriteNews";
    EMsg[EMsg["AMFindClanUser"] = 4143] = "AMFindClanUser";
    EMsg[EMsg["AMFindClanUserResponse"] = 4144] = "AMFindClanUserResponse";
    EMsg[EMsg["AMBanFromChat"] = 4145] = "AMBanFromChat";
    EMsg[EMsg["AMGetUserHistoryResponse"] = 4146] = "AMGetUserHistoryResponse";
    EMsg[EMsg["AMGetUserNewsSubscriptions"] = 4147] = "AMGetUserNewsSubscriptions";
    EMsg[EMsg["AMGetUserNewsSubscriptionsResponse"] = 4148] = "AMGetUserNewsSubscriptionsResponse";
    EMsg[EMsg["AMSetUserNewsSubscriptions"] = 4149] = "AMSetUserNewsSubscriptions";
    EMsg[EMsg["AMGetUserNews"] = 4150] = "AMGetUserNews";
    EMsg[EMsg["AMGetUserNewsResponse"] = 4151] = "AMGetUserNewsResponse";
    EMsg[EMsg["AMSendQueuedEmails"] = 4152] = "AMSendQueuedEmails";
    EMsg[EMsg["AMSetLicenseFlags"] = 4153] = "AMSetLicenseFlags";
    EMsg[EMsg["AMGetUserHistory"] = 4154] = "AMGetUserHistory";
    EMsg[EMsg["CommunityDeleteUserNews"] = 4155] = "CommunityDeleteUserNews";
    EMsg[EMsg["AMAllowUserFilesRequest"] = 4156] = "AMAllowUserFilesRequest";
    EMsg[EMsg["AMAllowUserFilesResponse"] = 4157] = "AMAllowUserFilesResponse";
    EMsg[EMsg["AMGetAccountStatus"] = 4158] = "AMGetAccountStatus";
    EMsg[EMsg["AMGetAccountStatusResponse"] = 4159] = "AMGetAccountStatusResponse";
    EMsg[EMsg["AMEditBanReason"] = 4160] = "AMEditBanReason";
    EMsg[EMsg["AMCheckClanMembershipResponse"] = 4161] = "AMCheckClanMembershipResponse";
    EMsg[EMsg["AMProbeClanMembershipList"] = 4162] = "AMProbeClanMembershipList";
    EMsg[EMsg["AMProbeClanMembershipListResponse"] = 4163] = "AMProbeClanMembershipListResponse";
    EMsg[EMsg["UGSGetUserAchievementStatusResponse"] = 4164] = "UGSGetUserAchievementStatusResponse";
    EMsg[EMsg["AMGetFriendsLobbies"] = 4165] = "AMGetFriendsLobbies";
    EMsg[EMsg["AMGetFriendsLobbiesResponse"] = 4166] = "AMGetFriendsLobbiesResponse";
    EMsg[EMsg["AMGetUserFriendNewsResponse"] = 4172] = "AMGetUserFriendNewsResponse";
    EMsg[EMsg["CommunityGetUserFriendNews"] = 4173] = "CommunityGetUserFriendNews";
    EMsg[EMsg["AMGetUserClansNewsResponse"] = 4174] = "AMGetUserClansNewsResponse";
    EMsg[EMsg["AMGetUserClansNews"] = 4175] = "AMGetUserClansNews";
    EMsg[EMsg["AMStoreInitPurchase"] = 4176] = "AMStoreInitPurchase";
    EMsg[EMsg["AMStoreInitPurchaseResponse"] = 4177] = "AMStoreInitPurchaseResponse";
    EMsg[EMsg["AMStoreGetFinalPrice"] = 4178] = "AMStoreGetFinalPrice";
    EMsg[EMsg["AMStoreGetFinalPriceResponse"] = 4179] = "AMStoreGetFinalPriceResponse";
    EMsg[EMsg["AMStoreCompletePurchase"] = 4180] = "AMStoreCompletePurchase";
    EMsg[EMsg["AMStoreCancelPurchase"] = 4181] = "AMStoreCancelPurchase";
    EMsg[EMsg["AMStorePurchaseResponse"] = 4182] = "AMStorePurchaseResponse";
    EMsg[EMsg["AMCreateAccountRecordInSteam3"] = 4183] = "AMCreateAccountRecordInSteam3";
    EMsg[EMsg["AMGetPreviousCBAccount"] = 4184] = "AMGetPreviousCBAccount";
    EMsg[EMsg["AMGetPreviousCBAccountResponse"] = 4185] = "AMGetPreviousCBAccountResponse";
    EMsg[EMsg["AMUpdateBillingAddress"] = 4186] = "AMUpdateBillingAddress";
    EMsg[EMsg["AMUpdateBillingAddressResponse"] = 4187] = "AMUpdateBillingAddressResponse";
    EMsg[EMsg["AMGetBillingAddress"] = 4188] = "AMGetBillingAddress";
    EMsg[EMsg["AMGetBillingAddressResponse"] = 4189] = "AMGetBillingAddressResponse";
    EMsg[EMsg["AMGetUserLicenseHistory"] = 4190] = "AMGetUserLicenseHistory";
    EMsg[EMsg["AMGetUserLicenseHistoryResponse"] = 4191] = "AMGetUserLicenseHistoryResponse";
    EMsg[EMsg["AMSupportChangePassword"] = 4194] = "AMSupportChangePassword";
    EMsg[EMsg["AMSupportChangeEmail"] = 4195] = "AMSupportChangeEmail";
    EMsg[EMsg["AMSupportChangeSecretQA"] = 4196] = "AMSupportChangeSecretQA";
    EMsg[EMsg["AMResetUserVerificationGSByIP"] = 4197] = "AMResetUserVerificationGSByIP";
    EMsg[EMsg["AMUpdateGSPlayStats"] = 4198] = "AMUpdateGSPlayStats";
    EMsg[EMsg["AMSupportEnableOrDisable"] = 4199] = "AMSupportEnableOrDisable";
    EMsg[EMsg["AMGetComments"] = 4200] = "AMGetComments";
    EMsg[EMsg["AMGetCommentsResponse"] = 4201] = "AMGetCommentsResponse";
    EMsg[EMsg["AMAddComment"] = 4202] = "AMAddComment";
    EMsg[EMsg["AMAddCommentResponse"] = 4203] = "AMAddCommentResponse";
    EMsg[EMsg["AMDeleteComment"] = 4204] = "AMDeleteComment";
    EMsg[EMsg["AMDeleteCommentResponse"] = 4205] = "AMDeleteCommentResponse";
    EMsg[EMsg["AMGetPurchaseStatus"] = 4206] = "AMGetPurchaseStatus";
    EMsg[EMsg["AMSupportIsAccountEnabled"] = 4209] = "AMSupportIsAccountEnabled";
    EMsg[EMsg["AMSupportIsAccountEnabledResponse"] = 4210] = "AMSupportIsAccountEnabledResponse";
    EMsg[EMsg["AMGetUserStats"] = 4211] = "AMGetUserStats";
    EMsg[EMsg["UGSGetUserStats"] = 4211] = "UGSGetUserStats";
    EMsg[EMsg["AMSupportKickSession"] = 4212] = "AMSupportKickSession";
    EMsg[EMsg["AMGSSearch"] = 4213] = "AMGSSearch";
    EMsg[EMsg["MarketingMessageUpdate"] = 4216] = "MarketingMessageUpdate";
    EMsg[EMsg["AMRouteFriendMsg"] = 4219] = "AMRouteFriendMsg";
    EMsg[EMsg["ChatServerRouteFriendMsg"] = 4219] = "ChatServerRouteFriendMsg";
    EMsg[EMsg["AMTicketAuthRequestOrResponse"] = 4220] = "AMTicketAuthRequestOrResponse";
    EMsg[EMsg["AMVerifyDepotManagementRights"] = 4222] = "AMVerifyDepotManagementRights";
    EMsg[EMsg["AMVerifyDepotManagementRightsResponse"] = 4223] = "AMVerifyDepotManagementRightsResponse";
    EMsg[EMsg["AMAddFreeLicense"] = 4224] = "AMAddFreeLicense";
    EMsg[EMsg["AMGetUserFriendsMinutesPlayed"] = 4225] = "AMGetUserFriendsMinutesPlayed";
    EMsg[EMsg["AMGetUserFriendsMinutesPlayedResponse"] = 4226] = "AMGetUserFriendsMinutesPlayedResponse";
    EMsg[EMsg["AMGetUserMinutesPlayed"] = 4227] = "AMGetUserMinutesPlayed";
    EMsg[EMsg["AMGetUserMinutesPlayedResponse"] = 4228] = "AMGetUserMinutesPlayedResponse";
    EMsg[EMsg["AMValidateEmailLink"] = 4231] = "AMValidateEmailLink";
    EMsg[EMsg["AMValidateEmailLinkResponse"] = 4232] = "AMValidateEmailLinkResponse";
    EMsg[EMsg["AMAddUsersToMarketingTreatment"] = 4234] = "AMAddUsersToMarketingTreatment";
    EMsg[EMsg["AMStoreUserStats"] = 4236] = "AMStoreUserStats";
    EMsg[EMsg["UGSStoreUserStats"] = 4236] = "UGSStoreUserStats";
    EMsg[EMsg["AMGetUserGameplayInfo"] = 4237] = "AMGetUserGameplayInfo";
    EMsg[EMsg["AMGetUserGameplayInfoResponse"] = 4238] = "AMGetUserGameplayInfoResponse";
    EMsg[EMsg["AMGetCardList"] = 4239] = "AMGetCardList";
    EMsg[EMsg["AMGetCardListResponse"] = 4240] = "AMGetCardListResponse";
    EMsg[EMsg["AMDeleteStoredCard"] = 4241] = "AMDeleteStoredCard";
    EMsg[EMsg["AMRevokeLegacyGameKeys"] = 4242] = "AMRevokeLegacyGameKeys";
    EMsg[EMsg["AMGetWalletDetails"] = 4244] = "AMGetWalletDetails";
    EMsg[EMsg["AMGetWalletDetailsResponse"] = 4245] = "AMGetWalletDetailsResponse";
    EMsg[EMsg["AMDeleteStoredPaymentInfo"] = 4246] = "AMDeleteStoredPaymentInfo";
    EMsg[EMsg["AMGetStoredPaymentSummary"] = 4247] = "AMGetStoredPaymentSummary";
    EMsg[EMsg["AMGetStoredPaymentSummaryResponse"] = 4248] = "AMGetStoredPaymentSummaryResponse";
    EMsg[EMsg["AMGetWalletConversionRate"] = 4249] = "AMGetWalletConversionRate";
    EMsg[EMsg["AMGetWalletConversionRateResponse"] = 4250] = "AMGetWalletConversionRateResponse";
    EMsg[EMsg["AMConvertWallet"] = 4251] = "AMConvertWallet";
    EMsg[EMsg["AMConvertWalletResponse"] = 4252] = "AMConvertWalletResponse";
    EMsg[EMsg["AMRelayGetFriendsWhoPlayGame"] = 4253] = "AMRelayGetFriendsWhoPlayGame";
    EMsg[EMsg["AMRelayGetFriendsWhoPlayGameResponse"] = 4254] = "AMRelayGetFriendsWhoPlayGameResponse";
    EMsg[EMsg["AMSetPreApproval"] = 4255] = "AMSetPreApproval";
    EMsg[EMsg["AMSetPreApprovalResponse"] = 4256] = "AMSetPreApprovalResponse";
    EMsg[EMsg["AMMarketingTreatmentUpdate"] = 4257] = "AMMarketingTreatmentUpdate";
    EMsg[EMsg["AMCreateRefund"] = 4258] = "AMCreateRefund";
    EMsg[EMsg["AMCreateRefundResponse"] = 4259] = "AMCreateRefundResponse";
    EMsg[EMsg["AMCreateChargeback"] = 4260] = "AMCreateChargeback";
    EMsg[EMsg["AMCreateChargebackResponse"] = 4261] = "AMCreateChargebackResponse";
    EMsg[EMsg["AMCreateDispute"] = 4262] = "AMCreateDispute";
    EMsg[EMsg["AMCreateDisputeResponse"] = 4263] = "AMCreateDisputeResponse";
    EMsg[EMsg["AMClearDispute"] = 4264] = "AMClearDispute";
    EMsg[EMsg["AMClearDisputeResponse"] = 4265] = "AMClearDisputeResponse";
    EMsg[EMsg["AMCreateFinancialAdjustment"] = 4265] = "AMCreateFinancialAdjustment";
    EMsg[EMsg["AMPlayerNicknameList"] = 4266] = "AMPlayerNicknameList";
    EMsg[EMsg["AMPlayerNicknameListResponse"] = 4267] = "AMPlayerNicknameListResponse";
    EMsg[EMsg["AMSetDRMTestConfig"] = 4268] = "AMSetDRMTestConfig";
    EMsg[EMsg["AMGetUserCurrentGameInfo"] = 4269] = "AMGetUserCurrentGameInfo";
    EMsg[EMsg["AMGetUserCurrentGameInfoResponse"] = 4270] = "AMGetUserCurrentGameInfoResponse";
    EMsg[EMsg["AMGetGSPlayerList"] = 4271] = "AMGetGSPlayerList";
    EMsg[EMsg["AMGetGSPlayerListResponse"] = 4272] = "AMGetGSPlayerListResponse";
    EMsg[EMsg["AMUpdatePersonaStateCache"] = 4275] = "AMUpdatePersonaStateCache";
    EMsg[EMsg["AMGetGameMembers"] = 4276] = "AMGetGameMembers";
    EMsg[EMsg["AMGetGameMembersResponse"] = 4277] = "AMGetGameMembersResponse";
    EMsg[EMsg["AMGetSteamIDForMicroTxn"] = 4278] = "AMGetSteamIDForMicroTxn";
    EMsg[EMsg["AMGetSteamIDForMicroTxnResponse"] = 4279] = "AMGetSteamIDForMicroTxnResponse";
    EMsg[EMsg["AMAddPublisherUser"] = 4280] = "AMAddPublisherUser";
    EMsg[EMsg["AMSetPartnerMember"] = 4280] = "AMSetPartnerMember";
    EMsg[EMsg["AMRemovePublisherUser"] = 4281] = "AMRemovePublisherUser";
    EMsg[EMsg["AMGetUserLicenseList"] = 4282] = "AMGetUserLicenseList";
    EMsg[EMsg["AMGetUserLicenseListResponse"] = 4283] = "AMGetUserLicenseListResponse";
    EMsg[EMsg["AMReloadGameGroupPolicy"] = 4284] = "AMReloadGameGroupPolicy";
    EMsg[EMsg["AMAddFreeLicenseResponse"] = 4285] = "AMAddFreeLicenseResponse";
    EMsg[EMsg["AMVACStatusUpdate"] = 4286] = "AMVACStatusUpdate";
    EMsg[EMsg["AMGetAccountDetails"] = 4287] = "AMGetAccountDetails";
    EMsg[EMsg["AMGetAccountDetailsResponse"] = 4288] = "AMGetAccountDetailsResponse";
    EMsg[EMsg["AMGetPlayerLinkDetails"] = 4289] = "AMGetPlayerLinkDetails";
    EMsg[EMsg["AMGetPlayerLinkDetailsResponse"] = 4290] = "AMGetPlayerLinkDetailsResponse";
    EMsg[EMsg["AMSubscribeToPersonaFeed"] = 4291] = "AMSubscribeToPersonaFeed";
    EMsg[EMsg["AMGetUserVacBanList"] = 4292] = "AMGetUserVacBanList";
    EMsg[EMsg["AMGetUserVacBanListResponse"] = 4293] = "AMGetUserVacBanListResponse";
    EMsg[EMsg["AMGetAccountFlagsForWGSpoofing"] = 4294] = "AMGetAccountFlagsForWGSpoofing";
    EMsg[EMsg["AMGetAccountFlagsForWGSpoofingResponse"] = 4295] = "AMGetAccountFlagsForWGSpoofingResponse";
    EMsg[EMsg["AMGetFriendsWishlistInfo"] = 4296] = "AMGetFriendsWishlistInfo";
    EMsg[EMsg["AMGetFriendsWishlistInfoResponse"] = 4297] = "AMGetFriendsWishlistInfoResponse";
    EMsg[EMsg["AMGetClanOfficers"] = 4298] = "AMGetClanOfficers";
    EMsg[EMsg["AMGetClanOfficersResponse"] = 4299] = "AMGetClanOfficersResponse";
    EMsg[EMsg["AMNameChange"] = 4300] = "AMNameChange";
    EMsg[EMsg["AMGetNameHistory"] = 4301] = "AMGetNameHistory";
    EMsg[EMsg["AMGetNameHistoryResponse"] = 4302] = "AMGetNameHistoryResponse";
    EMsg[EMsg["AMUpdateProviderStatus"] = 4305] = "AMUpdateProviderStatus";
    EMsg[EMsg["AMClearPersonaMetadataBlob"] = 4306] = "AMClearPersonaMetadataBlob";
    EMsg[EMsg["AMSupportRemoveAccountSecurity"] = 4307] = "AMSupportRemoveAccountSecurity";
    EMsg[EMsg["AMIsAccountInCaptchaGracePeriod"] = 4308] = "AMIsAccountInCaptchaGracePeriod";
    EMsg[EMsg["AMIsAccountInCaptchaGracePeriodResponse"] = 4309] = "AMIsAccountInCaptchaGracePeriodResponse";
    EMsg[EMsg["AMAccountPS3Unlink"] = 4310] = "AMAccountPS3Unlink";
    EMsg[EMsg["AMAccountPS3UnlinkResponse"] = 4311] = "AMAccountPS3UnlinkResponse";
    EMsg[EMsg["AMStoreUserStatsResponse"] = 4312] = "AMStoreUserStatsResponse";
    EMsg[EMsg["UGSStoreUserStatsResponse"] = 4312] = "UGSStoreUserStatsResponse";
    EMsg[EMsg["AMGetAccountPSNInfo"] = 4313] = "AMGetAccountPSNInfo";
    EMsg[EMsg["AMGetAccountPSNInfoResponse"] = 4314] = "AMGetAccountPSNInfoResponse";
    EMsg[EMsg["AMAuthenticatedPlayerList"] = 4315] = "AMAuthenticatedPlayerList";
    EMsg[EMsg["AMGetUserGifts"] = 4316] = "AMGetUserGifts";
    EMsg[EMsg["AMGetUserGiftsResponse"] = 4317] = "AMGetUserGiftsResponse";
    EMsg[EMsg["AMTransferLockedGifts"] = 4320] = "AMTransferLockedGifts";
    EMsg[EMsg["AMTransferLockedGiftsResponse"] = 4321] = "AMTransferLockedGiftsResponse";
    EMsg[EMsg["AMPlayerHostedOnGameServer"] = 4322] = "AMPlayerHostedOnGameServer";
    EMsg[EMsg["AMGetAccountBanInfo"] = 4323] = "AMGetAccountBanInfo";
    EMsg[EMsg["AMGetAccountBanInfoResponse"] = 4324] = "AMGetAccountBanInfoResponse";
    EMsg[EMsg["AMRecordBanEnforcement"] = 4325] = "AMRecordBanEnforcement";
    EMsg[EMsg["AMRollbackGiftTransfer"] = 4326] = "AMRollbackGiftTransfer";
    EMsg[EMsg["AMRollbackGiftTransferResponse"] = 4327] = "AMRollbackGiftTransferResponse";
    EMsg[EMsg["AMHandlePendingTransaction"] = 4328] = "AMHandlePendingTransaction";
    EMsg[EMsg["AMRequestClanDetails"] = 4329] = "AMRequestClanDetails";
    EMsg[EMsg["AMDeleteStoredPaypalAgreement"] = 4330] = "AMDeleteStoredPaypalAgreement";
    EMsg[EMsg["AMGameServerUpdate"] = 4331] = "AMGameServerUpdate";
    EMsg[EMsg["AMGameServerRemove"] = 4332] = "AMGameServerRemove";
    EMsg[EMsg["AMGetPaypalAgreements"] = 4333] = "AMGetPaypalAgreements";
    EMsg[EMsg["AMGetPaypalAgreementsResponse"] = 4334] = "AMGetPaypalAgreementsResponse";
    EMsg[EMsg["AMGameServerPlayerCompatibilityCheck"] = 4335] = "AMGameServerPlayerCompatibilityCheck";
    EMsg[EMsg["AMGameServerPlayerCompatibilityCheckResponse"] = 4336] = "AMGameServerPlayerCompatibilityCheckResponse";
    EMsg[EMsg["AMRenewLicense"] = 4337] = "AMRenewLicense";
    EMsg[EMsg["AMGetAccountCommunityBanInfo"] = 4338] = "AMGetAccountCommunityBanInfo";
    EMsg[EMsg["AMGetAccountCommunityBanInfoResponse"] = 4339] = "AMGetAccountCommunityBanInfoResponse";
    EMsg[EMsg["AMGameServerAccountChangePassword"] = 4340] = "AMGameServerAccountChangePassword";
    EMsg[EMsg["AMGameServerAccountDeleteAccount"] = 4341] = "AMGameServerAccountDeleteAccount";
    EMsg[EMsg["AMRenewAgreement"] = 4342] = "AMRenewAgreement";
    EMsg[EMsg["AMSendEmail"] = 4343] = "AMSendEmail";
    EMsg[EMsg["AMXsollaPayment"] = 4344] = "AMXsollaPayment";
    EMsg[EMsg["AMXsollaPaymentResponse"] = 4345] = "AMXsollaPaymentResponse";
    EMsg[EMsg["AMAcctAllowedToPurchase"] = 4346] = "AMAcctAllowedToPurchase";
    EMsg[EMsg["AMAcctAllowedToPurchaseResponse"] = 4347] = "AMAcctAllowedToPurchaseResponse";
    EMsg[EMsg["AMSwapKioskDeposit"] = 4348] = "AMSwapKioskDeposit";
    EMsg[EMsg["AMSwapKioskDepositResponse"] = 4349] = "AMSwapKioskDepositResponse";
    EMsg[EMsg["AMSetUserGiftUnowned"] = 4350] = "AMSetUserGiftUnowned";
    EMsg[EMsg["AMSetUserGiftUnownedResponse"] = 4351] = "AMSetUserGiftUnownedResponse";
    EMsg[EMsg["AMClaimUnownedUserGift"] = 4352] = "AMClaimUnownedUserGift";
    EMsg[EMsg["AMClaimUnownedUserGiftResponse"] = 4353] = "AMClaimUnownedUserGiftResponse";
    EMsg[EMsg["AMSetClanName"] = 4354] = "AMSetClanName";
    EMsg[EMsg["AMSetClanNameResponse"] = 4355] = "AMSetClanNameResponse";
    EMsg[EMsg["AMGrantCoupon"] = 4356] = "AMGrantCoupon";
    EMsg[EMsg["AMGrantCouponResponse"] = 4357] = "AMGrantCouponResponse";
    EMsg[EMsg["AMIsPackageRestrictedInUserCountry"] = 4358] = "AMIsPackageRestrictedInUserCountry";
    EMsg[EMsg["AMIsPackageRestrictedInUserCountryResponse"] = 4359] = "AMIsPackageRestrictedInUserCountryResponse";
    EMsg[EMsg["AMHandlePendingTransactionResponse"] = 4360] = "AMHandlePendingTransactionResponse";
    EMsg[EMsg["AMGrantGuestPasses2"] = 4361] = "AMGrantGuestPasses2";
    EMsg[EMsg["AMGrantGuestPasses2Response"] = 4362] = "AMGrantGuestPasses2Response";
    EMsg[EMsg["AMSessionQuery"] = 4363] = "AMSessionQuery";
    EMsg[EMsg["AMSessionQueryResponse"] = 4364] = "AMSessionQueryResponse";
    EMsg[EMsg["AMGetPlayerBanDetails"] = 4365] = "AMGetPlayerBanDetails";
    EMsg[EMsg["AMGetPlayerBanDetailsResponse"] = 4366] = "AMGetPlayerBanDetailsResponse";
    EMsg[EMsg["AMFinalizePurchase"] = 4367] = "AMFinalizePurchase";
    EMsg[EMsg["AMFinalizePurchaseResponse"] = 4368] = "AMFinalizePurchaseResponse";
    EMsg[EMsg["AMPersonaChangeResponse"] = 4372] = "AMPersonaChangeResponse";
    EMsg[EMsg["AMGetClanDetailsForForumCreation"] = 4373] = "AMGetClanDetailsForForumCreation";
    EMsg[EMsg["AMGetClanDetailsForForumCreationResponse"] = 4374] = "AMGetClanDetailsForForumCreationResponse";
    EMsg[EMsg["AMGetPendingNotificationCount"] = 4375] = "AMGetPendingNotificationCount";
    EMsg[EMsg["AMGetPendingNotificationCountResponse"] = 4376] = "AMGetPendingNotificationCountResponse";
    EMsg[EMsg["AMPasswordHashUpgrade"] = 4377] = "AMPasswordHashUpgrade";
    EMsg[EMsg["AMMoPayPayment"] = 4378] = "AMMoPayPayment";
    EMsg[EMsg["AMMoPayPaymentResponse"] = 4379] = "AMMoPayPaymentResponse";
    EMsg[EMsg["AMBoaCompraPayment"] = 4380] = "AMBoaCompraPayment";
    EMsg[EMsg["AMBoaCompraPaymentResponse"] = 4381] = "AMBoaCompraPaymentResponse";
    EMsg[EMsg["AMExpireCaptchaByGID"] = 4382] = "AMExpireCaptchaByGID";
    EMsg[EMsg["AMCompleteExternalPurchase"] = 4383] = "AMCompleteExternalPurchase";
    EMsg[EMsg["AMCompleteExternalPurchaseResponse"] = 4384] = "AMCompleteExternalPurchaseResponse";
    EMsg[EMsg["AMResolveNegativeWalletCredits"] = 4385] = "AMResolveNegativeWalletCredits";
    EMsg[EMsg["AMResolveNegativeWalletCreditsResponse"] = 4386] = "AMResolveNegativeWalletCreditsResponse";
    EMsg[EMsg["AMPayelpPayment"] = 4387] = "AMPayelpPayment";
    EMsg[EMsg["AMPayelpPaymentResponse"] = 4388] = "AMPayelpPaymentResponse";
    EMsg[EMsg["AMPlayerGetClanBasicDetails"] = 4389] = "AMPlayerGetClanBasicDetails";
    EMsg[EMsg["AMPlayerGetClanBasicDetailsResponse"] = 4390] = "AMPlayerGetClanBasicDetailsResponse";
    EMsg[EMsg["AMMOLPayment"] = 4391] = "AMMOLPayment";
    EMsg[EMsg["AMMOLPaymentResponse"] = 4392] = "AMMOLPaymentResponse";
    EMsg[EMsg["GetUserIPCountry"] = 4393] = "GetUserIPCountry";
    EMsg[EMsg["GetUserIPCountryResponse"] = 4394] = "GetUserIPCountryResponse";
    EMsg[EMsg["NotificationOfSuspiciousActivity"] = 4395] = "NotificationOfSuspiciousActivity";
    EMsg[EMsg["AMDegicaPayment"] = 4396] = "AMDegicaPayment";
    EMsg[EMsg["AMDegicaPaymentResponse"] = 4397] = "AMDegicaPaymentResponse";
    EMsg[EMsg["AMEClubPayment"] = 4398] = "AMEClubPayment";
    EMsg[EMsg["AMEClubPaymentResponse"] = 4399] = "AMEClubPaymentResponse";
    EMsg[EMsg["AMPayPalPaymentsHubPayment"] = 4400] = "AMPayPalPaymentsHubPayment";
    EMsg[EMsg["AMPayPalPaymentsHubPaymentResponse"] = 4401] = "AMPayPalPaymentsHubPaymentResponse";
    EMsg[EMsg["AMTwoFactorRecoverAuthenticatorRequest"] = 4402] = "AMTwoFactorRecoverAuthenticatorRequest";
    EMsg[EMsg["AMTwoFactorRecoverAuthenticatorResponse"] = 4403] = "AMTwoFactorRecoverAuthenticatorResponse";
    EMsg[EMsg["AMSmart2PayPayment"] = 4404] = "AMSmart2PayPayment";
    EMsg[EMsg["AMSmart2PayPaymentResponse"] = 4405] = "AMSmart2PayPaymentResponse";
    EMsg[EMsg["AMValidatePasswordResetCodeAndSendSmsRequest"] = 4406] = "AMValidatePasswordResetCodeAndSendSmsRequest";
    EMsg[EMsg["AMValidatePasswordResetCodeAndSendSmsResponse"] = 4407] = "AMValidatePasswordResetCodeAndSendSmsResponse";
    EMsg[EMsg["AMGetAccountResetDetailsRequest"] = 4408] = "AMGetAccountResetDetailsRequest";
    EMsg[EMsg["AMGetAccountResetDetailsResponse"] = 4409] = "AMGetAccountResetDetailsResponse";
    EMsg[EMsg["AMBitPayPayment"] = 4410] = "AMBitPayPayment";
    EMsg[EMsg["AMBitPayPaymentResponse"] = 4411] = "AMBitPayPaymentResponse";
    EMsg[EMsg["AMSendAccountInfoUpdate"] = 4412] = "AMSendAccountInfoUpdate";
    EMsg[EMsg["AMSendScheduledGift"] = 4413] = "AMSendScheduledGift";
    EMsg[EMsg["AMNodwinPayment"] = 4414] = "AMNodwinPayment";
    EMsg[EMsg["AMNodwinPaymentResponse"] = 4415] = "AMNodwinPaymentResponse";
    EMsg[EMsg["AMResolveWalletRevoke"] = 4416] = "AMResolveWalletRevoke";
    EMsg[EMsg["AMResolveWalletReverseRevoke"] = 4417] = "AMResolveWalletReverseRevoke";
    EMsg[EMsg["AMFundedPayment"] = 4418] = "AMFundedPayment";
    EMsg[EMsg["AMFundedPaymentResponse"] = 4419] = "AMFundedPaymentResponse";
    EMsg[EMsg["AMRequestPersonaUpdateForChatServer"] = 4420] = "AMRequestPersonaUpdateForChatServer";
    EMsg[EMsg["AMPerfectWorldPayment"] = 4421] = "AMPerfectWorldPayment";
    EMsg[EMsg["AMPerfectWorldPaymentResponse"] = 4422] = "AMPerfectWorldPaymentResponse";
    EMsg[EMsg["AMECommPayPayment"] = 4423] = "AMECommPayPayment";
    EMsg[EMsg["AMECommPayPaymentResponse"] = 4424] = "AMECommPayPaymentResponse";
    EMsg[EMsg["AMSetRemoteClientID"] = 4425] = "AMSetRemoteClientID";
    EMsg[EMsg["BasePSRange"] = 5000] = "BasePSRange";
    EMsg[EMsg["PSCreateShoppingCart"] = 5001] = "PSCreateShoppingCart";
    EMsg[EMsg["PSCreateShoppingCartResponse"] = 5002] = "PSCreateShoppingCartResponse";
    EMsg[EMsg["PSIsValidShoppingCart"] = 5003] = "PSIsValidShoppingCart";
    EMsg[EMsg["PSIsValidShoppingCartResponse"] = 5004] = "PSIsValidShoppingCartResponse";
    EMsg[EMsg["PSAddPackageToShoppingCart"] = 5005] = "PSAddPackageToShoppingCart";
    EMsg[EMsg["PSAddPackageToShoppingCartResponse"] = 5006] = "PSAddPackageToShoppingCartResponse";
    EMsg[EMsg["PSRemoveLineItemFromShoppingCart"] = 5007] = "PSRemoveLineItemFromShoppingCart";
    EMsg[EMsg["PSRemoveLineItemFromShoppingCartResponse"] = 5008] = "PSRemoveLineItemFromShoppingCartResponse";
    EMsg[EMsg["PSGetShoppingCartContents"] = 5009] = "PSGetShoppingCartContents";
    EMsg[EMsg["PSGetShoppingCartContentsResponse"] = 5010] = "PSGetShoppingCartContentsResponse";
    EMsg[EMsg["PSAddWalletCreditToShoppingCart"] = 5011] = "PSAddWalletCreditToShoppingCart";
    EMsg[EMsg["PSAddWalletCreditToShoppingCartResponse"] = 5012] = "PSAddWalletCreditToShoppingCartResponse";
    EMsg[EMsg["BaseUFSRange"] = 5200] = "BaseUFSRange";
    EMsg[EMsg["ClientUFSUploadFileRequest"] = 5202] = "ClientUFSUploadFileRequest";
    EMsg[EMsg["ClientUFSUploadFileResponse"] = 5203] = "ClientUFSUploadFileResponse";
    EMsg[EMsg["ClientUFSUploadFileChunk"] = 5204] = "ClientUFSUploadFileChunk";
    EMsg[EMsg["ClientUFSUploadFileFinished"] = 5205] = "ClientUFSUploadFileFinished";
    EMsg[EMsg["ClientUFSGetFileListForApp"] = 5206] = "ClientUFSGetFileListForApp";
    EMsg[EMsg["ClientUFSGetFileListForAppResponse"] = 5207] = "ClientUFSGetFileListForAppResponse";
    EMsg[EMsg["ClientUFSDownloadRequest"] = 5210] = "ClientUFSDownloadRequest";
    EMsg[EMsg["ClientUFSDownloadResponse"] = 5211] = "ClientUFSDownloadResponse";
    EMsg[EMsg["ClientUFSDownloadChunk"] = 5212] = "ClientUFSDownloadChunk";
    EMsg[EMsg["ClientUFSLoginRequest"] = 5213] = "ClientUFSLoginRequest";
    EMsg[EMsg["ClientUFSLoginResponse"] = 5214] = "ClientUFSLoginResponse";
    EMsg[EMsg["UFSReloadPartitionInfo"] = 5215] = "UFSReloadPartitionInfo";
    EMsg[EMsg["ClientUFSTransferHeartbeat"] = 5216] = "ClientUFSTransferHeartbeat";
    EMsg[EMsg["UFSSynchronizeFile"] = 5217] = "UFSSynchronizeFile";
    EMsg[EMsg["UFSSynchronizeFileResponse"] = 5218] = "UFSSynchronizeFileResponse";
    EMsg[EMsg["ClientUFSDeleteFileRequest"] = 5219] = "ClientUFSDeleteFileRequest";
    EMsg[EMsg["ClientUFSDeleteFileResponse"] = 5220] = "ClientUFSDeleteFileResponse";
    EMsg[EMsg["UFSDownloadRequest"] = 5221] = "UFSDownloadRequest";
    EMsg[EMsg["UFSDownloadResponse"] = 5222] = "UFSDownloadResponse";
    EMsg[EMsg["UFSDownloadChunk"] = 5223] = "UFSDownloadChunk";
    EMsg[EMsg["ClientUFSGetUGCDetails"] = 5226] = "ClientUFSGetUGCDetails";
    EMsg[EMsg["ClientUFSGetUGCDetailsResponse"] = 5227] = "ClientUFSGetUGCDetailsResponse";
    EMsg[EMsg["UFSUpdateFileFlags"] = 5228] = "UFSUpdateFileFlags";
    EMsg[EMsg["UFSUpdateFileFlagsResponse"] = 5229] = "UFSUpdateFileFlagsResponse";
    EMsg[EMsg["ClientUFSGetSingleFileInfo"] = 5230] = "ClientUFSGetSingleFileInfo";
    EMsg[EMsg["ClientUFSGetSingleFileInfoResponse"] = 5231] = "ClientUFSGetSingleFileInfoResponse";
    EMsg[EMsg["ClientUFSShareFile"] = 5232] = "ClientUFSShareFile";
    EMsg[EMsg["ClientUFSShareFileResponse"] = 5233] = "ClientUFSShareFileResponse";
    EMsg[EMsg["UFSReloadAccount"] = 5234] = "UFSReloadAccount";
    EMsg[EMsg["UFSReloadAccountResponse"] = 5235] = "UFSReloadAccountResponse";
    EMsg[EMsg["UFSUpdateRecordBatched"] = 5236] = "UFSUpdateRecordBatched";
    EMsg[EMsg["UFSUpdateRecordBatchedResponse"] = 5237] = "UFSUpdateRecordBatchedResponse";
    EMsg[EMsg["UFSMigrateFile"] = 5238] = "UFSMigrateFile";
    EMsg[EMsg["UFSMigrateFileResponse"] = 5239] = "UFSMigrateFileResponse";
    EMsg[EMsg["UFSGetUGCURLs"] = 5240] = "UFSGetUGCURLs";
    EMsg[EMsg["UFSGetUGCURLsResponse"] = 5241] = "UFSGetUGCURLsResponse";
    EMsg[EMsg["UFSHttpUploadFileFinishRequest"] = 5242] = "UFSHttpUploadFileFinishRequest";
    EMsg[EMsg["UFSHttpUploadFileFinishResponse"] = 5243] = "UFSHttpUploadFileFinishResponse";
    EMsg[EMsg["UFSDownloadStartRequest"] = 5244] = "UFSDownloadStartRequest";
    EMsg[EMsg["UFSDownloadStartResponse"] = 5245] = "UFSDownloadStartResponse";
    EMsg[EMsg["UFSDownloadChunkRequest"] = 5246] = "UFSDownloadChunkRequest";
    EMsg[EMsg["UFSDownloadChunkResponse"] = 5247] = "UFSDownloadChunkResponse";
    EMsg[EMsg["UFSDownloadFinishRequest"] = 5248] = "UFSDownloadFinishRequest";
    EMsg[EMsg["UFSDownloadFinishResponse"] = 5249] = "UFSDownloadFinishResponse";
    EMsg[EMsg["UFSFlushURLCache"] = 5250] = "UFSFlushURLCache";
    EMsg[EMsg["UFSUploadCommit"] = 5251] = "UFSUploadCommit";
    EMsg[EMsg["ClientUFSUploadCommit"] = 5251] = "ClientUFSUploadCommit";
    EMsg[EMsg["UFSUploadCommitResponse"] = 5252] = "UFSUploadCommitResponse";
    EMsg[EMsg["ClientUFSUploadCommitResponse"] = 5252] = "ClientUFSUploadCommitResponse";
    EMsg[EMsg["UFSMigrateFileAppID"] = 5253] = "UFSMigrateFileAppID";
    EMsg[EMsg["UFSMigrateFileAppIDResponse"] = 5254] = "UFSMigrateFileAppIDResponse";
    EMsg[EMsg["BaseClient2"] = 5400] = "BaseClient2";
    EMsg[EMsg["ClientRequestForgottenPasswordEmail"] = 5401] = "ClientRequestForgottenPasswordEmail";
    EMsg[EMsg["ClientRequestForgottenPasswordEmailResponse"] = 5402] = "ClientRequestForgottenPasswordEmailResponse";
    EMsg[EMsg["ClientCreateAccountResponse"] = 5403] = "ClientCreateAccountResponse";
    EMsg[EMsg["ClientResetForgottenPassword"] = 5404] = "ClientResetForgottenPassword";
    EMsg[EMsg["ClientResetForgottenPasswordResponse"] = 5405] = "ClientResetForgottenPasswordResponse";
    EMsg[EMsg["ClientCreateAccount2"] = 5406] = "ClientCreateAccount2";
    EMsg[EMsg["ClientInformOfResetForgottenPassword"] = 5407] = "ClientInformOfResetForgottenPassword";
    EMsg[EMsg["ClientInformOfResetForgottenPasswordResponse"] = 5408] = "ClientInformOfResetForgottenPasswordResponse";
    EMsg[EMsg["ClientAnonUserLogOn_Deprecated"] = 5409] = "ClientAnonUserLogOn_Deprecated";
    EMsg[EMsg["ClientGamesPlayedWithDataBlob"] = 5410] = "ClientGamesPlayedWithDataBlob";
    EMsg[EMsg["ClientUpdateUserGameInfo"] = 5411] = "ClientUpdateUserGameInfo";
    EMsg[EMsg["ClientFileToDownload"] = 5412] = "ClientFileToDownload";
    EMsg[EMsg["ClientFileToDownloadResponse"] = 5413] = "ClientFileToDownloadResponse";
    EMsg[EMsg["ClientLBSSetScore"] = 5414] = "ClientLBSSetScore";
    EMsg[EMsg["ClientLBSSetScoreResponse"] = 5415] = "ClientLBSSetScoreResponse";
    EMsg[EMsg["ClientLBSFindOrCreateLB"] = 5416] = "ClientLBSFindOrCreateLB";
    EMsg[EMsg["ClientLBSFindOrCreateLBResponse"] = 5417] = "ClientLBSFindOrCreateLBResponse";
    EMsg[EMsg["ClientLBSGetLBEntries"] = 5418] = "ClientLBSGetLBEntries";
    EMsg[EMsg["ClientLBSGetLBEntriesResponse"] = 5419] = "ClientLBSGetLBEntriesResponse";
    EMsg[EMsg["ClientMarketingMessageUpdate"] = 5420] = "ClientMarketingMessageUpdate";
    EMsg[EMsg["ClientChatDeclined"] = 5426] = "ClientChatDeclined";
    EMsg[EMsg["ClientFriendMsgIncoming"] = 5427] = "ClientFriendMsgIncoming";
    EMsg[EMsg["ClientAuthList_Deprecated"] = 5428] = "ClientAuthList_Deprecated";
    EMsg[EMsg["ClientTicketAuthComplete"] = 5429] = "ClientTicketAuthComplete";
    EMsg[EMsg["ClientIsLimitedAccount"] = 5430] = "ClientIsLimitedAccount";
    EMsg[EMsg["ClientRequestAuthList"] = 5431] = "ClientRequestAuthList";
    EMsg[EMsg["ClientAuthList"] = 5432] = "ClientAuthList";
    EMsg[EMsg["ClientStat"] = 5433] = "ClientStat";
    EMsg[EMsg["ClientP2PConnectionInfo"] = 5434] = "ClientP2PConnectionInfo";
    EMsg[EMsg["ClientP2PConnectionFailInfo"] = 5435] = "ClientP2PConnectionFailInfo";
    EMsg[EMsg["ClientGetNumberOfCurrentPlayers"] = 5436] = "ClientGetNumberOfCurrentPlayers";
    EMsg[EMsg["ClientGetNumberOfCurrentPlayersResponse"] = 5437] = "ClientGetNumberOfCurrentPlayersResponse";
    EMsg[EMsg["ClientGetDepotDecryptionKey"] = 5438] = "ClientGetDepotDecryptionKey";
    EMsg[EMsg["ClientGetDepotDecryptionKeyResponse"] = 5439] = "ClientGetDepotDecryptionKeyResponse";
    EMsg[EMsg["GSPerformHardwareSurvey"] = 5440] = "GSPerformHardwareSurvey";
    EMsg[EMsg["ClientGetAppBetaPasswords"] = 5441] = "ClientGetAppBetaPasswords";
    EMsg[EMsg["ClientGetAppBetaPasswordsResponse"] = 5442] = "ClientGetAppBetaPasswordsResponse";
    EMsg[EMsg["ClientEnableTestLicense"] = 5443] = "ClientEnableTestLicense";
    EMsg[EMsg["ClientEnableTestLicenseResponse"] = 5444] = "ClientEnableTestLicenseResponse";
    EMsg[EMsg["ClientDisableTestLicense"] = 5445] = "ClientDisableTestLicense";
    EMsg[EMsg["ClientDisableTestLicenseResponse"] = 5446] = "ClientDisableTestLicenseResponse";
    EMsg[EMsg["ClientRequestValidationMail"] = 5448] = "ClientRequestValidationMail";
    EMsg[EMsg["ClientRequestValidationMailResponse"] = 5449] = "ClientRequestValidationMailResponse";
    EMsg[EMsg["ClientCheckAppBetaPassword"] = 5450] = "ClientCheckAppBetaPassword";
    EMsg[EMsg["ClientCheckAppBetaPasswordResponse"] = 5451] = "ClientCheckAppBetaPasswordResponse";
    EMsg[EMsg["ClientToGC"] = 5452] = "ClientToGC";
    EMsg[EMsg["ClientFromGC"] = 5453] = "ClientFromGC";
    EMsg[EMsg["ClientRequestChangeMail"] = 5454] = "ClientRequestChangeMail";
    EMsg[EMsg["ClientRequestChangeMailResponse"] = 5455] = "ClientRequestChangeMailResponse";
    EMsg[EMsg["ClientEmailAddrInfo"] = 5456] = "ClientEmailAddrInfo";
    EMsg[EMsg["ClientPasswordChange3"] = 5457] = "ClientPasswordChange3";
    EMsg[EMsg["ClientEmailChange3"] = 5458] = "ClientEmailChange3";
    EMsg[EMsg["ClientPersonalQAChange3"] = 5459] = "ClientPersonalQAChange3";
    EMsg[EMsg["ClientResetForgottenPassword3"] = 5460] = "ClientResetForgottenPassword3";
    EMsg[EMsg["ClientRequestForgottenPasswordEmail3"] = 5461] = "ClientRequestForgottenPasswordEmail3";
    EMsg[EMsg["ClientCreateAccount3"] = 5462] = "ClientCreateAccount3";
    EMsg[EMsg["ClientNewLoginKey"] = 5463] = "ClientNewLoginKey";
    EMsg[EMsg["ClientNewLoginKeyAccepted"] = 5464] = "ClientNewLoginKeyAccepted";
    EMsg[EMsg["ClientLogOnWithHash_Deprecated"] = 5465] = "ClientLogOnWithHash_Deprecated";
    EMsg[EMsg["ClientStoreUserStats2"] = 5466] = "ClientStoreUserStats2";
    EMsg[EMsg["ClientStatsUpdated"] = 5467] = "ClientStatsUpdated";
    EMsg[EMsg["ClientActivateOEMLicense"] = 5468] = "ClientActivateOEMLicense";
    EMsg[EMsg["ClientRegisterOEMMachine"] = 5469] = "ClientRegisterOEMMachine";
    EMsg[EMsg["ClientRegisterOEMMachineResponse"] = 5470] = "ClientRegisterOEMMachineResponse";
    EMsg[EMsg["ClientRequestedClientStats"] = 5480] = "ClientRequestedClientStats";
    EMsg[EMsg["ClientStat2Int32"] = 5481] = "ClientStat2Int32";
    EMsg[EMsg["ClientStat2"] = 5482] = "ClientStat2";
    EMsg[EMsg["ClientVerifyPassword"] = 5483] = "ClientVerifyPassword";
    EMsg[EMsg["ClientVerifyPasswordResponse"] = 5484] = "ClientVerifyPasswordResponse";
    EMsg[EMsg["ClientDRMDownloadRequest"] = 5485] = "ClientDRMDownloadRequest";
    EMsg[EMsg["ClientDRMDownloadResponse"] = 5486] = "ClientDRMDownloadResponse";
    EMsg[EMsg["ClientDRMFinalResult"] = 5487] = "ClientDRMFinalResult";
    EMsg[EMsg["ClientGetFriendsWhoPlayGame"] = 5488] = "ClientGetFriendsWhoPlayGame";
    EMsg[EMsg["ClientGetFriendsWhoPlayGameResponse"] = 5489] = "ClientGetFriendsWhoPlayGameResponse";
    EMsg[EMsg["ClientOGSBeginSession"] = 5490] = "ClientOGSBeginSession";
    EMsg[EMsg["ClientOGSBeginSessionResponse"] = 5491] = "ClientOGSBeginSessionResponse";
    EMsg[EMsg["ClientOGSEndSession"] = 5492] = "ClientOGSEndSession";
    EMsg[EMsg["ClientOGSEndSessionResponse"] = 5493] = "ClientOGSEndSessionResponse";
    EMsg[EMsg["ClientOGSWriteRow"] = 5494] = "ClientOGSWriteRow";
    EMsg[EMsg["ClientDRMTest"] = 5495] = "ClientDRMTest";
    EMsg[EMsg["ClientDRMTestResult"] = 5496] = "ClientDRMTestResult";
    EMsg[EMsg["ClientStartPeerContentServer"] = 5497] = "ClientStartPeerContentServer";
    EMsg[EMsg["ClientStartPeerContentServerResponse"] = 5498] = "ClientStartPeerContentServerResponse";
    EMsg[EMsg["ClientServerUnavailable"] = 5500] = "ClientServerUnavailable";
    EMsg[EMsg["ClientServersAvailable"] = 5501] = "ClientServersAvailable";
    EMsg[EMsg["ClientRegisterAuthTicketWithCM"] = 5502] = "ClientRegisterAuthTicketWithCM";
    EMsg[EMsg["ClientGCMsgFailed"] = 5503] = "ClientGCMsgFailed";
    EMsg[EMsg["ClientMicroTxnAuthRequest"] = 5504] = "ClientMicroTxnAuthRequest";
    EMsg[EMsg["ClientMicroTxnAuthorize"] = 5505] = "ClientMicroTxnAuthorize";
    EMsg[EMsg["ClientMicroTxnAuthorizeResponse"] = 5506] = "ClientMicroTxnAuthorizeResponse";
    EMsg[EMsg["ClientAppMinutesPlayedData"] = 5507] = "ClientAppMinutesPlayedData";
    EMsg[EMsg["ClientGetMicroTxnInfo"] = 5508] = "ClientGetMicroTxnInfo";
    EMsg[EMsg["ClientGetMicroTxnInfoResponse"] = 5509] = "ClientGetMicroTxnInfoResponse";
    EMsg[EMsg["ClientMarketingMessageUpdate2"] = 5510] = "ClientMarketingMessageUpdate2";
    EMsg[EMsg["ClientDeregisterWithServer"] = 5511] = "ClientDeregisterWithServer";
    EMsg[EMsg["ClientSubscribeToPersonaFeed"] = 5512] = "ClientSubscribeToPersonaFeed";
    EMsg[EMsg["ClientLogon"] = 5514] = "ClientLogon";
    EMsg[EMsg["ClientGetClientDetails"] = 5515] = "ClientGetClientDetails";
    EMsg[EMsg["ClientGetClientDetailsResponse"] = 5516] = "ClientGetClientDetailsResponse";
    EMsg[EMsg["ClientReportOverlayDetourFailure"] = 5517] = "ClientReportOverlayDetourFailure";
    EMsg[EMsg["ClientGetClientAppList"] = 5518] = "ClientGetClientAppList";
    EMsg[EMsg["ClientGetClientAppListResponse"] = 5519] = "ClientGetClientAppListResponse";
    EMsg[EMsg["ClientInstallClientApp"] = 5520] = "ClientInstallClientApp";
    EMsg[EMsg["ClientInstallClientAppResponse"] = 5521] = "ClientInstallClientAppResponse";
    EMsg[EMsg["ClientUninstallClientApp"] = 5522] = "ClientUninstallClientApp";
    EMsg[EMsg["ClientUninstallClientAppResponse"] = 5523] = "ClientUninstallClientAppResponse";
    EMsg[EMsg["ClientSetClientAppUpdateState"] = 5524] = "ClientSetClientAppUpdateState";
    EMsg[EMsg["ClientSetClientAppUpdateStateResponse"] = 5525] = "ClientSetClientAppUpdateStateResponse";
    EMsg[EMsg["ClientRequestEncryptedAppTicket"] = 5526] = "ClientRequestEncryptedAppTicket";
    EMsg[EMsg["ClientRequestEncryptedAppTicketResponse"] = 5527] = "ClientRequestEncryptedAppTicketResponse";
    EMsg[EMsg["ClientWalletInfoUpdate"] = 5528] = "ClientWalletInfoUpdate";
    EMsg[EMsg["ClientLBSSetUGC"] = 5529] = "ClientLBSSetUGC";
    EMsg[EMsg["ClientLBSSetUGCResponse"] = 5530] = "ClientLBSSetUGCResponse";
    EMsg[EMsg["ClientAMGetClanOfficers"] = 5531] = "ClientAMGetClanOfficers";
    EMsg[EMsg["ClientAMGetClanOfficersResponse"] = 5532] = "ClientAMGetClanOfficersResponse";
    EMsg[EMsg["ClientCheckFileSignature"] = 5533] = "ClientCheckFileSignature";
    EMsg[EMsg["ClientCheckFileSignatureResponse"] = 5534] = "ClientCheckFileSignatureResponse";
    EMsg[EMsg["ClientFriendProfileInfo"] = 5535] = "ClientFriendProfileInfo";
    EMsg[EMsg["ClientFriendProfileInfoResponse"] = 5536] = "ClientFriendProfileInfoResponse";
    EMsg[EMsg["ClientUpdateMachineAuth"] = 5537] = "ClientUpdateMachineAuth";
    EMsg[EMsg["ClientUpdateMachineAuthResponse"] = 5538] = "ClientUpdateMachineAuthResponse";
    EMsg[EMsg["ClientReadMachineAuth"] = 5539] = "ClientReadMachineAuth";
    EMsg[EMsg["ClientReadMachineAuthResponse"] = 5540] = "ClientReadMachineAuthResponse";
    EMsg[EMsg["ClientRequestMachineAuth"] = 5541] = "ClientRequestMachineAuth";
    EMsg[EMsg["ClientRequestMachineAuthResponse"] = 5542] = "ClientRequestMachineAuthResponse";
    EMsg[EMsg["ClientScreenshotsChanged"] = 5543] = "ClientScreenshotsChanged";
    EMsg[EMsg["ClientEmailChange4"] = 5544] = "ClientEmailChange4";
    EMsg[EMsg["ClientEmailChangeResponse4"] = 5545] = "ClientEmailChangeResponse4";
    EMsg[EMsg["ClientGetCDNAuthToken"] = 5546] = "ClientGetCDNAuthToken";
    EMsg[EMsg["ClientGetCDNAuthTokenResponse"] = 5547] = "ClientGetCDNAuthTokenResponse";
    EMsg[EMsg["ClientDownloadRateStatistics"] = 5548] = "ClientDownloadRateStatistics";
    EMsg[EMsg["ClientRequestAccountData"] = 5549] = "ClientRequestAccountData";
    EMsg[EMsg["ClientRequestAccountDataResponse"] = 5550] = "ClientRequestAccountDataResponse";
    EMsg[EMsg["ClientResetForgottenPassword4"] = 5551] = "ClientResetForgottenPassword4";
    EMsg[EMsg["ClientHideFriend"] = 5552] = "ClientHideFriend";
    EMsg[EMsg["ClientFriendsGroupsList"] = 5553] = "ClientFriendsGroupsList";
    EMsg[EMsg["ClientGetClanActivityCounts"] = 5554] = "ClientGetClanActivityCounts";
    EMsg[EMsg["ClientGetClanActivityCountsResponse"] = 5555] = "ClientGetClanActivityCountsResponse";
    EMsg[EMsg["ClientOGSReportString"] = 5556] = "ClientOGSReportString";
    EMsg[EMsg["ClientOGSReportBug"] = 5557] = "ClientOGSReportBug";
    EMsg[EMsg["ClientSentLogs"] = 5558] = "ClientSentLogs";
    EMsg[EMsg["ClientLogonGameServer"] = 5559] = "ClientLogonGameServer";
    EMsg[EMsg["AMClientCreateFriendsGroup"] = 5560] = "AMClientCreateFriendsGroup";
    EMsg[EMsg["AMClientCreateFriendsGroupResponse"] = 5561] = "AMClientCreateFriendsGroupResponse";
    EMsg[EMsg["AMClientDeleteFriendsGroup"] = 5562] = "AMClientDeleteFriendsGroup";
    EMsg[EMsg["AMClientDeleteFriendsGroupResponse"] = 5563] = "AMClientDeleteFriendsGroupResponse";
    EMsg[EMsg["AMClientRenameFriendsGroup"] = 5564] = "AMClientRenameFriendsGroup";
    EMsg[EMsg["AMClientManageFriendsGroup"] = 5564] = "AMClientManageFriendsGroup";
    EMsg[EMsg["AMClientRenameFriendsGroupResponse"] = 5565] = "AMClientRenameFriendsGroupResponse";
    EMsg[EMsg["AMClientManageFriendsGroupResponse"] = 5565] = "AMClientManageFriendsGroupResponse";
    EMsg[EMsg["AMClientAddFriendToGroup"] = 5566] = "AMClientAddFriendToGroup";
    EMsg[EMsg["AMClientAddFriendToGroupResponse"] = 5567] = "AMClientAddFriendToGroupResponse";
    EMsg[EMsg["AMClientRemoveFriendFromGroup"] = 5568] = "AMClientRemoveFriendFromGroup";
    EMsg[EMsg["AMClientRemoveFriendFromGroupResponse"] = 5569] = "AMClientRemoveFriendFromGroupResponse";
    EMsg[EMsg["ClientAMGetPersonaNameHistory"] = 5570] = "ClientAMGetPersonaNameHistory";
    EMsg[EMsg["ClientAMGetPersonaNameHistoryResponse"] = 5571] = "ClientAMGetPersonaNameHistoryResponse";
    EMsg[EMsg["ClientRequestFreeLicense"] = 5572] = "ClientRequestFreeLicense";
    EMsg[EMsg["ClientRequestFreeLicenseResponse"] = 5573] = "ClientRequestFreeLicenseResponse";
    EMsg[EMsg["ClientDRMDownloadRequestWithCrashData"] = 5574] = "ClientDRMDownloadRequestWithCrashData";
    EMsg[EMsg["ClientAuthListAck"] = 5575] = "ClientAuthListAck";
    EMsg[EMsg["ClientItemAnnouncements"] = 5576] = "ClientItemAnnouncements";
    EMsg[EMsg["ClientRequestItemAnnouncements"] = 5577] = "ClientRequestItemAnnouncements";
    EMsg[EMsg["ClientFriendMsgEchoToSender"] = 5578] = "ClientFriendMsgEchoToSender";
    EMsg[EMsg["ClientChangeSteamGuardOptions"] = 5579] = "ClientChangeSteamGuardOptions";
    EMsg[EMsg["ClientChangeSteamGuardOptionsResponse"] = 5580] = "ClientChangeSteamGuardOptionsResponse";
    EMsg[EMsg["ClientOGSGameServerPingSample"] = 5581] = "ClientOGSGameServerPingSample";
    EMsg[EMsg["ClientCommentNotifications"] = 5582] = "ClientCommentNotifications";
    EMsg[EMsg["ClientRequestCommentNotifications"] = 5583] = "ClientRequestCommentNotifications";
    EMsg[EMsg["ClientPersonaChangeResponse"] = 5584] = "ClientPersonaChangeResponse";
    EMsg[EMsg["ClientRequestWebAPIAuthenticateUserNonce"] = 5585] = "ClientRequestWebAPIAuthenticateUserNonce";
    EMsg[EMsg["ClientRequestWebAPIAuthenticateUserNonceResponse"] = 5586] = "ClientRequestWebAPIAuthenticateUserNonceResponse";
    EMsg[EMsg["ClientPlayerNicknameList"] = 5587] = "ClientPlayerNicknameList";
    EMsg[EMsg["AMClientSetPlayerNickname"] = 5588] = "AMClientSetPlayerNickname";
    EMsg[EMsg["AMClientSetPlayerNicknameResponse"] = 5589] = "AMClientSetPlayerNicknameResponse";
    EMsg[EMsg["ClientCreateAccountProto"] = 5590] = "ClientCreateAccountProto";
    EMsg[EMsg["ClientRequestOAuthTokenForApp"] = 5590] = "ClientRequestOAuthTokenForApp";
    EMsg[EMsg["ClientCreateAccountProtoResponse"] = 5591] = "ClientCreateAccountProtoResponse";
    EMsg[EMsg["ClientRequestOAuthTokenForAppResponse"] = 5591] = "ClientRequestOAuthTokenForAppResponse";
    EMsg[EMsg["ClientGetNumberOfCurrentPlayersDP"] = 5592] = "ClientGetNumberOfCurrentPlayersDP";
    EMsg[EMsg["ClientGetNumberOfCurrentPlayersDPResponse"] = 5593] = "ClientGetNumberOfCurrentPlayersDPResponse";
    EMsg[EMsg["ClientServiceMethod"] = 5594] = "ClientServiceMethod";
    EMsg[EMsg["ClientServiceMethodLegacy"] = 5594] = "ClientServiceMethodLegacy";
    EMsg[EMsg["ClientServiceMethodResponse"] = 5595] = "ClientServiceMethodResponse";
    EMsg[EMsg["ClientServiceMethodLegacyResponse"] = 5595] = "ClientServiceMethodLegacyResponse";
    EMsg[EMsg["ClientFriendUserStatusPublished"] = 5596] = "ClientFriendUserStatusPublished";
    EMsg[EMsg["ClientCurrentUIMode"] = 5597] = "ClientCurrentUIMode";
    EMsg[EMsg["ClientVanityURLChangedNotification"] = 5598] = "ClientVanityURLChangedNotification";
    EMsg[EMsg["ClientUserNotifications"] = 5599] = "ClientUserNotifications";
    EMsg[EMsg["BaseDFS"] = 5600] = "BaseDFS";
    EMsg[EMsg["DFSGetFile"] = 5601] = "DFSGetFile";
    EMsg[EMsg["DFSInstallLocalFile"] = 5602] = "DFSInstallLocalFile";
    EMsg[EMsg["DFSConnection"] = 5603] = "DFSConnection";
    EMsg[EMsg["DFSConnectionReply"] = 5604] = "DFSConnectionReply";
    EMsg[EMsg["ClientDFSAuthenticateRequest"] = 5605] = "ClientDFSAuthenticateRequest";
    EMsg[EMsg["ClientDFSAuthenticateResponse"] = 5606] = "ClientDFSAuthenticateResponse";
    EMsg[EMsg["ClientDFSEndSession"] = 5607] = "ClientDFSEndSession";
    EMsg[EMsg["DFSPurgeFile"] = 5608] = "DFSPurgeFile";
    EMsg[EMsg["DFSRouteFile"] = 5609] = "DFSRouteFile";
    EMsg[EMsg["DFSGetFileFromServer"] = 5610] = "DFSGetFileFromServer";
    EMsg[EMsg["DFSAcceptedResponse"] = 5611] = "DFSAcceptedResponse";
    EMsg[EMsg["DFSRequestPingback"] = 5612] = "DFSRequestPingback";
    EMsg[EMsg["DFSRecvTransmitFile"] = 5613] = "DFSRecvTransmitFile";
    EMsg[EMsg["DFSSendTransmitFile"] = 5614] = "DFSSendTransmitFile";
    EMsg[EMsg["DFSRequestPingback2"] = 5615] = "DFSRequestPingback2";
    EMsg[EMsg["DFSResponsePingback2"] = 5616] = "DFSResponsePingback2";
    EMsg[EMsg["ClientDFSDownloadStatus"] = 5617] = "ClientDFSDownloadStatus";
    EMsg[EMsg["DFSStartTransfer"] = 5618] = "DFSStartTransfer";
    EMsg[EMsg["DFSTransferComplete"] = 5619] = "DFSTransferComplete";
    EMsg[EMsg["DFSRouteFileResponse"] = 5620] = "DFSRouteFileResponse";
    EMsg[EMsg["ClientNetworkingCertRequest"] = 5621] = "ClientNetworkingCertRequest";
    EMsg[EMsg["ClientNetworkingCertRequestResponse"] = 5622] = "ClientNetworkingCertRequestResponse";
    EMsg[EMsg["ClientChallengeRequest"] = 5623] = "ClientChallengeRequest";
    EMsg[EMsg["ClientChallengeResponse"] = 5624] = "ClientChallengeResponse";
    EMsg[EMsg["BadgeCraftedNotification"] = 5625] = "BadgeCraftedNotification";
    EMsg[EMsg["ClientNetworkingMobileCertRequest"] = 5626] = "ClientNetworkingMobileCertRequest";
    EMsg[EMsg["ClientNetworkingMobileCertRequestResponse"] = 5627] = "ClientNetworkingMobileCertRequestResponse";
    EMsg[EMsg["BaseMDS"] = 5800] = "BaseMDS";
    EMsg[EMsg["ClientMDSLoginRequest"] = 5801] = "ClientMDSLoginRequest";
    EMsg[EMsg["ClientMDSLoginResponse"] = 5802] = "ClientMDSLoginResponse";
    EMsg[EMsg["ClientMDSUploadManifestRequest"] = 5803] = "ClientMDSUploadManifestRequest";
    EMsg[EMsg["ClientMDSUploadManifestResponse"] = 5804] = "ClientMDSUploadManifestResponse";
    EMsg[EMsg["ClientMDSTransmitManifestDataChunk"] = 5805] = "ClientMDSTransmitManifestDataChunk";
    EMsg[EMsg["ClientMDSHeartbeat"] = 5806] = "ClientMDSHeartbeat";
    EMsg[EMsg["ClientMDSUploadDepotChunks"] = 5807] = "ClientMDSUploadDepotChunks";
    EMsg[EMsg["ClientMDSUploadDepotChunksResponse"] = 5808] = "ClientMDSUploadDepotChunksResponse";
    EMsg[EMsg["ClientMDSInitDepotBuildRequest"] = 5809] = "ClientMDSInitDepotBuildRequest";
    EMsg[EMsg["ClientMDSInitDepotBuildResponse"] = 5810] = "ClientMDSInitDepotBuildResponse";
    EMsg[EMsg["AMToMDSGetDepotDecryptionKey"] = 5812] = "AMToMDSGetDepotDecryptionKey";
    EMsg[EMsg["MDSToAMGetDepotDecryptionKeyResponse"] = 5813] = "MDSToAMGetDepotDecryptionKeyResponse";
    EMsg[EMsg["MDSGetVersionsForDepot"] = 5814] = "MDSGetVersionsForDepot";
    EMsg[EMsg["MDSGetVersionsForDepotResponse"] = 5815] = "MDSGetVersionsForDepotResponse";
    EMsg[EMsg["ClientMDSInitWorkshopBuildRequest"] = 5816] = "ClientMDSInitWorkshopBuildRequest";
    EMsg[EMsg["MDSSetPublicVersionForDepot"] = 5816] = "MDSSetPublicVersionForDepot";
    EMsg[EMsg["ClientMDSInitWorkshopBuildResponse"] = 5817] = "ClientMDSInitWorkshopBuildResponse";
    EMsg[EMsg["MDSSetPublicVersionForDepotResponse"] = 5817] = "MDSSetPublicVersionForDepotResponse";
    EMsg[EMsg["ClientMDSGetDepotManifest"] = 5818] = "ClientMDSGetDepotManifest";
    EMsg[EMsg["ClientMDSGetDepotManifestResponse"] = 5819] = "ClientMDSGetDepotManifestResponse";
    EMsg[EMsg["ClientMDSGetDepotManifestChunk"] = 5820] = "ClientMDSGetDepotManifestChunk";
    EMsg[EMsg["ClientMDSUploadRateTest"] = 5823] = "ClientMDSUploadRateTest";
    EMsg[EMsg["ClientMDSUploadRateTestResponse"] = 5824] = "ClientMDSUploadRateTestResponse";
    EMsg[EMsg["MDSDownloadDepotChunksAck"] = 5825] = "MDSDownloadDepotChunksAck";
    EMsg[EMsg["MDSContentServerStatsBroadcast"] = 5826] = "MDSContentServerStatsBroadcast";
    EMsg[EMsg["MDSContentServerConfigRequest"] = 5827] = "MDSContentServerConfigRequest";
    EMsg[EMsg["MDSContentServerConfig"] = 5828] = "MDSContentServerConfig";
    EMsg[EMsg["MDSGetDepotManifest"] = 5829] = "MDSGetDepotManifest";
    EMsg[EMsg["MDSGetDepotManifestResponse"] = 5830] = "MDSGetDepotManifestResponse";
    EMsg[EMsg["MDSGetDepotManifestChunk"] = 5831] = "MDSGetDepotManifestChunk";
    EMsg[EMsg["MDSGetDepotChunk"] = 5832] = "MDSGetDepotChunk";
    EMsg[EMsg["MDSGetDepotChunkResponse"] = 5833] = "MDSGetDepotChunkResponse";
    EMsg[EMsg["MDSGetDepotChunkChunk"] = 5834] = "MDSGetDepotChunkChunk";
    EMsg[EMsg["MDSUpdateContentServerConfig"] = 5835] = "MDSUpdateContentServerConfig";
    EMsg[EMsg["MDSGetServerListForUser"] = 5836] = "MDSGetServerListForUser";
    EMsg[EMsg["MDSGetServerListForUserResponse"] = 5837] = "MDSGetServerListForUserResponse";
    EMsg[EMsg["ClientMDSRegisterAppBuild"] = 5838] = "ClientMDSRegisterAppBuild";
    EMsg[EMsg["ClientMDSRegisterAppBuildResponse"] = 5839] = "ClientMDSRegisterAppBuildResponse";
    EMsg[EMsg["ClientMDSSetAppBuildLive"] = 5840] = "ClientMDSSetAppBuildLive";
    EMsg[EMsg["ClientMDSSetAppBuildLiveResponse"] = 5841] = "ClientMDSSetAppBuildLiveResponse";
    EMsg[EMsg["ClientMDSGetPrevDepotBuild"] = 5842] = "ClientMDSGetPrevDepotBuild";
    EMsg[EMsg["ClientMDSGetPrevDepotBuildResponse"] = 5843] = "ClientMDSGetPrevDepotBuildResponse";
    EMsg[EMsg["MDSToCSFlushChunk"] = 5844] = "MDSToCSFlushChunk";
    EMsg[EMsg["ClientMDSSignInstallScript"] = 5845] = "ClientMDSSignInstallScript";
    EMsg[EMsg["ClientMDSSignInstallScriptResponse"] = 5846] = "ClientMDSSignInstallScriptResponse";
    EMsg[EMsg["MDSMigrateChunk"] = 5847] = "MDSMigrateChunk";
    EMsg[EMsg["MDSMigrateChunkResponse"] = 5848] = "MDSMigrateChunkResponse";
    EMsg[EMsg["MDSToCSFlushManifest"] = 5849] = "MDSToCSFlushManifest";
    EMsg[EMsg["CSBase"] = 6200] = "CSBase";
    EMsg[EMsg["CSPing"] = 6201] = "CSPing";
    EMsg[EMsg["CSPingResponse"] = 6202] = "CSPingResponse";
    EMsg[EMsg["GMSBase"] = 6400] = "GMSBase";
    EMsg[EMsg["GMSGameServerReplicate"] = 6401] = "GMSGameServerReplicate";
    EMsg[EMsg["ClientGMSServerQuery"] = 6403] = "ClientGMSServerQuery";
    EMsg[EMsg["GMSClientServerQueryResponse"] = 6404] = "GMSClientServerQueryResponse";
    EMsg[EMsg["AMGMSGameServerUpdate"] = 6405] = "AMGMSGameServerUpdate";
    EMsg[EMsg["AMGMSGameServerRemove"] = 6406] = "AMGMSGameServerRemove";
    EMsg[EMsg["GameServerOutOfDate"] = 6407] = "GameServerOutOfDate";
    EMsg[EMsg["DeviceAuthorizationBase"] = 6500] = "DeviceAuthorizationBase";
    EMsg[EMsg["ClientAuthorizeLocalDeviceRequest"] = 6501] = "ClientAuthorizeLocalDeviceRequest";
    EMsg[EMsg["ClientAuthorizeLocalDevice"] = 6502] = "ClientAuthorizeLocalDevice";
    EMsg[EMsg["ClientAuthorizeLocalDeviceResponse"] = 6502] = "ClientAuthorizeLocalDeviceResponse";
    EMsg[EMsg["ClientDeauthorizeDeviceRequest"] = 6503] = "ClientDeauthorizeDeviceRequest";
    EMsg[EMsg["ClientDeauthorizeDevice"] = 6504] = "ClientDeauthorizeDevice";
    EMsg[EMsg["ClientUseLocalDeviceAuthorizations"] = 6505] = "ClientUseLocalDeviceAuthorizations";
    EMsg[EMsg["ClientGetAuthorizedDevices"] = 6506] = "ClientGetAuthorizedDevices";
    EMsg[EMsg["ClientGetAuthorizedDevicesResponse"] = 6507] = "ClientGetAuthorizedDevicesResponse";
    EMsg[EMsg["AMNotifySessionDeviceAuthorized"] = 6508] = "AMNotifySessionDeviceAuthorized";
    EMsg[EMsg["ClientAuthorizeLocalDeviceNotification"] = 6509] = "ClientAuthorizeLocalDeviceNotification";
    EMsg[EMsg["MMSBase"] = 6600] = "MMSBase";
    EMsg[EMsg["ClientMMSCreateLobby"] = 6601] = "ClientMMSCreateLobby";
    EMsg[EMsg["ClientMMSCreateLobbyResponse"] = 6602] = "ClientMMSCreateLobbyResponse";
    EMsg[EMsg["ClientMMSJoinLobby"] = 6603] = "ClientMMSJoinLobby";
    EMsg[EMsg["ClientMMSJoinLobbyResponse"] = 6604] = "ClientMMSJoinLobbyResponse";
    EMsg[EMsg["ClientMMSLeaveLobby"] = 6605] = "ClientMMSLeaveLobby";
    EMsg[EMsg["ClientMMSLeaveLobbyResponse"] = 6606] = "ClientMMSLeaveLobbyResponse";
    EMsg[EMsg["ClientMMSGetLobbyList"] = 6607] = "ClientMMSGetLobbyList";
    EMsg[EMsg["ClientMMSGetLobbyListResponse"] = 6608] = "ClientMMSGetLobbyListResponse";
    EMsg[EMsg["ClientMMSSetLobbyData"] = 6609] = "ClientMMSSetLobbyData";
    EMsg[EMsg["ClientMMSSetLobbyDataResponse"] = 6610] = "ClientMMSSetLobbyDataResponse";
    EMsg[EMsg["ClientMMSGetLobbyData"] = 6611] = "ClientMMSGetLobbyData";
    EMsg[EMsg["ClientMMSLobbyData"] = 6612] = "ClientMMSLobbyData";
    EMsg[EMsg["ClientMMSSendLobbyChatMsg"] = 6613] = "ClientMMSSendLobbyChatMsg";
    EMsg[EMsg["ClientMMSLobbyChatMsg"] = 6614] = "ClientMMSLobbyChatMsg";
    EMsg[EMsg["ClientMMSSetLobbyOwner"] = 6615] = "ClientMMSSetLobbyOwner";
    EMsg[EMsg["ClientMMSSetLobbyOwnerResponse"] = 6616] = "ClientMMSSetLobbyOwnerResponse";
    EMsg[EMsg["ClientMMSSetLobbyGameServer"] = 6617] = "ClientMMSSetLobbyGameServer";
    EMsg[EMsg["ClientMMSLobbyGameServerSet"] = 6618] = "ClientMMSLobbyGameServerSet";
    EMsg[EMsg["ClientMMSUserJoinedLobby"] = 6619] = "ClientMMSUserJoinedLobby";
    EMsg[EMsg["ClientMMSUserLeftLobby"] = 6620] = "ClientMMSUserLeftLobby";
    EMsg[EMsg["ClientMMSInviteToLobby"] = 6621] = "ClientMMSInviteToLobby";
    EMsg[EMsg["ClientMMSFlushFrenemyListCache"] = 6622] = "ClientMMSFlushFrenemyListCache";
    EMsg[EMsg["ClientMMSFlushFrenemyListCacheResponse"] = 6623] = "ClientMMSFlushFrenemyListCacheResponse";
    EMsg[EMsg["ClientMMSSetLobbyLinked"] = 6624] = "ClientMMSSetLobbyLinked";
    EMsg[EMsg["ClientMMSSetRatelimitPolicyOnClient"] = 6625] = "ClientMMSSetRatelimitPolicyOnClient";
    EMsg[EMsg["ClientMMSGetLobbyStatus"] = 6626] = "ClientMMSGetLobbyStatus";
    EMsg[EMsg["ClientMMSGetLobbyStatusResponse"] = 6627] = "ClientMMSGetLobbyStatusResponse";
    EMsg[EMsg["MMSGetLobbyList"] = 6628] = "MMSGetLobbyList";
    EMsg[EMsg["MMSGetLobbyListResponse"] = 6629] = "MMSGetLobbyListResponse";
    EMsg[EMsg["NonStdMsgBase"] = 6800] = "NonStdMsgBase";
    EMsg[EMsg["NonStdMsgMemcached"] = 6801] = "NonStdMsgMemcached";
    EMsg[EMsg["NonStdMsgHTTPServer"] = 6802] = "NonStdMsgHTTPServer";
    EMsg[EMsg["NonStdMsgHTTPClient"] = 6803] = "NonStdMsgHTTPClient";
    EMsg[EMsg["NonStdMsgWGResponse"] = 6804] = "NonStdMsgWGResponse";
    EMsg[EMsg["NonStdMsgPHPSimulator"] = 6805] = "NonStdMsgPHPSimulator";
    EMsg[EMsg["NonStdMsgChase"] = 6806] = "NonStdMsgChase";
    EMsg[EMsg["NonStdMsgDFSTransfer"] = 6807] = "NonStdMsgDFSTransfer";
    EMsg[EMsg["NonStdMsgTests"] = 6808] = "NonStdMsgTests";
    EMsg[EMsg["NonStdMsgUMQpipeAAPL"] = 6809] = "NonStdMsgUMQpipeAAPL";
    EMsg[EMsg["NonStdMsgSyslog"] = 6810] = "NonStdMsgSyslog";
    EMsg[EMsg["NonStdMsgLogsink"] = 6811] = "NonStdMsgLogsink";
    EMsg[EMsg["NonStdMsgSteam2Emulator"] = 6812] = "NonStdMsgSteam2Emulator";
    EMsg[EMsg["NonStdMsgRTMPServer"] = 6813] = "NonStdMsgRTMPServer";
    EMsg[EMsg["NonStdMsgWebSocket"] = 6814] = "NonStdMsgWebSocket";
    EMsg[EMsg["NonStdMsgRedis"] = 6815] = "NonStdMsgRedis";
    EMsg[EMsg["UDSBase"] = 7000] = "UDSBase";
    EMsg[EMsg["ClientUDSP2PSessionStarted"] = 7001] = "ClientUDSP2PSessionStarted";
    EMsg[EMsg["ClientUDSP2PSessionEnded"] = 7002] = "ClientUDSP2PSessionEnded";
    EMsg[EMsg["UDSRenderUserAuth"] = 7003] = "UDSRenderUserAuth";
    EMsg[EMsg["UDSRenderUserAuthResponse"] = 7004] = "UDSRenderUserAuthResponse";
    EMsg[EMsg["ClientUDSInviteToGame"] = 7005] = "ClientUDSInviteToGame";
    EMsg[EMsg["ClientInviteToGame"] = 7005] = "ClientInviteToGame";
    EMsg[EMsg["UDSFindSession"] = 7006] = "UDSFindSession";
    EMsg[EMsg["UDSHasSession"] = 7006] = "UDSHasSession";
    EMsg[EMsg["UDSFindSessionResponse"] = 7007] = "UDSFindSessionResponse";
    EMsg[EMsg["UDSHasSessionResponse"] = 7007] = "UDSHasSessionResponse";
    EMsg[EMsg["MPASBase"] = 7100] = "MPASBase";
    EMsg[EMsg["MPASVacBanReset"] = 7101] = "MPASVacBanReset";
    EMsg[EMsg["KGSBase"] = 7200] = "KGSBase";
    EMsg[EMsg["KGSAllocateKeyRange"] = 7201] = "KGSAllocateKeyRange";
    EMsg[EMsg["KGSAllocateKeyRangeResponse"] = 7202] = "KGSAllocateKeyRangeResponse";
    EMsg[EMsg["KGSGenerateKeys"] = 7203] = "KGSGenerateKeys";
    EMsg[EMsg["KGSGenerateKeysResponse"] = 7204] = "KGSGenerateKeysResponse";
    EMsg[EMsg["KGSRemapKeys"] = 7205] = "KGSRemapKeys";
    EMsg[EMsg["KGSRemapKeysResponse"] = 7206] = "KGSRemapKeysResponse";
    EMsg[EMsg["KGSGenerateGameStopWCKeys"] = 7207] = "KGSGenerateGameStopWCKeys";
    EMsg[EMsg["KGSGenerateGameStopWCKeysResponse"] = 7208] = "KGSGenerateGameStopWCKeysResponse";
    EMsg[EMsg["UCMBase"] = 7300] = "UCMBase";
    EMsg[EMsg["ClientUCMAddScreenshot"] = 7301] = "ClientUCMAddScreenshot";
    EMsg[EMsg["ClientUCMAddScreenshotResponse"] = 7302] = "ClientUCMAddScreenshotResponse";
    EMsg[EMsg["UCMValidateObjectExists"] = 7303] = "UCMValidateObjectExists";
    EMsg[EMsg["UCMValidateObjectExistsResponse"] = 7304] = "UCMValidateObjectExistsResponse";
    EMsg[EMsg["UCMResetCommunityContent"] = 7307] = "UCMResetCommunityContent";
    EMsg[EMsg["UCMResetCommunityContentResponse"] = 7308] = "UCMResetCommunityContentResponse";
    EMsg[EMsg["ClientUCMDeleteScreenshot"] = 7309] = "ClientUCMDeleteScreenshot";
    EMsg[EMsg["ClientUCMDeleteScreenshotResponse"] = 7310] = "ClientUCMDeleteScreenshotResponse";
    EMsg[EMsg["ClientUCMPublishFile"] = 7311] = "ClientUCMPublishFile";
    EMsg[EMsg["ClientUCMPublishFileResponse"] = 7312] = "ClientUCMPublishFileResponse";
    EMsg[EMsg["ClientUCMGetPublishedFileDetails"] = 7313] = "ClientUCMGetPublishedFileDetails";
    EMsg[EMsg["ClientUCMGetPublishedFileDetailsResponse"] = 7314] = "ClientUCMGetPublishedFileDetailsResponse";
    EMsg[EMsg["ClientUCMDeletePublishedFile"] = 7315] = "ClientUCMDeletePublishedFile";
    EMsg[EMsg["ClientUCMDeletePublishedFileResponse"] = 7316] = "ClientUCMDeletePublishedFileResponse";
    EMsg[EMsg["ClientUCMEnumerateUserPublishedFiles"] = 7317] = "ClientUCMEnumerateUserPublishedFiles";
    EMsg[EMsg["ClientUCMEnumerateUserPublishedFilesResponse"] = 7318] = "ClientUCMEnumerateUserPublishedFilesResponse";
    EMsg[EMsg["ClientUCMSubscribePublishedFile"] = 7319] = "ClientUCMSubscribePublishedFile";
    EMsg[EMsg["ClientUCMSubscribePublishedFileResponse"] = 7320] = "ClientUCMSubscribePublishedFileResponse";
    EMsg[EMsg["ClientUCMEnumerateUserSubscribedFiles"] = 7321] = "ClientUCMEnumerateUserSubscribedFiles";
    EMsg[EMsg["ClientUCMEnumerateUserSubscribedFilesResponse"] = 7322] = "ClientUCMEnumerateUserSubscribedFilesResponse";
    EMsg[EMsg["ClientUCMUnsubscribePublishedFile"] = 7323] = "ClientUCMUnsubscribePublishedFile";
    EMsg[EMsg["ClientUCMUnsubscribePublishedFileResponse"] = 7324] = "ClientUCMUnsubscribePublishedFileResponse";
    EMsg[EMsg["ClientUCMUpdatePublishedFile"] = 7325] = "ClientUCMUpdatePublishedFile";
    EMsg[EMsg["ClientUCMUpdatePublishedFileResponse"] = 7326] = "ClientUCMUpdatePublishedFileResponse";
    EMsg[EMsg["UCMUpdatePublishedFile"] = 7327] = "UCMUpdatePublishedFile";
    EMsg[EMsg["UCMUpdatePublishedFileResponse"] = 7328] = "UCMUpdatePublishedFileResponse";
    EMsg[EMsg["UCMDeletePublishedFile"] = 7329] = "UCMDeletePublishedFile";
    EMsg[EMsg["UCMDeletePublishedFileResponse"] = 7330] = "UCMDeletePublishedFileResponse";
    EMsg[EMsg["UCMUpdatePublishedFileStat"] = 7331] = "UCMUpdatePublishedFileStat";
    EMsg[EMsg["UCMUpdatePublishedFileBan"] = 7332] = "UCMUpdatePublishedFileBan";
    EMsg[EMsg["UCMUpdatePublishedFileBanResponse"] = 7333] = "UCMUpdatePublishedFileBanResponse";
    EMsg[EMsg["UCMUpdateTaggedScreenshot"] = 7334] = "UCMUpdateTaggedScreenshot";
    EMsg[EMsg["UCMAddTaggedScreenshot"] = 7335] = "UCMAddTaggedScreenshot";
    EMsg[EMsg["UCMRemoveTaggedScreenshot"] = 7336] = "UCMRemoveTaggedScreenshot";
    EMsg[EMsg["UCMReloadPublishedFile"] = 7337] = "UCMReloadPublishedFile";
    EMsg[EMsg["UCMReloadUserFileListCaches"] = 7338] = "UCMReloadUserFileListCaches";
    EMsg[EMsg["UCMPublishedFileReported"] = 7339] = "UCMPublishedFileReported";
    EMsg[EMsg["UCMUpdatePublishedFileIncompatibleStatus"] = 7340] = "UCMUpdatePublishedFileIncompatibleStatus";
    EMsg[EMsg["UCMPublishedFilePreviewAdd"] = 7341] = "UCMPublishedFilePreviewAdd";
    EMsg[EMsg["UCMPublishedFilePreviewAddResponse"] = 7342] = "UCMPublishedFilePreviewAddResponse";
    EMsg[EMsg["UCMPublishedFilePreviewRemove"] = 7343] = "UCMPublishedFilePreviewRemove";
    EMsg[EMsg["UCMPublishedFilePreviewRemoveResponse"] = 7344] = "UCMPublishedFilePreviewRemoveResponse";
    EMsg[EMsg["UCMPublishedFilePreviewChangeSortOrder"] = 7345] = "UCMPublishedFilePreviewChangeSortOrder";
    EMsg[EMsg["UCMPublishedFilePreviewChangeSortOrderResponse"] = 7346] = "UCMPublishedFilePreviewChangeSortOrderResponse";
    EMsg[EMsg["ClientUCMPublishedFileSubscribed"] = 7347] = "ClientUCMPublishedFileSubscribed";
    EMsg[EMsg["ClientUCMPublishedFileUnsubscribed"] = 7348] = "ClientUCMPublishedFileUnsubscribed";
    EMsg[EMsg["UCMPublishedFileSubscribed"] = 7349] = "UCMPublishedFileSubscribed";
    EMsg[EMsg["UCMPublishedFileUnsubscribed"] = 7350] = "UCMPublishedFileUnsubscribed";
    EMsg[EMsg["UCMPublishFile"] = 7351] = "UCMPublishFile";
    EMsg[EMsg["UCMPublishFileResponse"] = 7352] = "UCMPublishFileResponse";
    EMsg[EMsg["UCMPublishedFileChildAdd"] = 7353] = "UCMPublishedFileChildAdd";
    EMsg[EMsg["UCMPublishedFileChildAddResponse"] = 7354] = "UCMPublishedFileChildAddResponse";
    EMsg[EMsg["UCMPublishedFileChildRemove"] = 7355] = "UCMPublishedFileChildRemove";
    EMsg[EMsg["UCMPublishedFileChildRemoveResponse"] = 7356] = "UCMPublishedFileChildRemoveResponse";
    EMsg[EMsg["UCMPublishedFileChildChangeSortOrder"] = 7357] = "UCMPublishedFileChildChangeSortOrder";
    EMsg[EMsg["UCMPublishedFileChildChangeSortOrderResponse"] = 7358] = "UCMPublishedFileChildChangeSortOrderResponse";
    EMsg[EMsg["UCMPublishedFileParentChanged"] = 7359] = "UCMPublishedFileParentChanged";
    EMsg[EMsg["ClientUCMGetPublishedFilesForUser"] = 7360] = "ClientUCMGetPublishedFilesForUser";
    EMsg[EMsg["ClientUCMGetPublishedFilesForUserResponse"] = 7361] = "ClientUCMGetPublishedFilesForUserResponse";
    EMsg[EMsg["UCMGetPublishedFilesForUser"] = 7362] = "UCMGetPublishedFilesForUser";
    EMsg[EMsg["UCMGetPublishedFilesForUserResponse"] = 7363] = "UCMGetPublishedFilesForUserResponse";
    EMsg[EMsg["ClientUCMSetUserPublishedFileAction"] = 7364] = "ClientUCMSetUserPublishedFileAction";
    EMsg[EMsg["ClientUCMSetUserPublishedFileActionResponse"] = 7365] = "ClientUCMSetUserPublishedFileActionResponse";
    EMsg[EMsg["ClientUCMEnumeratePublishedFilesByUserAction"] = 7366] = "ClientUCMEnumeratePublishedFilesByUserAction";
    EMsg[EMsg["ClientUCMEnumeratePublishedFilesByUserActionResponse"] = 7367] = "ClientUCMEnumeratePublishedFilesByUserActionResponse";
    EMsg[EMsg["ClientUCMPublishedFileDeleted"] = 7368] = "ClientUCMPublishedFileDeleted";
    EMsg[EMsg["UCMGetUserSubscribedFiles"] = 7369] = "UCMGetUserSubscribedFiles";
    EMsg[EMsg["UCMGetUserSubscribedFilesResponse"] = 7370] = "UCMGetUserSubscribedFilesResponse";
    EMsg[EMsg["UCMFixStatsPublishedFile"] = 7371] = "UCMFixStatsPublishedFile";
    EMsg[EMsg["UCMDeleteOldScreenshot"] = 7372] = "UCMDeleteOldScreenshot";
    EMsg[EMsg["UCMDeleteOldScreenshotResponse"] = 7373] = "UCMDeleteOldScreenshotResponse";
    EMsg[EMsg["UCMDeleteOldVideo"] = 7374] = "UCMDeleteOldVideo";
    EMsg[EMsg["UCMDeleteOldVideoResponse"] = 7375] = "UCMDeleteOldVideoResponse";
    EMsg[EMsg["UCMUpdateOldScreenshotPrivacy"] = 7376] = "UCMUpdateOldScreenshotPrivacy";
    EMsg[EMsg["UCMUpdateOldScreenshotPrivacyResponse"] = 7377] = "UCMUpdateOldScreenshotPrivacyResponse";
    EMsg[EMsg["ClientUCMEnumerateUserSubscribedFilesWithUpdates"] = 7378] = "ClientUCMEnumerateUserSubscribedFilesWithUpdates";
    EMsg[EMsg["ClientUCMEnumerateUserSubscribedFilesWithUpdatesResponse"] = 7379] = "ClientUCMEnumerateUserSubscribedFilesWithUpdatesResponse";
    EMsg[EMsg["UCMPublishedFileContentUpdated"] = 7380] = "UCMPublishedFileContentUpdated";
    EMsg[EMsg["UCMPublishedFileUpdated"] = 7381] = "UCMPublishedFileUpdated";
    EMsg[EMsg["ClientUCMPublishedFileUpdated"] = 7381] = "ClientUCMPublishedFileUpdated";
    EMsg[EMsg["ClientWorkshopItemChangesRequest"] = 7382] = "ClientWorkshopItemChangesRequest";
    EMsg[EMsg["ClientWorkshopItemChangesResponse"] = 7383] = "ClientWorkshopItemChangesResponse";
    EMsg[EMsg["ClientWorkshopItemInfoRequest"] = 7384] = "ClientWorkshopItemInfoRequest";
    EMsg[EMsg["ClientWorkshopItemInfoResponse"] = 7385] = "ClientWorkshopItemInfoResponse";
    EMsg[EMsg["FSBase"] = 7500] = "FSBase";
    EMsg[EMsg["ClientRichPresenceUpload"] = 7501] = "ClientRichPresenceUpload";
    EMsg[EMsg["ClientRichPresenceRequest"] = 7502] = "ClientRichPresenceRequest";
    EMsg[EMsg["ClientRichPresenceInfo"] = 7503] = "ClientRichPresenceInfo";
    EMsg[EMsg["FSRichPresenceRequest"] = 7504] = "FSRichPresenceRequest";
    EMsg[EMsg["FSRichPresenceResponse"] = 7505] = "FSRichPresenceResponse";
    EMsg[EMsg["FSComputeFrenematrix"] = 7506] = "FSComputeFrenematrix";
    EMsg[EMsg["FSComputeFrenematrixResponse"] = 7507] = "FSComputeFrenematrixResponse";
    EMsg[EMsg["FSPlayStatusNotification"] = 7508] = "FSPlayStatusNotification";
    EMsg[EMsg["FSPublishPersonaStatus"] = 7509] = "FSPublishPersonaStatus";
    EMsg[EMsg["FSAddOrRemoveFollower"] = 7510] = "FSAddOrRemoveFollower";
    EMsg[EMsg["FSAddOrRemoveFollowerResponse"] = 7511] = "FSAddOrRemoveFollowerResponse";
    EMsg[EMsg["FSUpdateFollowingList"] = 7512] = "FSUpdateFollowingList";
    EMsg[EMsg["FSCommentNotification"] = 7513] = "FSCommentNotification";
    EMsg[EMsg["FSCommentNotificationViewed"] = 7514] = "FSCommentNotificationViewed";
    EMsg[EMsg["ClientFSGetFollowerCount"] = 7515] = "ClientFSGetFollowerCount";
    EMsg[EMsg["ClientFSGetFollowerCountResponse"] = 7516] = "ClientFSGetFollowerCountResponse";
    EMsg[EMsg["ClientFSGetIsFollowing"] = 7517] = "ClientFSGetIsFollowing";
    EMsg[EMsg["ClientFSGetIsFollowingResponse"] = 7518] = "ClientFSGetIsFollowingResponse";
    EMsg[EMsg["ClientFSEnumerateFollowingList"] = 7519] = "ClientFSEnumerateFollowingList";
    EMsg[EMsg["ClientFSEnumerateFollowingListResponse"] = 7520] = "ClientFSEnumerateFollowingListResponse";
    EMsg[EMsg["FSGetPendingNotificationCount"] = 7521] = "FSGetPendingNotificationCount";
    EMsg[EMsg["FSGetPendingNotificationCountResponse"] = 7522] = "FSGetPendingNotificationCountResponse";
    EMsg[EMsg["ClientFSOfflineMessageNotification"] = 7523] = "ClientFSOfflineMessageNotification";
    EMsg[EMsg["ClientChatOfflineMessageNotification"] = 7523] = "ClientChatOfflineMessageNotification";
    EMsg[EMsg["ClientFSRequestOfflineMessageCount"] = 7524] = "ClientFSRequestOfflineMessageCount";
    EMsg[EMsg["ClientChatRequestOfflineMessageCount"] = 7524] = "ClientChatRequestOfflineMessageCount";
    EMsg[EMsg["ClientFSGetFriendMessageHistory"] = 7525] = "ClientFSGetFriendMessageHistory";
    EMsg[EMsg["ClientChatGetFriendMessageHistory"] = 7525] = "ClientChatGetFriendMessageHistory";
    EMsg[EMsg["ClientFSGetFriendMessageHistoryResponse"] = 7526] = "ClientFSGetFriendMessageHistoryResponse";
    EMsg[EMsg["ClientChatGetFriendMessageHistoryResponse"] = 7526] = "ClientChatGetFriendMessageHistoryResponse";
    EMsg[EMsg["ClientFSGetFriendMessageHistoryForOfflineMessages"] = 7527] = "ClientFSGetFriendMessageHistoryForOfflineMessages";
    EMsg[EMsg["ClientChatGetFriendMessageHistoryForOfflineMessages"] = 7527] = "ClientChatGetFriendMessageHistoryForOfflineMessages";
    EMsg[EMsg["ClientFSGetFriendsSteamLevels"] = 7528] = "ClientFSGetFriendsSteamLevels";
    EMsg[EMsg["ClientFSGetFriendsSteamLevelsResponse"] = 7529] = "ClientFSGetFriendsSteamLevelsResponse";
    EMsg[EMsg["AMRequestFriendData"] = 7530] = "AMRequestFriendData";
    EMsg[EMsg["FSRequestFriendData"] = 7530] = "FSRequestFriendData";
    EMsg[EMsg["CEGVersionSetEnableDisableRequest"] = 7600] = "CEGVersionSetEnableDisableRequest";
    EMsg[EMsg["DRMRange2"] = 7600] = "DRMRange2";
    EMsg[EMsg["CEGVersionSetEnableDisableResponse"] = 7601] = "CEGVersionSetEnableDisableResponse";
    EMsg[EMsg["CEGPropStatusDRMSRequest"] = 7602] = "CEGPropStatusDRMSRequest";
    EMsg[EMsg["CEGPropStatusDRMSResponse"] = 7603] = "CEGPropStatusDRMSResponse";
    EMsg[EMsg["CEGWhackFailureReportRequest"] = 7604] = "CEGWhackFailureReportRequest";
    EMsg[EMsg["CEGWhackFailureReportResponse"] = 7605] = "CEGWhackFailureReportResponse";
    EMsg[EMsg["DRMSFetchVersionSet"] = 7606] = "DRMSFetchVersionSet";
    EMsg[EMsg["DRMSFetchVersionSetResponse"] = 7607] = "DRMSFetchVersionSetResponse";
    EMsg[EMsg["EconBase"] = 7700] = "EconBase";
    EMsg[EMsg["EconTrading_InitiateTradeRequest"] = 7701] = "EconTrading_InitiateTradeRequest";
    EMsg[EMsg["EconTrading_InitiateTradeProposed"] = 7702] = "EconTrading_InitiateTradeProposed";
    EMsg[EMsg["EconTrading_InitiateTradeResponse"] = 7703] = "EconTrading_InitiateTradeResponse";
    EMsg[EMsg["EconTrading_InitiateTradeResult"] = 7704] = "EconTrading_InitiateTradeResult";
    EMsg[EMsg["EconTrading_StartSession"] = 7705] = "EconTrading_StartSession";
    EMsg[EMsg["EconTrading_CancelTradeRequest"] = 7706] = "EconTrading_CancelTradeRequest";
    EMsg[EMsg["EconFlushInventoryCache"] = 7707] = "EconFlushInventoryCache";
    EMsg[EMsg["EconFlushInventoryCacheResponse"] = 7708] = "EconFlushInventoryCacheResponse";
    EMsg[EMsg["EconCDKeyProcessTransaction"] = 7711] = "EconCDKeyProcessTransaction";
    EMsg[EMsg["EconCDKeyProcessTransactionResponse"] = 7712] = "EconCDKeyProcessTransactionResponse";
    EMsg[EMsg["EconGetErrorLogs"] = 7713] = "EconGetErrorLogs";
    EMsg[EMsg["EconGetErrorLogsResponse"] = 7714] = "EconGetErrorLogsResponse";
    EMsg[EMsg["RMRange"] = 7800] = "RMRange";
    EMsg[EMsg["RMTestVerisignOTP"] = 7800] = "RMTestVerisignOTP";
    EMsg[EMsg["RMTestVerisignOTPResponse"] = 7801] = "RMTestVerisignOTPResponse";
    EMsg[EMsg["RMDeleteMemcachedKeys"] = 7803] = "RMDeleteMemcachedKeys";
    EMsg[EMsg["RMRemoteInvoke"] = 7804] = "RMRemoteInvoke";
    EMsg[EMsg["BadLoginIPList"] = 7805] = "BadLoginIPList";
    EMsg[EMsg["RMMsgTraceAddTrigger"] = 7806] = "RMMsgTraceAddTrigger";
    EMsg[EMsg["RMMsgTraceRemoveTrigger"] = 7807] = "RMMsgTraceRemoveTrigger";
    EMsg[EMsg["RMMsgTraceEvent"] = 7808] = "RMMsgTraceEvent";
    EMsg[EMsg["UGSBase"] = 7900] = "UGSBase";
    EMsg[EMsg["UGSUpdateGlobalStats"] = 7900] = "UGSUpdateGlobalStats";
    EMsg[EMsg["ClientUGSGetGlobalStats"] = 7901] = "ClientUGSGetGlobalStats";
    EMsg[EMsg["ClientUGSGetGlobalStatsResponse"] = 7902] = "ClientUGSGetGlobalStatsResponse";
    EMsg[EMsg["StoreUpdateRecommendationCount"] = 8000] = "StoreUpdateRecommendationCount";
    EMsg[EMsg["StoreBase"] = 8000] = "StoreBase";
    EMsg[EMsg["UMQBase"] = 8100] = "UMQBase";
    EMsg[EMsg["UMQLogonRequest"] = 8100] = "UMQLogonRequest";
    EMsg[EMsg["UMQLogonResponse"] = 8101] = "UMQLogonResponse";
    EMsg[EMsg["UMQLogoffRequest"] = 8102] = "UMQLogoffRequest";
    EMsg[EMsg["UMQLogoffResponse"] = 8103] = "UMQLogoffResponse";
    EMsg[EMsg["UMQSendChatMessage"] = 8104] = "UMQSendChatMessage";
    EMsg[EMsg["UMQIncomingChatMessage"] = 8105] = "UMQIncomingChatMessage";
    EMsg[EMsg["UMQPoll"] = 8106] = "UMQPoll";
    EMsg[EMsg["UMQPollResults"] = 8107] = "UMQPollResults";
    EMsg[EMsg["UMQ2AM_ClientMsgBatch"] = 8108] = "UMQ2AM_ClientMsgBatch";
    EMsg[EMsg["UMQEnqueueMobileSalePromotions"] = 8109] = "UMQEnqueueMobileSalePromotions";
    EMsg[EMsg["UMQEnqueueMobileAnnouncements"] = 8110] = "UMQEnqueueMobileAnnouncements";
    EMsg[EMsg["WorkshopAcceptTOSRequest"] = 8200] = "WorkshopAcceptTOSRequest";
    EMsg[EMsg["WorkshopBase"] = 8200] = "WorkshopBase";
    EMsg[EMsg["WorkshopAcceptTOSResponse"] = 8201] = "WorkshopAcceptTOSResponse";
    EMsg[EMsg["WebAPIBase"] = 8300] = "WebAPIBase";
    EMsg[EMsg["WebAPIValidateOAuth2Token"] = 8300] = "WebAPIValidateOAuth2Token";
    EMsg[EMsg["WebAPIValidateOAuth2TokenResponse"] = 8301] = "WebAPIValidateOAuth2TokenResponse";
    EMsg[EMsg["WebAPIInvalidateTokensForAccount"] = 8302] = "WebAPIInvalidateTokensForAccount";
    EMsg[EMsg["WebAPIRegisterGCInterfaces"] = 8303] = "WebAPIRegisterGCInterfaces";
    EMsg[EMsg["WebAPIInvalidateOAuthClientCache"] = 8304] = "WebAPIInvalidateOAuthClientCache";
    EMsg[EMsg["WebAPIInvalidateOAuthTokenCache"] = 8305] = "WebAPIInvalidateOAuthTokenCache";
    EMsg[EMsg["WebAPISetSecrets"] = 8306] = "WebAPISetSecrets";
    EMsg[EMsg["BackpackBase"] = 8400] = "BackpackBase";
    EMsg[EMsg["BackpackAddToCurrency"] = 8401] = "BackpackAddToCurrency";
    EMsg[EMsg["BackpackAddToCurrencyResponse"] = 8402] = "BackpackAddToCurrencyResponse";
    EMsg[EMsg["CREBase"] = 8500] = "CREBase";
    EMsg[EMsg["CRERankByTrend"] = 8501] = "CRERankByTrend";
    EMsg[EMsg["CRERankByTrendResponse"] = 8502] = "CRERankByTrendResponse";
    EMsg[EMsg["CREItemVoteSummary"] = 8503] = "CREItemVoteSummary";
    EMsg[EMsg["CREItemVoteSummaryResponse"] = 8504] = "CREItemVoteSummaryResponse";
    EMsg[EMsg["CRERankByVote"] = 8505] = "CRERankByVote";
    EMsg[EMsg["CRERankByVoteResponse"] = 8506] = "CRERankByVoteResponse";
    EMsg[EMsg["CREUpdateUserPublishedItemVote"] = 8507] = "CREUpdateUserPublishedItemVote";
    EMsg[EMsg["CREUpdateUserPublishedItemVoteResponse"] = 8508] = "CREUpdateUserPublishedItemVoteResponse";
    EMsg[EMsg["CREGetUserPublishedItemVoteDetails"] = 8509] = "CREGetUserPublishedItemVoteDetails";
    EMsg[EMsg["CREGetUserPublishedItemVoteDetailsResponse"] = 8510] = "CREGetUserPublishedItemVoteDetailsResponse";
    EMsg[EMsg["CREEnumeratePublishedFiles"] = 8511] = "CREEnumeratePublishedFiles";
    EMsg[EMsg["CREEnumeratePublishedFilesResponse"] = 8512] = "CREEnumeratePublishedFilesResponse";
    EMsg[EMsg["CREPublishedFileVoteAdded"] = 8513] = "CREPublishedFileVoteAdded";
    EMsg[EMsg["SecretsBase"] = 8600] = "SecretsBase";
    EMsg[EMsg["SecretsRequestCredentialPair"] = 8600] = "SecretsRequestCredentialPair";
    EMsg[EMsg["SecretsCredentialPairResponse"] = 8601] = "SecretsCredentialPairResponse";
    EMsg[EMsg["SecretsRequestServerIdentity"] = 8602] = "SecretsRequestServerIdentity";
    EMsg[EMsg["SecretsServerIdentityResponse"] = 8603] = "SecretsServerIdentityResponse";
    EMsg[EMsg["SecretsUpdateServerIdentities"] = 8604] = "SecretsUpdateServerIdentities";
    EMsg[EMsg["BoxMonitorBase"] = 8700] = "BoxMonitorBase";
    EMsg[EMsg["BoxMonitorReportRequest"] = 8700] = "BoxMonitorReportRequest";
    EMsg[EMsg["BoxMonitorReportResponse"] = 8701] = "BoxMonitorReportResponse";
    EMsg[EMsg["LogsinkBase"] = 8800] = "LogsinkBase";
    EMsg[EMsg["LogsinkWriteReport"] = 8800] = "LogsinkWriteReport";
    EMsg[EMsg["PICSBase"] = 8900] = "PICSBase";
    EMsg[EMsg["ClientPICSChangesSinceRequest"] = 8901] = "ClientPICSChangesSinceRequest";
    EMsg[EMsg["ClientPICSChangesSinceResponse"] = 8902] = "ClientPICSChangesSinceResponse";
    EMsg[EMsg["ClientPICSProductInfoRequest"] = 8903] = "ClientPICSProductInfoRequest";
    EMsg[EMsg["ClientPICSProductInfoResponse"] = 8904] = "ClientPICSProductInfoResponse";
    EMsg[EMsg["ClientPICSAccessTokenRequest"] = 8905] = "ClientPICSAccessTokenRequest";
    EMsg[EMsg["ClientPICSAccessTokenResponse"] = 8906] = "ClientPICSAccessTokenResponse";
    EMsg[EMsg["WorkerProcess"] = 9000] = "WorkerProcess";
    EMsg[EMsg["WorkerProcessPingRequest"] = 9000] = "WorkerProcessPingRequest";
    EMsg[EMsg["WorkerProcessPingResponse"] = 9001] = "WorkerProcessPingResponse";
    EMsg[EMsg["WorkerProcessShutdown"] = 9002] = "WorkerProcessShutdown";
    EMsg[EMsg["DRMWorkerProcess"] = 9100] = "DRMWorkerProcess";
    EMsg[EMsg["DRMWorkerProcessDRMAndSign"] = 9100] = "DRMWorkerProcessDRMAndSign";
    EMsg[EMsg["DRMWorkerProcessDRMAndSignResponse"] = 9101] = "DRMWorkerProcessDRMAndSignResponse";
    EMsg[EMsg["DRMWorkerProcessSteamworksInfoRequest"] = 9102] = "DRMWorkerProcessSteamworksInfoRequest";
    EMsg[EMsg["DRMWorkerProcessSteamworksInfoResponse"] = 9103] = "DRMWorkerProcessSteamworksInfoResponse";
    EMsg[EMsg["DRMWorkerProcessInstallDRMDLLRequest"] = 9104] = "DRMWorkerProcessInstallDRMDLLRequest";
    EMsg[EMsg["DRMWorkerProcessInstallDRMDLLResponse"] = 9105] = "DRMWorkerProcessInstallDRMDLLResponse";
    EMsg[EMsg["DRMWorkerProcessSecretIdStringRequest"] = 9106] = "DRMWorkerProcessSecretIdStringRequest";
    EMsg[EMsg["DRMWorkerProcessSecretIdStringResponse"] = 9107] = "DRMWorkerProcessSecretIdStringResponse";
    EMsg[EMsg["DRMWorkerProcessGetDRMGuidsFromFileRequest"] = 9108] = "DRMWorkerProcessGetDRMGuidsFromFileRequest";
    EMsg[EMsg["DRMWorkerProcessGetDRMGuidsFromFileResponse"] = 9109] = "DRMWorkerProcessGetDRMGuidsFromFileResponse";
    EMsg[EMsg["DRMWorkerProcessInstallProcessedFilesRequest"] = 9110] = "DRMWorkerProcessInstallProcessedFilesRequest";
    EMsg[EMsg["DRMWorkerProcessInstallProcessedFilesResponse"] = 9111] = "DRMWorkerProcessInstallProcessedFilesResponse";
    EMsg[EMsg["DRMWorkerProcessExamineBlobRequest"] = 9112] = "DRMWorkerProcessExamineBlobRequest";
    EMsg[EMsg["DRMWorkerProcessExamineBlobResponse"] = 9113] = "DRMWorkerProcessExamineBlobResponse";
    EMsg[EMsg["DRMWorkerProcessDescribeSecretRequest"] = 9114] = "DRMWorkerProcessDescribeSecretRequest";
    EMsg[EMsg["DRMWorkerProcessDescribeSecretResponse"] = 9115] = "DRMWorkerProcessDescribeSecretResponse";
    EMsg[EMsg["DRMWorkerProcessBackfillOriginalRequest"] = 9116] = "DRMWorkerProcessBackfillOriginalRequest";
    EMsg[EMsg["DRMWorkerProcessBackfillOriginalResponse"] = 9117] = "DRMWorkerProcessBackfillOriginalResponse";
    EMsg[EMsg["DRMWorkerProcessValidateDRMDLLRequest"] = 9118] = "DRMWorkerProcessValidateDRMDLLRequest";
    EMsg[EMsg["DRMWorkerProcessValidateDRMDLLResponse"] = 9119] = "DRMWorkerProcessValidateDRMDLLResponse";
    EMsg[EMsg["DRMWorkerProcessValidateFileRequest"] = 9120] = "DRMWorkerProcessValidateFileRequest";
    EMsg[EMsg["DRMWorkerProcessValidateFileResponse"] = 9121] = "DRMWorkerProcessValidateFileResponse";
    EMsg[EMsg["DRMWorkerProcessSplitAndInstallRequest"] = 9122] = "DRMWorkerProcessSplitAndInstallRequest";
    EMsg[EMsg["DRMWorkerProcessSplitAndInstallResponse"] = 9123] = "DRMWorkerProcessSplitAndInstallResponse";
    EMsg[EMsg["DRMWorkerProcessGetBlobRequest"] = 9124] = "DRMWorkerProcessGetBlobRequest";
    EMsg[EMsg["DRMWorkerProcessGetBlobResponse"] = 9125] = "DRMWorkerProcessGetBlobResponse";
    EMsg[EMsg["DRMWorkerProcessEvaluateCrashRequest"] = 9126] = "DRMWorkerProcessEvaluateCrashRequest";
    EMsg[EMsg["DRMWorkerProcessEvaluateCrashResponse"] = 9127] = "DRMWorkerProcessEvaluateCrashResponse";
    EMsg[EMsg["DRMWorkerProcessAnalyzeFileRequest"] = 9128] = "DRMWorkerProcessAnalyzeFileRequest";
    EMsg[EMsg["DRMWorkerProcessAnalyzeFileResponse"] = 9129] = "DRMWorkerProcessAnalyzeFileResponse";
    EMsg[EMsg["DRMWorkerProcessUnpackBlobRequest"] = 9130] = "DRMWorkerProcessUnpackBlobRequest";
    EMsg[EMsg["DRMWorkerProcessUnpackBlobResponse"] = 9131] = "DRMWorkerProcessUnpackBlobResponse";
    EMsg[EMsg["DRMWorkerProcessInstallAllRequest"] = 9132] = "DRMWorkerProcessInstallAllRequest";
    EMsg[EMsg["DRMWorkerProcessInstallAllResponse"] = 9133] = "DRMWorkerProcessInstallAllResponse";
    EMsg[EMsg["TestWorkerProcess"] = 9200] = "TestWorkerProcess";
    EMsg[EMsg["TestWorkerProcessLoadUnloadModuleRequest"] = 9200] = "TestWorkerProcessLoadUnloadModuleRequest";
    EMsg[EMsg["TestWorkerProcessLoadUnloadModuleResponse"] = 9201] = "TestWorkerProcessLoadUnloadModuleResponse";
    EMsg[EMsg["TestWorkerProcessServiceModuleCallRequest"] = 9202] = "TestWorkerProcessServiceModuleCallRequest";
    EMsg[EMsg["TestWorkerProcessServiceModuleCallResponse"] = 9203] = "TestWorkerProcessServiceModuleCallResponse";
    EMsg[EMsg["QuestServerBase"] = 9300] = "QuestServerBase";
    EMsg[EMsg["ClientGetEmoticonList"] = 9330] = "ClientGetEmoticonList";
    EMsg[EMsg["ClientEmoticonList"] = 9331] = "ClientEmoticonList";
    EMsg[EMsg["ClientSharedLibraryBase"] = 9400] = "ClientSharedLibraryBase";
    EMsg[EMsg["SLCBase"] = 9400] = "SLCBase";
    EMsg[EMsg["SLCUserSessionStatus"] = 9400] = "SLCUserSessionStatus";
    EMsg[EMsg["SLCRequestUserSessionStatus"] = 9401] = "SLCRequestUserSessionStatus";
    EMsg[EMsg["SLCSharedLicensesLockStatus"] = 9402] = "SLCSharedLicensesLockStatus";
    EMsg[EMsg["ClientSharedLicensesLockStatus"] = 9403] = "ClientSharedLicensesLockStatus";
    EMsg[EMsg["ClientSharedLicensesStopPlaying"] = 9404] = "ClientSharedLicensesStopPlaying";
    EMsg[EMsg["ClientSharedLibraryLockStatus"] = 9405] = "ClientSharedLibraryLockStatus";
    EMsg[EMsg["ClientSharedLibraryStopPlaying"] = 9406] = "ClientSharedLibraryStopPlaying";
    EMsg[EMsg["SLCOwnerLibraryChanged"] = 9407] = "SLCOwnerLibraryChanged";
    EMsg[EMsg["SLCSharedLibraryChanged"] = 9408] = "SLCSharedLibraryChanged";
    EMsg[EMsg["RemoteClientAuth"] = 9500] = "RemoteClientAuth";
    EMsg[EMsg["RemoteClientBase"] = 9500] = "RemoteClientBase";
    EMsg[EMsg["RemoteClientAuthResponse"] = 9501] = "RemoteClientAuthResponse";
    EMsg[EMsg["RemoteClientAppStatus"] = 9502] = "RemoteClientAppStatus";
    EMsg[EMsg["RemoteClientStartStream"] = 9503] = "RemoteClientStartStream";
    EMsg[EMsg["RemoteClientStartStreamResponse"] = 9504] = "RemoteClientStartStreamResponse";
    EMsg[EMsg["RemoteClientPing"] = 9505] = "RemoteClientPing";
    EMsg[EMsg["RemoteClientPingResponse"] = 9506] = "RemoteClientPingResponse";
    EMsg[EMsg["ClientUnlockStreaming"] = 9507] = "ClientUnlockStreaming";
    EMsg[EMsg["ClientUnlockStreamingResponse"] = 9508] = "ClientUnlockStreamingResponse";
    EMsg[EMsg["RemoteClientAcceptEULA"] = 9509] = "RemoteClientAcceptEULA";
    EMsg[EMsg["RemoteClientGetControllerConfig"] = 9510] = "RemoteClientGetControllerConfig";
    EMsg[EMsg["RemoteClientGetControllerConfigResposne"] = 9511] = "RemoteClientGetControllerConfigResposne";
    EMsg[EMsg["RemoteClientGetControllerConfigResponse"] = 9511] = "RemoteClientGetControllerConfigResponse";
    EMsg[EMsg["RemoteClientStreamingEnabled"] = 9512] = "RemoteClientStreamingEnabled";
    EMsg[EMsg["ClientUnlockHEVC"] = 9513] = "ClientUnlockHEVC";
    EMsg[EMsg["ClientUnlockHEVCResponse"] = 9514] = "ClientUnlockHEVCResponse";
    EMsg[EMsg["RemoteClientStatusRequest"] = 9515] = "RemoteClientStatusRequest";
    EMsg[EMsg["RemoteClientStatusResponse"] = 9516] = "RemoteClientStatusResponse";
    EMsg[EMsg["ClientConcurrentSessionsBase"] = 9600] = "ClientConcurrentSessionsBase";
    EMsg[EMsg["ClientPlayingSessionState"] = 9600] = "ClientPlayingSessionState";
    EMsg[EMsg["ClientKickPlayingSession"] = 9601] = "ClientKickPlayingSession";
    EMsg[EMsg["ClientBroadcastBase"] = 9700] = "ClientBroadcastBase";
    EMsg[EMsg["ClientBroadcastInit"] = 9700] = "ClientBroadcastInit";
    EMsg[EMsg["ClientBroadcastFrames"] = 9701] = "ClientBroadcastFrames";
    EMsg[EMsg["ClientBroadcastDisconnect"] = 9702] = "ClientBroadcastDisconnect";
    EMsg[EMsg["ClientBroadcastScreenshot"] = 9703] = "ClientBroadcastScreenshot";
    EMsg[EMsg["ClientBroadcastUploadConfig"] = 9704] = "ClientBroadcastUploadConfig";
    EMsg[EMsg["BaseClient3"] = 9800] = "BaseClient3";
    EMsg[EMsg["ClientVoiceCallPreAuthorize"] = 9800] = "ClientVoiceCallPreAuthorize";
    EMsg[EMsg["ClientVoiceCallPreAuthorizeResponse"] = 9801] = "ClientVoiceCallPreAuthorizeResponse";
    EMsg[EMsg["ClientServerTimestampRequest"] = 9802] = "ClientServerTimestampRequest";
    EMsg[EMsg["ClientServerTimestampResponse"] = 9803] = "ClientServerTimestampResponse";
    EMsg[EMsg["ServiceMethodCallFromClientNonAuthed"] = 9804] = "ServiceMethodCallFromClientNonAuthed";
    EMsg[EMsg["ClientHello"] = 9805] = "ClientHello";
    EMsg[EMsg["ClientEnableOrDisableDownloads"] = 9806] = "ClientEnableOrDisableDownloads";
    EMsg[EMsg["ClientEnableOrDisableDownloadsResponse"] = 9807] = "ClientEnableOrDisableDownloadsResponse";
    EMsg[EMsg["ClientLANP2PBase"] = 9900] = "ClientLANP2PBase";
    EMsg[EMsg["ClientLANP2PRequestChunk"] = 9900] = "ClientLANP2PRequestChunk";
    EMsg[EMsg["ClientLANP2PRequestChunkResponse"] = 9901] = "ClientLANP2PRequestChunkResponse";
    EMsg[EMsg["ClientPeerChunkRequest"] = 9902] = "ClientPeerChunkRequest";
    EMsg[EMsg["ClientPeerChunkResponse"] = 9903] = "ClientPeerChunkResponse";
    EMsg[EMsg["ClientLANP2PMax"] = 9999] = "ClientLANP2PMax";
    EMsg[EMsg["NotifyWatchdog"] = 10000] = "NotifyWatchdog";
    EMsg[EMsg["BaseWatchdogServer"] = 10000] = "BaseWatchdogServer";
    EMsg[EMsg["ClientSiteLicenseBase"] = 10100] = "ClientSiteLicenseBase";
    EMsg[EMsg["ClientSiteLicenseSiteInfoNotification"] = 10100] = "ClientSiteLicenseSiteInfoNotification";
    EMsg[EMsg["ClientSiteLicenseCheckout"] = 10101] = "ClientSiteLicenseCheckout";
    EMsg[EMsg["ClientSiteLicenseCheckoutResponse"] = 10102] = "ClientSiteLicenseCheckoutResponse";
    EMsg[EMsg["ClientSiteLicenseGetAvailableSeats"] = 10103] = "ClientSiteLicenseGetAvailableSeats";
    EMsg[EMsg["ClientSiteLicenseGetAvailableSeatsResponse"] = 10104] = "ClientSiteLicenseGetAvailableSeatsResponse";
    EMsg[EMsg["ClientSiteLicenseGetContentCacheInfo"] = 10105] = "ClientSiteLicenseGetContentCacheInfo";
    EMsg[EMsg["ClientSiteLicenseGetContentCacheInfoResponse"] = 10106] = "ClientSiteLicenseGetContentCacheInfoResponse";
    EMsg[EMsg["BaseChatServer"] = 12000] = "BaseChatServer";
    EMsg[EMsg["ChatServerGetPendingNotificationCount"] = 12000] = "ChatServerGetPendingNotificationCount";
    EMsg[EMsg["ChatServerGetPendingNotificationCountResponse"] = 12001] = "ChatServerGetPendingNotificationCountResponse";
    EMsg[EMsg["BaseSecretServer"] = 12100] = "BaseSecretServer";
    EMsg[EMsg["ServerSecretChanged"] = 12100] = "ServerSecretChanged";
})(EMsg || (EMsg = {}));
exports.default = EMsg;
