// steam-session/dist/enums-steam/EOSType.js
"use strict";
/* eslint-disable */
// Auto-generated by generate-enums script on Thu Mar 02 2023 22:18:52 GMT-0500 (Eastern Standard Time)
Object.defineProperty(exports, "__esModule", { value: true });
var EOSType;
(function (EOSType) {
    EOSType[EOSType["Web"] = -700] = "Web";
    EOSType[EOSType["IOSUnknown"] = -600] = "IOSUnknown";
    EOSType[EOSType["IOS1"] = -599] = "IOS1";
    EOSType[EOSType["IOS2"] = -598] = "IOS2";
    EOSType[EOSType["IOS3"] = -597] = "IOS3";
    EOSType[EOSType["IOS4"] = -596] = "IOS4";
    EOSType[EOSType["IOS5"] = -595] = "IOS5";
    EOSType[EOSType["IOS6"] = -594] = "IOS6";
    EOSType[EOSType["IOS6_1"] = -593] = "IOS6_1";
    EOSType[EOSType["IOS7"] = -592] = "IOS7";
    EOSType[EOSType["IOS7_1"] = -591] = "IOS7_1";
    EOSType[EOSType["IOS8"] = -590] = "IOS8";
    EOSType[EOSType["IOS8_1"] = -589] = "IOS8_1";
    EOSType[EOSType["IOS8_2"] = -588] = "IOS8_2";
    EOSType[EOSType["IOS8_3"] = -587] = "IOS8_3";
    EOSType[EOSType["IOS8_4"] = -586] = "IOS8_4";
    EOSType[EOSType["IOS9"] = -585] = "IOS9";
    EOSType[EOSType["IOS9_1"] = -584] = "IOS9_1";
    EOSType[EOSType["IOS9_2"] = -583] = "IOS9_2";
    EOSType[EOSType["IOS9_3"] = -582] = "IOS9_3";
    EOSType[EOSType["IOS10"] = -581] = "IOS10";
    EOSType[EOSType["IOS10_1"] = -580] = "IOS10_1";
    EOSType[EOSType["IOS10_2"] = -579] = "IOS10_2";
    EOSType[EOSType["IOS10_3"] = -578] = "IOS10_3";
    EOSType[EOSType["IOS11"] = -577] = "IOS11";
    EOSType[EOSType["IOS11_1"] = -576] = "IOS11_1";
    EOSType[EOSType["IOS11_2"] = -575] = "IOS11_2";
    EOSType[EOSType["IOS11_3"] = -574] = "IOS11_3";
    EOSType[EOSType["IOS11_4"] = -573] = "IOS11_4";
    EOSType[EOSType["IOS12"] = -572] = "IOS12";
    EOSType[EOSType["IOS12_1"] = -571] = "IOS12_1";
    EOSType[EOSType["AndroidUnknown"] = -500] = "AndroidUnknown";
    EOSType[EOSType["Android6"] = -499] = "Android6";
    EOSType[EOSType["Android7"] = -498] = "Android7";
    EOSType[EOSType["Android8"] = -497] = "Android8";
    EOSType[EOSType["Android9"] = -496] = "Android9";
    EOSType[EOSType["UMQ"] = -400] = "UMQ";
    EOSType[EOSType["PS3"] = -300] = "PS3";
    EOSType[EOSType["LinuxUnknown"] = -203] = "LinuxUnknown";
    EOSType[EOSType["Linux22"] = -202] = "Linux22";
    EOSType[EOSType["Linux24"] = -201] = "Linux24";
    EOSType[EOSType["Linux26"] = -200] = "Linux26";
    EOSType[EOSType["Linux32"] = -199] = "Linux32";
    EOSType[EOSType["Linux35"] = -198] = "Linux35";
    EOSType[EOSType["Linux36"] = -197] = "Linux36";
    EOSType[EOSType["Linux310"] = -196] = "Linux310";
    EOSType[EOSType["Linux316"] = -195] = "Linux316";
    EOSType[EOSType["Linux318"] = -194] = "Linux318";
    EOSType[EOSType["Linux3x"] = -193] = "Linux3x";
    EOSType[EOSType["Linux4x"] = -192] = "Linux4x";
    EOSType[EOSType["Linux41"] = -191] = "Linux41";
    EOSType[EOSType["Linux44"] = -190] = "Linux44";
    EOSType[EOSType["Linux49"] = -189] = "Linux49";
    EOSType[EOSType["Linux414"] = -188] = "Linux414";
    EOSType[EOSType["Linux419"] = -187] = "Linux419";
    EOSType[EOSType["Linux5x"] = -186] = "Linux5x";
    EOSType[EOSType["Linux54"] = -185] = "Linux54";
    EOSType[EOSType["Linux6x"] = -184] = "Linux6x";
    EOSType[EOSType["Linux7x"] = -183] = "Linux7x";
    EOSType[EOSType["Linux510"] = -182] = "Linux510";
    EOSType[EOSType["MacOSUnknown"] = -102] = "MacOSUnknown";
    EOSType[EOSType["LinuxMax"] = -101] = "LinuxMax";
    EOSType[EOSType["MacOS104"] = -101] = "MacOS104";
    EOSType[EOSType["MacOS105"] = -100] = "MacOS105";
    EOSType[EOSType["MacOS1058"] = -99] = "MacOS1058";
    EOSType[EOSType["MacOS106"] = -95] = "MacOS106";
    EOSType[EOSType["MacOS1063"] = -94] = "MacOS1063";
    EOSType[EOSType["MacOS1064_slgu"] = -93] = "MacOS1064_slgu";
    EOSType[EOSType["MacOS1067"] = -92] = "MacOS1067";
    EOSType[EOSType["MacOS107"] = -90] = "MacOS107";
    EOSType[EOSType["MacOS108"] = -89] = "MacOS108";
    EOSType[EOSType["MacOS109"] = -88] = "MacOS109";
    EOSType[EOSType["MacOS1010"] = -87] = "MacOS1010";
    EOSType[EOSType["MacOS1011"] = -86] = "MacOS1011";
    EOSType[EOSType["MacOS1012"] = -85] = "MacOS1012";
    EOSType[EOSType["Macos1013"] = -84] = "Macos1013";
    EOSType[EOSType["Macos1014"] = -83] = "Macos1014";
    EOSType[EOSType["Macos1015"] = -82] = "Macos1015";
    EOSType[EOSType["MacOS1016"] = -81] = "MacOS1016";
    EOSType[EOSType["MacOS11"] = -80] = "MacOS11";
    EOSType[EOSType["MacOS111"] = -79] = "MacOS111";
    EOSType[EOSType["MacOS1017"] = -78] = "MacOS1017";
    EOSType[EOSType["MacOS12"] = -77] = "MacOS12";
    EOSType[EOSType["MacOS13"] = -76] = "MacOS13";
    EOSType[EOSType["MacOSMax"] = -1] = "MacOSMax";
    EOSType[EOSType["Unknown"] = -1] = "Unknown";
    EOSType[EOSType["WinUnknown"] = 0] = "WinUnknown";
    EOSType[EOSType["Win311"] = 1] = "Win311";
    EOSType[EOSType["Win95"] = 2] = "Win95";
    EOSType[EOSType["Win98"] = 3] = "Win98";
    EOSType[EOSType["WinME"] = 4] = "WinME";
    EOSType[EOSType["WinNT"] = 5] = "WinNT";
    EOSType[EOSType["Win200"] = 6] = "Win200";
    EOSType[EOSType["Win2000"] = 6] = "Win2000";
    EOSType[EOSType["WinXP"] = 7] = "WinXP";
    EOSType[EOSType["Win2003"] = 8] = "Win2003";
    EOSType[EOSType["WinVista"] = 9] = "WinVista";
    EOSType[EOSType["Win7"] = 10] = "Win7";
    EOSType[EOSType["Windows7"] = 10] = "Windows7";
    EOSType[EOSType["Win2008"] = 11] = "Win2008";
    EOSType[EOSType["Win2012"] = 12] = "Win2012";
    EOSType[EOSType["Win8"] = 13] = "Win8";
    EOSType[EOSType["Windows8"] = 13] = "Windows8";
    EOSType[EOSType["Win81"] = 14] = "Win81";
    EOSType[EOSType["Windows81"] = 14] = "Windows81";
    EOSType[EOSType["Win2012R2"] = 15] = "Win2012R2";
    EOSType[EOSType["Win10"] = 16] = "Win10";
    EOSType[EOSType["Windows10"] = 16] = "Windows10";
    EOSType[EOSType["Win2016"] = 17] = "Win2016";
    EOSType[EOSType["Win2019"] = 18] = "Win2019";
    EOSType[EOSType["Win2022"] = 19] = "Win2022";
    EOSType[EOSType["Win11"] = 20] = "Win11";
    EOSType[EOSType["WinMAX"] = 21] = "WinMAX";
})(EOSType || (EOSType = {}));
exports.default = EOSType;
