// steam-session/dist/enums-steam/EProtoClanEventType.js
"use strict";
/* eslint-disable */
// Auto-generated by generate-enums script on Thu Mar 02 2023 22:18:52 GMT-0500 (Eastern Standard Time)
Object.defineProperty(exports, "__esModule", { value: true });
var EProtoClanEventType;
(function (EProtoClanEventType) {
    EProtoClanEventType[EProtoClanEventType["k_EClanOtherEvent"] = 1] = "k_EClanOtherEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanGameEvent"] = 2] = "k_EClanGameEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanPartyEvent"] = 3] = "k_EClanPartyEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanMeetingEvent"] = 4] = "k_EClanMeetingEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanSpecialCauseEvent"] = 5] = "k_EClanSpecialCauseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanMusicAndArtsEvent"] = 6] = "k_EClanMusicAndArtsEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanSportsEvent"] = 7] = "k_EClanSportsEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanTripEvent"] = 8] = "k_EClanTripEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanChatEvent"] = 9] = "k_EClanChatEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanGameReleaseEvent"] = 10] = "k_EClanGameReleaseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanBroadcastEvent"] = 11] = "k_EClanBroadcastEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanSmallUpdateEvent"] = 12] = "k_EClanSmallUpdateEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanPreAnnounceMajorUpdateEvent"] = 13] = "k_EClanPreAnnounceMajorUpdateEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanMajorUpdateEvent"] = 14] = "k_EClanMajorUpdateEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanDLCReleaseEvent"] = 15] = "k_EClanDLCReleaseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanFutureReleaseEvent"] = 16] = "k_EClanFutureReleaseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanESportTournamentStreamEvent"] = 17] = "k_EClanESportTournamentStreamEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanDevStreamEvent"] = 18] = "k_EClanDevStreamEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanFamousStreamEvent"] = 19] = "k_EClanFamousStreamEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanGameSalesEvent"] = 20] = "k_EClanGameSalesEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanGameItemSalesEvent"] = 21] = "k_EClanGameItemSalesEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameBonusXPEvent"] = 22] = "k_EClanInGameBonusXPEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameLootEvent"] = 23] = "k_EClanInGameLootEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGamePerksEvent"] = 24] = "k_EClanInGamePerksEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameChallengeEvent"] = 25] = "k_EClanInGameChallengeEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameContestEvent"] = 26] = "k_EClanInGameContestEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanIRLEvent"] = 27] = "k_EClanIRLEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanNewsEvent"] = 28] = "k_EClanNewsEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanBetaReleaseEvent"] = 29] = "k_EClanBetaReleaseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameContentReleaseEvent"] = 30] = "k_EClanInGameContentReleaseEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanFreeTrial"] = 31] = "k_EClanFreeTrial";
    EProtoClanEventType[EProtoClanEventType["k_EClanSeasonRelease"] = 32] = "k_EClanSeasonRelease";
    EProtoClanEventType[EProtoClanEventType["k_EClanSeasonUpdate"] = 33] = "k_EClanSeasonUpdate";
    EProtoClanEventType[EProtoClanEventType["k_EClanCrosspostEvent"] = 34] = "k_EClanCrosspostEvent";
    EProtoClanEventType[EProtoClanEventType["k_EClanInGameEventGeneral"] = 35] = "k_EClanInGameEventGeneral";
})(EProtoClanEventType || (EProtoClanEventType = {}));
exports.default = EProtoClanEventType;
