// @bbob/plugin-helper/lib/char.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
_export(exports, {
    N: function() {
        return N;
    },
    F: function() {
        return F;
    },
    R: function() {
        return R;
    },
    EQ: function() {
        return EQ;
    },
    TAB: function() {
        return TAB;
    },
    SPACE: function() {
        return SPACE;
    },
    SLASH: function() {
        return SLASH;
    },
    BACKSLASH: function() {
        return BACKSLASH;
    },
    QUOTEMARK: function() {
        return QUOTEMARK;
    },
    OPEN_BRAKET: function() {
        return OPEN_BRAKET;
    },
    CLOSE_BRAKET: function() {
        return CLOSE_BRAKET;
    }
});
var N = "\n";
var TAB = "	";
var F = "\f";
var R = "\r";
var EQ = "=";
var QUOTEMARK = '"';
var SPACE = " ";
var OPEN_BRAKET = "[";
var CLOSE_BRAKET = "]";
var SLASH = "/";
var BACKSLASH = "\\";
